
        <!DOCTYPE html>



<!-- The following setting enables collapsible lists -->


<!-- Relative paths to support mirror sites with non-standard GB docs install -->
<!-- gbPageStartHardcoded.html - Include file for GB static pages on hgdownload to return to https:genome.ucsc.edu-->

<html>
    <head>
        <meta charset="UTF-8">
        <title>
            UCSC Genome Browser Downloads
        </title>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Oswald:700|Lato:700,700italic,300,400,400italic">
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">

        <!-- Redirects don't forward ROOT, so can't determine relative paths -->
        
        <!-- Use ROOT variable set in caller to determine relative path to style sheets -->

        
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
        

        <link rel="stylesheet" href="./style/gb.css">
        <link rel="stylesheet" href="./staticStyle/gbStatic.css">
        <LINK rel='STYLESHEET' href="./style/nice_menu.css" TYPE='text/css'>

        
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
        

    </head>

    <body>
  
      <!-- Hardcoded version of Blue menu bar / hotlinks developed for hgdownload server -->

        <noscript><div class='noscript'><div class='noscript-inner'><p><b>JavaScript is disabled in your web browser</b></p><p>You must have JavaScript enabled in your web browser to use the Genome Browser</p></div></div></noscript>
        <script type='text/javascript' SRC="./js/jquery.js"></script>
        <script type='text/javascript' SRC="./js/jquery.plugins.js"></script>
        <div id="main-menu-whole">
          <div id="main-menu"><div id="block-nice_menus-1" class="clear-block block block-nice_menus">
              <div class="upcorner"></div>
              <div class="blockbg">
                <div class="blockcontent">
                  <div class="content">

        <ul class="nice-menu nice-menu-down sf-js-enabled" id="nice-menu-1">
                      
          <li class="first even" id="home"><a href="https://genome.ucsc.edu/index.html" ><span id="home-link">Home</span></a></li> 

          <li class="menuparent" id="tools1"><a href="https://genome.ucsc.edu/cgi-bin/hgGateway">Genomes</a>
            <ul style="display: none; visibility: hidden;">
              <li class="first"><a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&amp;position=lastDbPos">Human GRCh38/hg38</a></li>
              <li><a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;position=lastDbPos">Human GRCh37/hg19</a></li>
              <li><a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=mm10&amp;position=lastDbPos">Mouse GRCm38/mm10</a></li>
              <li><a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=mm9&amp;position=lastDbPos">Mouse NCBI37/mm9</a></li>
              <li><a href="http://hgdownload.soe.ucsc.edu/hubs/mouseStrains/hubIndex.html">Mouse: 16 strains</a></li>
              <li class="last"><a href="https://genome.ucsc.edu/cgi-bin/hgGateway">Other</a></li>
            </ul>
          </li>

          <li class="menuparent" id="tools2"><a href="https://genome.ucsc.edu/cgi-bin/hgTracks">Genome Browser</a>
            <ul style="display: none; visibility: hidden;">
              <li class="first"><a id="configureMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgTracks?hgTracksConfigPage=configure">Configure</a></li>
              <li><a id="trackSearchMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgTracks?hgt_tSearch=track+search">Track Search</a></li>
              <li><a id="cartResetMenuLink" href="https://genome.ucsc.edu/cgi-bin/cartReset">Reset All User Settings</a></li>
            </ul>
          </li>

          <li class="menuparent" id="tools3"><span>Tools</span>
            <ul style="display: none; visibility: hidden;">
              <li class="first"><a id="blatMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgBlat?command=start">Blat</a></li>
              <li><a id="tableBrowserMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgTables">Table Browser</a></li>
              <li><a id="vaiMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgVai">Variant Annotation Integrator</a></li>
              <li><a id="diMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgIntegrator">Data Integrator</a></li>
              <li><a title="Explore gene/protein interactions and pathways" id="interactionMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgGeneGraph">Gene Interactions</a></li>
              <li><a id="gsMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgNear">Gene Sorter</a></li>
              <li><a id="ggMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgGenome">Genome Graphs</a></li>
              <li><a id="ispMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgPcr">In-Silico PCR</a></li>
              <li><a id="liftOverMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgLiftOver">LiftOver</a></li>
              <li><a id="visiGeneMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgVisiGene">VisiGene</a></li>
              <li class="last"><a href="https://genome.ucsc.edu/util.html">Other Utilities</a></li>
            </ul>
          </li>
         
          <li class="menuparent" id="mirrors"><span>Mirrors</span>
            <ul style="display: none; visibility: hidden;">
              <li class="first"><a href="https://genome.ucsc.edu/mirror.html">Third Party Mirrors</a></li>
              <li><a href="https://genome.ucsc.edu/goldenPath/help/mirror.html">Mirroring Instructions</a></li>
            </ul>
          </li>

          <li class="menuparent" id="downloads"><span>Downloads</span>
            <ul style="display: none; visibility: hidden;">
              <li class="first"><a href="http://hgdownload.soe.ucsc.edu/downloads.html">Genome Data</a></li>
              <li><a href="http://hgdownload.soe.ucsc.edu/downloads.html#source_downloads">Source Code</a></li>
              <li><a href="https://genome-store.ucsc.edu">Genome Browser Store</a></li>
              <li><a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">Utilities</a></li>
              <li class="last"><a href="https://genome.ucsc.edu/goldenPath/help/ftp.html">FTP</a></li>
              <li class="last"><a href="https://genome.ucsc.edu/goldenPath/help/mysql.html">MySQL Access</a></li>
            </ul>
          </li>
          
          <li class="menuparent" id="myData"><span>My Data</span>
            <ul style="display: none; visibility: hidden;">
              <li><a id="sessionsMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgSession?hgS_doMainPage=1">My Sessions</a></li>
              <li><a id="publicSessionsMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgPublicSessions">Public Sessions</a></li>
              <li><a id="trackHubsMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgHubConnect">Track Hubs</a></li>
              <li class="last"><a id="customTracksMenuLink" href="https://genome.ucsc.edu/cgi-bin/hgCustom">Custom Tracks</a></li>
            </ul>
          </li>

          <li class="menuparent" id="help"><span>Help</span><ul style="display: none; visibility: hidden;">
              <li><a href="https://genome.ucsc.edu/FAQ/">FAQs & Search</a></li>
              <li><a href="https://genome.ucsc.edu/goldenPath/help/hgTracksHelp.html">Browser Documentation</a></li>
              <li><a href="https://genome.ucsc.edu/training/index.html">Training</a></li>
              <li><a href="https://genome.ucsc.edu/contacts.html">Mailing Lists</a></li>
              <li class="last"><a href="http://genomewiki.ucsc.edu/" target="_blank">GenomeWiki</a></li>
            </ul>
          </li>
          
          <li class="menuparent last" id="aboutUs"><span>About Us</span>
            <ul style="display: none; visibility: hidden;">
              <li class="first odd"><a href="https://genome.ucsc.edu/goldenPath/newsarch.html">News</a></li>
              <li class="even"><a href="https://genome.ucsc.edu/goldenPath/pubs.html">Publications</a></li>
              <li class="even"><a href="http://genome.ucsc.edu/blog/">Blog</a></li>
              <li class="odd"><a href="https://genome.ucsc.edu/cite.html">Cite Us</a></li>
              <li class="even"><a href="https://genome.ucsc.edu/goldenPath/credits.html">Credits</a></li>
              <li class="odd"><a href="https://genome.ucsc.edu/goldenPath/releaseLog.html">Release Log</a></li>
              <li class="even"><a href="https://genome.ucsc.edu/staff.html">Staff</a></li>
              <li class="odd"><a href="https://genome.ucsc.edu/contacts.html">Contact Us</a></li>
              <li class="even"><a href="https://genome.ucsc.edu/conditions.html">Conditions of Use</a></li>
              <li class="odd"><a href="https://ucscgenomics.soe.ucsc.edu/careers/" TARGET="_blank">Jobs</a></li>
              <li class="even last"><a href="https://genome.ucsc.edu/license/">Licenses</a></li>
            </ul>
          </li>

        </ul>
                  </div>
                </div>
              </div>
              <div class="downcorner"></div>
              
            </div>
          </div><!-- end main-menu -->
        </div><!-- end main-menu-whole -->

        <!-- Grid layout from bootstrap.css -->
        <div class="container-fluid">
            <!-- Wrapper for GB content -->
            <div class="gbsPage">




<!-- Table of Contents -->
<h1>Sequence and Annotation Downloads</h1>
<p>
This page contains links to sequence and annotation data downloads for the genome assemblies
featured in the UCSC Genome Browser. Table downloads are also available via the Genome
Browser <a href="ftp://hgdownload.soe.ucsc.edu/goldenPath/" target="_blank">FTP server</a>. For 
quick access to the most recent assembly of each genome, see the
<a href="ftp://hgdownload.soe.ucsc.edu/goldenPath/currentGenomes/" 
target="_blank">current genomes</a> directory. This directory may be useful to individuals with 
automated scripts that must always reference the most recent assembly.</p> 
<p>
To view the current descriptions and formats of the tables in the annotation database, use
the &quot;describe table schema&quot; button in the Table Browser. </p>

<a name="terms"></a>
<p>
All tables in the Genome Browser are freely usable for any purpose except as indicated in the 
README.txt files in the download directories. To view restrictions specific to a particular data
set, click on the corresponding download link and review the README text. These data were 
contributed by many researchers, as listed on the Genome Browser 
<a href="http://genome.ucsc.edu/goldenPath/credits.html">credits</a> page. Please acknowledge the
contributor(s) of the data you use.  </p>

<p>
<a href="#human">Human</a></p>

<p class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mammals>Mammals</a>
  <div class="collapse" id="mammals">
  <ul>
    <li><a href="#alpaca">Alpaca</a>
    <li><a href="#armadillo">Armadillo</a>
    <li><a href="#sequence_only">Armadillo</a> (sequence only)
    <li><a href="#baboon">Baboon</a>
    <li><a href="#bison">Bison</a>
    <li><a href="#bonobo">Bonobo</a>
    <li><a href="#bushbaby">Bushbaby</a>
    <li><a href="#sequence_only">Bushbaby</a> (sequence only)
    <li><a href="#cat">Cat</a>
    <li><a href="#chimp">Chimpanzee</a>
    <li><a href="#chinese_hamster">Chinese hamster</a>
    <li><a href="#chinese_pangolin">Chinese pangolin</a>
    <li><a href="#cow">Cow</a>
    <li><a href="#crab-eating_macaque">Crab-eating_macaque</a>
    <li><a href="#denisova">Denisova</a></li>
    <li><a href="#dog">Dog</a>
    <li><a href="#dolphin">Dolphin</a>
    <li><a href="#elephant">Elephant</a>
    <li><a href="#ferret">Ferret</a>
    <li><a href="#gibbon">Gibbon</a>
    <li><a href="#golden_snub-nosed_monkey">Golden snub-nosed monkey</a>
    <li><a href="#gorilla">Gorilla</a>
    <li><a href="#green_monkey">Green monkey</a>
    <li><a href="#guinea_pig">Guinea pig</a>
    <li><a href="#hedgehog">Hedgehog</a>
    <li><a href="#horse">Horse</a>
    <li><a href="#sequence_only">J. Craig Venter</a> (sequence only)
    <li><a href="#kangaroo_rat">Kangaroo rat</a>
    <li><a href="#malayan_flying_lemur">Malayan flying lemur</a>
    <li><a href="#manatee">Manatee</a>
    <li><a href="#marmoset">Marmoset</a>
    <li><a href="#megabat">Megabat</a>
    <li><a href="#microbat">Microbat</a>
    <li><a href="#minke_whale">Minke whale</a>
    <li><a href="#mouse">Mouse</a>
    <li><a href="#mouse_lemur">Mouse lemur</a>
    <li><a href="#naked_mole-rat">Naked mole-rat</a>
    <li><a href="#opossum">Opossum</a>
    <li><a href="#orangutan">Orangutan</a>
    <li><a href="#panda">Panda</a>
    <li><a href="#pig">Pig</a>
    <li><a href="#pika">Pika</a>
    <li><a href="#platypus">Platypus</a>
    <li><a href="#proboscis_monkey">Proboscis monkey</a>
    <li><a href="#rabbit">Rabbit</a>
    <li><a href="#rat">Rat</a>
    <li><a href="#rhesus">Rhesus</a>
    <li><a href="#rock_hyrax">Rock hyrax</a>
    <li><a href="#sheep">Sheep</a>
    <li><a href="#shrew">Shrew</a>
    <li><a href="#sloth">Sloth</a>
    <li><a href="#squirrel">Squirrel</a>
    <li><a href="#squirrel_monkey">Squirrel monkey</a>
    <li><a href="#tarsier">Tarsier</a>
    <li><a href="#tasmanian_devil">Tasmanian devil</a>
    <li><a href="#tenrec">Tenrec</a>
    <li><a href="#tree_shrew">Tree shrew</a>
    <li><a href="#wallaby">Wallaby</a>
    <li><a href="#white_rhinoceros">White rhinoceros</a>
    <li><a href="#sequence_only">Data access for non-browser assemblies</a>

  </ul>
  </div>
</p>

<p class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href=#verts>Other vertebrates</a>
  <div class="collapse" id="verts">
  <ul>
    <li><a href="#african_clawed_frog">African clawed frog</a>
    <li><a href="#american_alligator">American alligator</a>
    <li><a href="#atlantic_cod">Atlantic cod</a>
    <li><a href="#brown_kiwi">Brown kiwi</a>
    <li><a href="#budgerigar">Budgerigar</a>
    <li><a href="#chicken">Chicken</a>
    <li><a href="#coelacanth">Coelacanth</a>
    <li><a href="#elephant_shark">Elephant shark</a>
    <li><a href="#fugu">Fugu</a>
    <li><a href="#garter_snake">Garter snake</a>
    <li><a href="#golden_eagle">Golden eagle</a>
    <li><a href="#lamprey">Lamprey</a>
    <li><a href="#lizard">Lizard</a>
    <li><a href="#medaka">Medaka</a>
    <li><a href="#medium_ground_finch">Medium ground finch</a>
    <li><a href="#nile_tilapia">Nile tilapia</a>
    <li><a href="#painted_turtle">Painted turtle</a>
    <li><a href="#stickleback">Stickleback</a>
    <li><a href="#tetraodon">Tetraodon</a>
    <li><a href="#tibetan_frog">Tibetan frog</a>
    <li><a href="#turkey">Turkey</a>
    <li><a href="#xentro">X. tropicalis</a>
    <li><a href="#zebra_finch">Zebra finch</a>
    <li><a href="#zebrafish">Zebrafish</a>
  </ul>
  </div>
</p>

<p class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href=#deuts>Deuterostomes</a>
  <div class="collapse" id="deuts">
  <ul>
    <li><a href="#ciona">C. intestinalis</a>
    <li><a href="#lancelet">Lancelet</a>
    <li><a href="#urchin">S. purpuratus</a>
  </ul>
  </div>
</p>

<p class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href=#insects>Insects</a>
  <div class="collapse" id="insects">
  <ul>
    <li><a href="#anoGam">A. gambiae</a></li>
    <li><a href="#apiMel">A. mellifera</a></li>
    <li><a href="#droAna">D. ananassae</a></li>
    <li><a href="#droEre">D. erecta</a></li>
    <li><a href="#droGri">D. grimshawi</a></li>
    <li><a href="#fruitfly">D. melanogaster</a></li>
    <li><a href="#droMoj">D. mojavensis</a></li>
    <li><a href="#droPer">D. persimilis</a></li>
    <li><a href="#dp">D. pseudoobscura</a></li>
    <li><a href="#droSec">D. sechellia</a></li>
    <li><a href="#droSim">D. simulans</a></li>
    <li><a href="#droVir">D. virilis</a></li>
    <li><a href="#droWil">D. willistoni</a></li>
    <li><a href="#droYak">D. yakuba</a></li>
    <li><a href="#triCas">T. castaneum</a></li>
  </ul>
  </div>
</p>

<p class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href=#nematodes>Nematodes</a>
  <div class="collapse" id="nematodes">
  <ul>
    <li><a href="#caePb">C. brenneri</a></li>
    <li><a href="#c_briggsae">C. briggsae</a></li>
    <li><a href="#c_elegans">C. elegans</a></li>
    <li><a href="#caeJap">C. japonica</a></li>
    <li><a href="#caeRem">C. remanei</a></li>
    <li><a href="#priPac">P. pacificus</a></li>
  </ul>
  </div>
</p>

<p class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href=#others>Other genomes</a>
  <div class="collapse" id="others">
  <ul>
    <li><a href="#denisova">Denisova</a></li>
    <li><a href="#yeast">S. cerevisiae</a></li>
    <li><a href="#seahare">Sea hare</a></li>
    <li><a href="#ebola_virus">Ebola virus</li>
  </ul>
  </div>
</p>

<p class="collapse-section">
  <a class="collapsed collapse-toggle" data-toggle="collapse" href=#other_downloads>Other downloads</a>
  <div class="collapse" id="other_downloads">
  <ul>
    <li><a href="#gbdb">Access the gbdb fileserver</a></li>
    <li><a href="#shared">Shared data (Protein DBs, hgFixed, visiGene)</a></li>
    <li><a href="#liftover">LiftOver files</a></li>
    <li><a href="http://genome.ucsc.edu/ENCODE/downloads.html" 
      target="_blank">ENCODE Project Files (Genome-wide Phase)</li>
    <li><a href="../goldenPath/encode/"
      target="_blank">ENCODE Project Files (Pilot Phase)</li>
  </ul>
  </div>
</p>

<!-- Source & Utilities Downloads -->
<a name="source_downloads"></a>
<h2>Source and utilities downloads</h2>
<p>
The source for the Genome Browser, Blat, liftOver and other utilities is free for non-profit 
academic research and for personal use. For information on commercial licensing, see the 
<a href="http://genome.ucsc.edu/license/">Genome Browser</a> and
<a href="http://www.kentinformatics.com" target="_blank">Blat</a> licensing requirements.
The source and executables for several of these products can be downloaded or purchased from our 
<a href="https://genome-store.ucsc.edu">online store</a>.</p>

<h3>Mirroring the Genome Browser</h3>
<p>
Precompiled executable binaries are available for installing a local mirrored copy of the Genome 
Browser website on your web server, eliminating the need to compile the entire source tree.</p> 
<ul> 
  <li>
  <a href="http://genome.ucsc.edu/admin/mirror.html">Mirror instructions</a></li> 
  <li>
  <a href="http://genome-source.soe.ucsc.edu/gitlist/kent.git/tree/master/src/product">Build 
  instructions</a></li> 
  <li>
  <a href="http://hgdownload.soe.ucsc.edu/admin/hgcentral.sql">Genome Browser hgcentral 
  tables</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#GBsource>Genome Browser source
    code downloads</a>
    <div class="collapse" id="GBsource">
    <ul> 
      <li>
      <a href="http://genome.ucsc.edu/admin/git.html">Access source using git</a></li> 
      <li>
      <a href="http://hgdownload.soe.ucsc.edu/admin/jksrc.zip">Download source code .zip file</a></li> 
      <li>
      <a href="https://genome-store.ucsc.edu/">Access source via online store</a></li> 
      </ul>
    </div>
  </li> 
</ul>
<p>
If you encounter difficulties with slow download speeds, try using 
<a href="https://github.com/LabAdvComp/UDR" target="_blank">UDT Enabled Rsync</a> (UDR), which 
improves the throughput of large data transfers over long distances. The 32-bit and 64-bit versions 
can be downloaded <a href="http://hgdownload.soe.ucsc.edu/admin/udr/">here</a>.</p>

<a name="utilities_downloads"></a>
<h3>Utilities</h3>
<p>
The <a href="http://hgdownload.soe.ucsc.edu/admin/exe/">utilities directory</a> offers downloads of 
pre-compiled standalone binaries for:</p>
<ul>
  <li>
  LiftOver (which may also be accessed via the 
  <a href="http://genome.ucsc.edu/cgi-bin/hgLiftOver">web version</a>). The over.chain liftOver 
  conversion files are located in the individual assembly download sections.</li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#blat_source>Blat</a>
    <div class="collapse" id="blat_source">
      <ul>
        <li>
        <a href="https://users.soe.ucsc.edu/~kent/src/">Source code download</a> (look for the
        blatSrc* zip file with the most recent date) </li>
        <li>
        <a href="http://genome.ucsc.edu/goldenPath/help/blatSpec.html">Blat documentation</a></li>
        <li>
        The <a href="http://hgdownload.soe.ucsc.edu/admin/exe/">utilities directory</a>
        contains pre-compiled standalone blat, gfServer, and gfClient binaries nested within
        a blat subdirectory for each system</li>
      </ul>
    </div>
  </li>
  <li>
  Other command-line utilities</li><br>
<b>Note about 'permission denied' error</b>:<br> 
In order for your computer to run a freshly downloaded utility, you will need to update the file
system permissions to allow your operating system to run the program.<br>
	 To make utilities usable, turn on its 'executable'
	 bit: 
	<pre><code>$ chmod +x ./filePath </code></pre>
	<pre><code>$ ./filePath/utility_name</code></pre>
	Example:
	<pre>$ chmod +x /home/user/liftover/liftOver</pre>
	See also: <a href="http://en.wikipedia.org/wiki/Chmod" target="_blank">http://en.wikipedia.org/wiki/Chmod</a> 
</ul>
<p>
Please review the userApps 
<a href="http://genome-source.soe.ucsc.edu/gitlist/kent.git/blob/master/src/userApps/README">README</a>
for information on fetching specific directories from the kent source tree or downloading
userApps.src.tgz to install all kent utilities.</p>
<p>
Note that commercial download and installation of the Blat and In-Silico PCR software requires
a licence, which may be obtained from <a href="http://www.kentinformatics.com"
target="_blank">Kent Informatics</a>. </p>

<!-- Human Downloads -->
<a name="human"></a>
<h2>Human genome</h2>
<h3>Dec. 2013 (GRCh38/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=hg38"
    target="_blank">hg38</a>)</h3>
<ul>
<!--
<ul class="gbsNoBullet">
-->
  <li><a href="/goldenPath/hg38/bigZips/">Full data set</a>
  <li><a href="/goldenPath/hg38/chromosomes/">Data set by chromosome</a>
  <li><a href="/goldenPath/hg38/database/">Annotation database</a>

  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg38snp>SNP-masked fasta 
    files</a>
    <div class="collapse" id="hg38snp">
    <ul>
      <li><a href="/goldenPath/hg38/snp151Mask/">SNP151-masked FASTA files</a> </li> 
      <li><a href="/goldenPath/hg38/snp150Mask/">SNP150-masked FASTA files</a> </li> 
      <li><a href="/goldenPath/hg38/snp147Mask/">SNP147-masked FASTA files</a> </li> 
      <li><a href="/goldenPath/hg38/snp146Mask/">SNP146-masked FASTA files</a> </li>
      <li><a href="/goldenPath/hg38/snp144Mask/">SNP144-masked FASTA files</a> </li>
      <li><a href="/goldenPath/hg38/snp142Mask/">SNP142-masked FASTA files</a> </li>
      <li><a href="/goldenPath/hg38/snp141Mask/">SNP141-masked FASTA files</a></li> 
    </ul>
    </div>
  </li>

  <li><a href="/goldenPath/hg38/liftOver/">LiftOver files</a>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg38pairwise>Pairwise 
    alignments</a>
    <div class="collapse" id="hg38pairwise">
    <ul> 
      <li><a href="/goldenPath/hg38/vsSelf/">Human self alignments</a></li> 
      <li><a href="/goldenPath/hg38/vsVicPac2/">Human/Alpaca (vicPac2)</a></li> 
      <li><a href="/goldenPath/hg38/vsXenLae2/">Human/African clawed frog (xenLae2)</a></li>
      <li><a href="/goldenPath/hg38/vsPapAnu4/">Human/Baboon (papAnu4)</a></li>
      <li><a href="/goldenPath/hg38/vsPapAnu2/">Human/Baboon (papAnu2)</a></li> 
      <li><a href="/goldenPath/hg38/vsPanPan2/">Human/Bonobo (panPan2)</a></li>
      <li><a href="/goldenPath/hg38/vsPanPan1/">Human/Bonobo (panPan1)</a></li> 
      <li><a href="/goldenPath/hg38/vsAptMan1/">Human/Brown kiwi (aptMan1)</a></li>
      <li><a href="/goldenPath/hg38/vsFelCat8/">Human/Cat (felCat8)</a></li>
      <li><a href="/goldenPath/hg38/vsFelCat9/">Human/Cat (felCat9)</a></li> 
      <li><a href="/goldenPath/hg38/vsGalGal6/">Human/Chicken (galGal6)</a></li> 
      <li><a href="/goldenPath/hg38/vsManPen1/">Human/Chinese Pangolin (manPen1)</a></li> 
      <li><a href="/goldenPath/hg38/vsCriGriChoV1/">Human/Chinese hamster ovary (CHO) K1 cell line
        (criGriChoV1)</a></li> 
      <li><a href="/goldenPath/hg38/vsCriGriChoV2/">Human/Chinese hamster ovary (CHO) K1 cell line (criGriChoV2)</a></li>
      <li><a href="/goldenPath/hg38/vsPanTro6/">Human/Chimp (panTro6)</a></li> 
      <li><a href="/goldenPath/hg38/vsPanTro5/">Human/Chimp (panTro5)</a></li> 
      <li><a href="/goldenPath/hg38/vsPanTro4/">Human/Chimp (panTro4)</a></li> 
      <li><a href="/goldenPath/hg38/vsBosTau9/">Human/Cow (bosTau9)</a></li>
      <li><a href="/goldenPath/hg38/vsBosTau8/">Human/Cow (bosTau8)</a></li> 
      <li><a href="/goldenPath/hg38/vsMacFas5/">Human/Crab-eating macaque (macFas5)</a></li> 
      <li><a href="/goldenPath/hg38/vsTurTru2/">Human/Dolphin (turTru2)</a></li> 
      <li><a href="/goldenPath/hg38/vsCanFam3/">Human/Dog (canFam3)</a></li> 
      <li><a href="/goldenPath/hg38/vsMusFur1/">Human/Ferret (musFur1)</a></li>
      <li><a href="/goldenPath/hg38/vsThaSir1/">Human/Garter snake (thaSir1)</a></li> 
      <li><a href="/goldenPath/hg38/vsAquChr2/">Human/Golden eagle (aquChr2)</a></li> 
      <li><a href="/goldenPath/hg38/vsGorGor5/">Human/Gorilla (gorGor5)</a></li> 
      <li><a href="/goldenPath/hg38/vsGorGor4/">Human/Gorilla (gorGor4)</a></li> 
      <li><a href="/goldenPath/hg38/vsChlSab2/">Human/Green Monkey (chlSab2)</a></li> 
      <li><a href="/goldenPath/hg38/vsEquCab2/">Human/Horse (equCab2)</a></li>
      <li><a href="/goldenPath/hg38/vsPetMar3/">Human/Lamprey (petMar3)</a></li>
      <li><a href="/goldenPath/hg38/vsGalVar1/">Human/Malayan flying lemur (galVar1)</a></li> 
      <li><a href="/goldenPath/hg38/vsTriMan1/">Human/Manatee (triMan1)</a></li> 
      <li><a href="/goldenPath/hg38/vsCalJac3/">Human/Marmoset (calJac3)</a></li> 
      <li><a href="/goldenPath/hg38/vsMm10/">Human/Mouse (mm10)</a></li> 
      <li><a href="/goldenPath/hg38/vsMicMur2/">Human/Mouse Lemur (micMur2)</a></li> 
      <li><a href="/goldenPath/hg38/vsHetGla2/">Human/Naked Mole Rat (hetGla2)</a></li> 
      <li><a href="/goldenPath/hg38/vsMonDom5/">Human/Opossum (monDom5)</a></li>
      <li><a href="/goldenPath/hg38/vsPonAbe2/">Human/Orangutan(ponAbe2)</a></li>
      <li><a href="/goldenPath/hg38/vsPonAbe3/">Human/Orangutan(ponAbe3)</a></li> 
      <li><a href="/goldenPath/hg38/vsSusScr11/">Human/Pig (susScr11)</a></li>
      <li><a href="/goldenPath/hg38/vsOrnAna2/">Human/Platypus (ornAna2)</a></li> 
      <li><a href="/goldenPath/hg38/vsNasLar1/">Human/Proboscis Monkey (nasLar1)</a></li>
      <li><a href="/goldenPath/hg38/vsOryCun2/">Human/Rabbit (oryCun2)</a></li> 
      <li><a href="/goldenPath/hg38/vsRn6/">Human/Rat (rn6)</a></li> 
      <li><a href="/goldenPath/hg38/vsRn5/">Human/Rat (rn5)</a></li> 
      <li><a href="/goldenPath/hg38/vsRheMac8/">Human/Rhesus (rheMac8)</a></li> 
      <li><a href="/goldenPath/hg38/vsRheMac3/">Human/Rhesus (rheMac3)</a></li> 
      <li><a href="/goldenPath/hg38/vsOviAri4/">Human/Sheep (oviAri4)</a></li>    
      <li><a href="/goldenPath/hg38/vsTarSyr2/">Human/Tarsier (tarSyr2)</a></li> 
      <li><a href="/goldenPath/hg38/vsMacEug2/">Human/Wallaby (macEug2)</a></li> 
      <li><a href="/goldenPath/hg38/vsXenTro7/">Human/X. tropicalis(xenTro7)</a></li> 
      <li><a href="/goldenPath/hg38/vsDanRer10/">Human/Zebrafish (danRer10)</a></li>
    </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg38mult>Multiple 
    alignments</a>
    <div class="collapse" id="hg38mult">
    <ul>
      <li>
      <a href="/goldenPath/hg38/multiz100way/">Multiple alignments of 99 vertebrate genomes with 
      human</a> </li>
      <li>
      <a href="/goldenPath/hg38/phastCons100way/">Conservation scores for alignments of 99
      vertebrate genomes with human</a> </li>
      <li>
      <a href="/goldenPath/hg38/phyloP100way/">Basewise conservation scores (phyloP) of 99
      vertebrate genomes with human</a> </li>
      <li>
      <a href="/goldenPath/hg38/multiz100way/alignments/">FASTA alignments of 99 vertebrate genomes 
      with human for CDS regions</a> </li>
      <li>
      <a href="/goldenPath/hg38/multiz20way/">Multiple alignments of 19 mammalian (16 primate) 
      genomes with human</a> </li>
      <li>
      <a href="/goldenPath/hg38/phastCons20way/">Conservation scores for alignments of 19 mammalian 
      (16 primate) genomes with human</a> </li>
      <li>
      <a href="/goldenPath/hg38/phyloP20way/">Basewise conservation scores (phyloP) of 19 mammalian 
      (16 primate) genomes with human</a> </li>
      <li>
      <a href="/goldenPath/hg38/multiz20way/alignments/">FASTA alignments of 19 mammalian (16 
      primate) genomes with human for CDS regions</a> </li>
      <li>
      <a href="/goldenPath/hg38/multiz7way/">Multiple alignments of 6 vertebrate genomes with 
      human</a> </li>
      <li>
      <a href="/goldenPath/hg38/phastCons7way/">Conservation scores for alignments of 6 vertebrate 
      genomes with human</a> </li>
      <li>
      <a href="/goldenPath/hg38/phyloP7way/">Basewise conservation scores (phyloP) of 6 vertebrate 
      genomes with human</a> </li>
      <li>
      <a href="/goldenPath/hg38/multiz7way/alignments/">FASTA alignments of 6 vertebrate genomes 
      with human for CDS regions</a> </li>
      <li>
      <a href="/goldenPath/hg38/multiz30way/">Multiple alignments of 30 mammalian (27 primates) 
      genomes with human</a> </li>
      <li>
      <a href="/goldenPath/hg38/phastCons30way/">Conservation scores for alignments of 30 mammalian
      (27 primate) genomes with human</a> </li>
     <li>
     <a href="/goldenPath/hg38/phyloP30way/">Basewise conservation scores (phyloP) of 30 mammalian
     (27 primate) genomes with human</a> </li>
     <li>
     <a href="/goldenPath/hg38/multiz30way/alignments/">FASTA alignments of 30 mammalian
     (27 primate) genomes with human for CDS regions</a> </li>




  </ul> 
    </div>
  </li>

  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg38patch>Patches</a>
    <div class="collapse" id="hg38patch">
    <ul> 
      <li><a href="/goldenPath/hg38/hg38Patch11/">GRCh38 Patch 11 - Full data set</a></li>
      <li><a href="/goldenPath/hg38/hg38Patch9/">GRCh38 Patch 9 - Full data set</a></li>
      <li><a href="/goldenPath/hg38/hg38Patch7/">GRCh38 Patch 7 - Full data set</a></li> 
      <li><a href="/goldenPath/hg38/hg38Patch6/">GRCh38 Patch 6 - Full data set</a></li>
      <li><a href="/goldenPath/hg38/hg38Patch3/">GRCh38 Patch 3 - Full data set</a></li>
      <li><a href="/goldenPath/hg38/hg38Patch2/">GRCh38 Patch 2 - Full data set</a></li>
     </ul>
    </div>
  </li>
</ul>

<!-- hg19 -->
<h3>Feb. 2009 (GRCh37/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=hg19"
    target="_blank">hg19</a>)</h3>
<ul> 
  <li><a href="/goldenPath/hg19/bigZips/">Full data set</a> 
  <li><a href="/goldenPath/hg19/chromosomes/">Data set by chromosome</a> 
  <li><a href="/goldenPath/hg19/database/">Annotation database</a> 
  <li><a href="/goldenPath/hg19/gc5Base/">GC percent data</a> 
  <li><a href="/goldenPath/proteinDB/proteins140122/database/">Protein database for hg19</a> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg37snp>SNP-masked fasta 
    files</a>
    <div class="collapse" id="hg37snp">
    <ul>
      <li><a href="/goldenPath/hg19/snp151Mask/">SNP151-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp150Mask/">SNP150-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp147Mask/">SNP147-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp146Mask/">SNP146-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp144Mask/">SNP144-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp142Mask/">SNP142-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp141Mask/">SNP141-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp138Mask/">SNP138-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp137Mask/">SNP137-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp135Mask/">SNP135-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp132Mask/">SNP132-masked FASTA files</a></li>
      <li><a href="/goldenPath/hg19/snp131Mask/">SNP131-masked FASTA files</a></li>
    </ul>
    </div>
  </li>
  <li><a href="/goldenPath/hg19/liftOver/">LiftOver files</a>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg37-primate-pairwise>Pairwise
    alignments (primates)</a>
    <div class="collapse" id="hg37-primate-pairwise">
    <ul> 
      <li><a href="/goldenPath/hg19/vsSelf/">Human self alignments</a></li> 
      <li><a href="/goldenPath/hg19/vsPapHam1/">Human/Baboon (papHam1)</a></li> 
      <li><a href="/goldenPath/hg19/vsPapAnu2/">Human/Baboon (papAnu2)</a></li> 
      <li><a href="/goldenPath/hg19/vsOtoGar1/">Human/Bushbaby (otoGar1)</a></li> 
      <li><a href="/goldenPath/hg19/vsPanTro6/">Human/Chimp (panTro6)</a></li>
      <li><a href="/goldenPath/hg19/vsPanTro5/">Human/Chimp (panTro5)</a></li> 
      <li><a href="/goldenPath/hg19/vsPanTro4/">Human/Chimp (panTro4)</a></li> 
      <li><a href="/goldenPath/hg19/vsPanTro3/">Human/Chimp (panTro3)</a></li> 
      <li><a href="/goldenPath/hg19/vsPanTro2/">Human/Chimp (panTro2)</a></li> 
      <li><a href="/goldenPath/hg19/vsMacFas5/">Human/Crab-eating macaque (macFas5)</a></li> 
      <li><a href="/goldenPath/hg19/vsNomLeu1/">Human/Gibbon (nomLeu1)</a></li> 
      <li><a href="/goldenPath/hg19/vsGorGor5/">Human/Gorilla (gorGor5)</a>
      <li><a href="/goldenPath/hg19/vsGorGor3/">Human/Gorilla (gorGor3)</a> 
      <li><a href="/goldenPath/hg19/vsGorGor1/">Human/Gorilla (gorGor1)</a></li> 
      <li><a href="/goldenPath/hg19/vsCalJac3/">Human/Marmoset (calJac3)</a></li> 
      <li><a href="/goldenPath/hg19/vsCalJac1/">Human/Marmoset (calJac1)</a></li> 
      <li><a href="/goldenPath/hg19/vsMicMur1/">Human/Mouse lemur (micMur1)</a></li> 
      <li><a href="/goldenPath/hg19/vsPonAbe2/">Human/Orangutan (ponAbe2)</a></li> 
      <li><a href="/goldenPath/hg19/vsRheMac3/">Human/Rhesus (rheMac3)</a></li> 
      <li><a href="/goldenPath/hg19/vsRheMac2/">Human/Rhesus (rheMac2)</a></li> 
      <li><a href="/goldenPath/hg19/vsTarSyr1/">Human/Tarsier (tarSyr1)</a></li> 
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg37-mammal-pairwise>Pairwise 
    alignments (other mammals)</a>
    <div class="collapse" id="hg37-mammal-pairwise">
    <ul> 
      <li><a href="/goldenPath/hg19/vsVicPac2/">Human/Alpaca (vicPac2)</a></li> 
      <li><a href="/goldenPath/hg19/vsVicPac1/">Human/Alpaca (vicPac1)</a></li> 
      <li><a href="/goldenPath/hg19/vsDasNov2/">Human/Armadillo (dasNov2)</a></li> 
      <li><a href="/goldenPath/hg19/vsFelCat5/">Human/Cat (felCat5)</a></li> 
      <li><a href="/goldenPath/hg19/vsFelCat4/">Human/Cat (felCat4)</a></li> 
      <li><a href="/goldenPath/hg19/vsFelCat3/">Human/Cat (felCat3)</a></li> 
      <li><a href="/goldenPath/hg19/vsCriGri1/">Human/Chinese hamster (criGri1)</a></li> 
      <li><a href="/goldenPath/hg19/vsBosTau7/">Human/Cow (bosTau7)</a></li> 
      <li><a href="/goldenPath/hg19/vsBosTau6/">Human/Cow (bosTau6)</a></li> 
      <li><a href="/goldenPath/hg19/vsBosTau4/">Human/Cow (bosTau4)</a></li> 
      <li><a href="/goldenPath/hg19/vsTurTru1/">Human/Dolphin (turTru1)</a></li> 
      <li><a href="/goldenPath/hg19/vsCanFam3/">Human/Dog (canFam3)</a></li> 
      <li><a href="/goldenPath/hg19/vsCanFam2/">Human/Dog (canFam2)</a></li> 
      <li><a href="/goldenPath/hg19/vsLoxAfr3/">Human/Elephant (loxAfr3)</a></li> 
      <li><a href="/goldenPath/hg19/vsCavPor3/">Human/Guinea pig (cavPor3)</a></li> 
      <li><a href="/goldenPath/hg19/vsEriEur2/">Human/Hedgehog (eriEur2)</a></li> 
      <li><a href="/goldenPath/hg19/vsEriEur1/">Human/Hedgehog (eriEur1)</a></li> 
      <li><a href="/goldenPath/hg19/vsEquCab2/">Human/Horse (equCab2)</a></li> 
      <li><a href="/goldenPath/hg19/vsDipOrd1/">Human/Kangaroo rat (dipOrd1)</a></li> 
      <li><a href="/goldenPath/hg19/vsMyoLuc1/">Human/Microbat (myoLuc1)</a></li> 
      <li><a href="/goldenPath/hg19/vsPteVam1/">Human/Megabat (pteVam1)</a></li> 
      <li><a href="/goldenPath/hg19/vsMm10/">Human/Mouse (mm10)</a></li> 
      <li><a href="/goldenPath/hg19/vsMm9/">Human/Mouse (mm9)</a></li> 
      <li><a href="/goldenPath/hg19/vsMonDom5/">Human/Opossum (monDom5)</a></li> 
      <li><a href="/goldenPath/hg19/vsAilMel1/">Human/Panda (ailMel1)</a></li> 
      <li><a href="/goldenPath/hg19/vsSusScr2/">Human/Pig (susScr2)</a></li> 
      <li><a href="/goldenPath/hg19/vsOchPri3/">Human/Pika (ochPri3)</a></li> 
      <li><a href="/goldenPath/hg19/vsOchPri2/">Human/Pika (ochPri2)</a></li> 
      <li><a href="/goldenPath/hg19/vsOryCun2/">Human/Rabbit (oryCun2)</a></li> 
      <li><a href="/goldenPath/hg19/vsRn6/">Human/Rat (rn6)</a></li> 
      <li><a href="/goldenPath/hg19/vsRn4/">Human/Rat (rn4)</a></li> 
      <li><a href="/goldenPath/hg19/vsProCap1/">Human/Rock hyrax (proCap1)</a></li>
      <li><a href="/goldenPath/hg19/vsOviAri3/">Human/Sheep (oviAri3)</a></li> 
      <li><a href="/goldenPath/hg19/vsOviAri1/">Human/Sheep (oviAri1)</a></li> 
      <li><a href="/goldenPath/hg19/vsSorAra2/">Human/Shrew (sorAra2)</a></li> 
      <li><a href="/goldenPath/hg19/vsSorAra1/">Human/Shrew (sorAra1)</a></li> 
      <li><a href="/goldenPath/hg19/vsChoHof1/">Human/Sloth (choHof1)</a></li> 
      <li><a href="/goldenPath/hg19/vsSpeTri1/">Human/Squirrel (speTri1)</a></li> 
      <li><a href="/goldenPath/hg19/vsSarHar1/">Human/Tasmanian devil (sarHar1)</a></li> 
      <li><a href="/goldenPath/hg19/vsEchTel2/">Human/Tenrec (echTel2)</a></li> 
      <li><a href="/goldenPath/hg19/vsEchTel1/">Human/Tenrec (echTel1)</a></li> 
      <li><a href="/goldenPath/hg19/vsTupBel1/">Human/Tree shrew (tupBel1)</a></li> 
      <li><a href="/goldenPath/hg19/vsMacEug1/">Human/Wallaby (macEug1)</a></li> 
      <li><a href="/goldenPath/hg19/vsCerSim1/">Human/White rhinoceros (cerSim1)</a></li> 
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg19-vert>Pairwise
    alignments (other vertebrates)</a>
    <div class="collapse" id="hg19-vert"> 
      <ul>
      <li><a href="/goldenPath/hg19/vsAllMis1/">Human/Alligator (allMis1)</a></li>
      <li><a href="/goldenPath/hg38/vsGalGal5/">Human/Chicken (galGal5)</a></li> 
      <li><a href="/goldenPath/hg38/vsGalGal4/">Human/Chicken (galGal4)</a></li>
      <li><a href="/goldenPath/hg19/vsGalGal3/">Human/Chicken (galGal3)</a></li> 
      <li><a href="/goldenPath/hg19/vsFr2/">Human/Fugu (fr2)</a></li> 
      <li><a href="/goldenPath/hg19/vsPetMar2/">Human/Lamprey (petMar2)</a></li> 
      <li><a href="/goldenPath/hg19/vsPetMar1/">Human/Lamprey (petMar1)</a></li> 
      <li><a href="/goldenPath/hg19/vsAnoCar2/">Human/Lizard (anoCar2)</a></li> 
      <li><a href="/goldenPath/hg19/vsAnoCar1/">Human/Lizard (anoCar1)</a></li> 
      <li><a href="/goldenPath/hg19/vsOryLat2/">Human/Medaka (oryLat2)</a></li> 
      <li><a href="/goldenPath/hg19/vsGasAcu1/">Human/Stickleback (gasAcu1)</a></li> 
      <li><a href="/goldenPath/hg19/vsTetNig2/">Human/Tetraodon (tetNig2)</a></li> 
      <li><a href="/goldenPath/hg19/vsTetNig1/">Human/Tetraodon (tetNig1)</a></li> 
      <li><a href="/goldenPath/hg19/vsMelGal1/">Human/Turkey (melGal1)</a></li> 
      <li><a href="/goldenPath/hg19/vsXenTro3/">Human/X. tropicalis(xenTro3)</a></li> 
      <li><a href="/goldenPath/hg19/vsXenTro2/">Human/X. tropicalis(xenTro2)</a></li> 
      <li><a href="/goldenPath/hg19/vsTaeGut2/">Human/Zebra finch (taeGut2)</a></li> 
      <li><a href="/goldenPath/hg19/vsTaeGut1/">Human/Zebra finch (taeGut1)</a></li> 
      <li><a href="/goldenPath/hg19/vsDanRer10/">Human/Zebrafish (danRer10)</a></li> 
      <li><a href="/goldenPath/hg19/vsDanRer7/">Human/Zebrafish (danRer7)</a></li> 
      <li><a href="/goldenPath/hg19/vsDanRer6/">Human/Zebrafish (danRer6)</a></li> 
      <li><a href="/goldenPath/hg19/vsDanRer5/">Human/Zebrafish (danRer5)</a></li> 
    </ul>
    </div>
  </li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg37mult>Multiple 
    alignments</a>
    <div class="collapse" id="hg37mult">
    <ul> 
      <li>
      <a href="/goldenPath/hg19/multiz100way/">Multiple alignments of 99 vertebrate genomes with 
      human</a></li> 
      <li>
      <a href="/goldenPath/hg19/phastCons100way/">Conservation scores for alignments of 99 
      vertebrate genomes with human</a></li> 
      <li>
      <a href="/goldenPath/hg19/phyloP100way/">Basewise conservation scores (phyloP) of 99 
      vertebrate genomes with human</a></li> 
      <li>
      <a href="/goldenPath/hg19/multiz100way/alignments/">FASTA alignments of 99 vertebrate genomes 
      with human for CDS regions</a></li> 
      <li>
      <a href="/goldenPath/hg19/multiz46way/">Multiple alignments of 45 vertebrate genomes with 
      human</a></li> 
      <li>
      <a href="/goldenPath/hg19/phastCons46way/">Conservation scores for alignments of 45 vertebrate
      genomes with human</a></li> 
      <li>
      <a href="/goldenPath/hg19/phyloP46way/">Basewise conservation scores (phyloP) of 45 vertebrate
       genomes with human</a></li> 
      <li>
      <a href="/goldenPath/hg19/multiz46way/alignments/">FASTA alignments of 45 vertebrate genomes 
      with human for CDS regions</a></li> 
    </ul> 
    </div>
  </li>

  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg19patch>Patches</a>
    <div class="collapse" id="hg19patch">
    <ul> 
      <li><a href="/goldenPath/hg19/hg19Patch13/">GRCh37 Patch 13 - Full data set</a></li>
     </ul>
    </div>
  </li>
</ul>

<!-- hg18 -->
<h3>Mar. 2006 (NCBI36/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=hg18"
    target="_blank">hg18</a>)</h3>
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg18Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg18Downloads">
    <ul>
      <li><a href="/goldenPath/hg18/bigZips/">Full data set</a></li> 
      <li><a href="/goldenPath/hg18/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg18/database/">Annotation database</a></li>
      <li><a href="/goldenPath/hg18/affyTxnPhase3/">Affymetrix transcriptome phase 3 data</a></li>
      <li><a href="/goldenPath/hg18/regPotential7X/">Regulatory potential data</a></li>
      <li><a href="/goldenPath/hg18/gc5Base/">GC percent data</a></li>
      <li class="collapse-section">
	<a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg18snp>SNP-masked FASTA 
	files</a>
	<div class="collapse" id="hg18snp">
	<ul>
	  <li><a href="/goldenPath/hg18/snp130Mask/">SNP130-masked FASTA files</a></li>
	  <li><a href="/goldenPath/hg18/snp129Mask/">SNP129-masked FASTA files</a></li>
	  <li><a href="/goldenPath/hg18/snp128Mask/">SNP128-masked FASTA files</a></li>
	</ul>
	</div>
      </li>
      <li><a href="/goldenPath/hg18/liftOver/">LiftOver files</a></li>
      <li><a href="http://genome.ucsc.edu/ENCODE/downloads.html">ENCODE production phase whole-genome 
      data</a></li>
      <li><a href="/goldenPath/hg18/pilot/wgEncode/wig/">ENCODE pilot phase whole-genome wiggle 
      data</a></li>
      <li class="collapse-section">
	<a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg18pairwise>Pairwise 
	alignments (mammals)</a>
	<div class="collapse" id="hg18pairwise"> 
	<ul>
	  <li><a href="/goldenPath/hg18/vsSelf/">Human self alignments</a></li>
	  <li><a href="/goldenPath/hg18/vsFelCat3/">Human/Cat (felCat3)</a> </li>
	  <li><a href="/goldenPath/hg18/vsPanTro2/">Human/Chimp (panTro2)</a></li>
	  <li><a href="/goldenPath/hg18/vsPanTro1/">Human/Chimp (panTro1)</a></li>
	  <li><a href="/goldenPath/hg18/vsBosTau4/">Human/Cow (bosTau4)</a></li>
	  <li><a href="/goldenPath/hg18/vsBosTau3/">Human/Cow (bosTau3)</a></li>
	  <li><a href="/goldenPath/hg18/vsBosTau2/">Human/Cow (bosTau2)</a></li>
	  <li><a href="/goldenPath/hg18/vsCanFam2/">Human/Dog (canFam2)</a></li>
	  <li><a href="/goldenPath/hg18/vsCavPor3/">Human/Guinea pig (cavPor3)</a></li>
	  <li><a href="/goldenPath/hg18/vsEquCab1/">Human/Horse (equCab1)</a></li>
	  <li><a href="/goldenPath/hg18/vsCalJac1/">Human/Marmoset (calJac1)</a></li>
	  <li><a href="/goldenPath/hg18/vsMm9/">Human/Mouse (mm9)</a></li>
	  <li><a href="/goldenPath/hg18/vsMm8/">Human/Mouse (mm8)</a></li>
	  <li><a href="/goldenPath/hg18/vsMm7/">Human/Mouse (mm7)</a></li>
	  <li><a href="/goldenPath/hg18/vsMonDom4/">Human/Opossum (monDom4)</a></li>
	  <li><a href="/goldenPath/hg18/vsPonAbe2/">Human/Orangutan (ponAbe2)</a></li>
	  <li><a href="/goldenPath/hg18/vsOrnAna1/">Human/Platypus (ornAna1)</a></li>
	  <li><a href="/goldenPath/hg18/vsRn4/">Human/Rat (rn4)</a></li>
	  <li><a href="/goldenPath/hg18/vsRheMac2/">Human/Rhesus (rheMac2)</a></li>
	  <li><a href="/goldenPath/hg18/vsEchTel1/">Human/Tenrec (echTel1)</a></li>
	  </ul>
	</div>
      </li> 
      <li class="collapse-section">
	<a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg18pairwise-vert>Pairwise 
	alignments (other vertebrates)</a>
	<div class="collapse" id="hg18pairwise-vert"> 
	<ul>
	  <li><a href="/goldenPath/hg18/vsGalGal3/">Human/Chicken (galGal3)</a></li>
	  <li><a href="/goldenPath/hg18/vsGalGal2/">Human/Chicken (galGal2)</a></li>
	  <li><a href="/goldenPath/hg18/vsFr2/">Human/Fugu (fr2)</a></li>
	  <li><a href="/goldenPath/hg18/vsFr1/">Human/Fugu (fr1)</a></li>
	  <li><a href="/goldenPath/hg18/vsBraFlo1/">Human/Lancelet (braFlo1)</a></li>
	  <li><a href="/goldenPath/hg18/vsAnoCar1/">Human/Lizard (anoCar1)</a></li>
	  <li><a href="/goldenPath/hg18/vsOryLat2/">Human/Medaka (oryLat2)</a></li>
	  <li><a href="/goldenPath/hg18/vsGasAcu1/">Human/Stickleback (gasAcu1)</a></li>
	  <li><a href="/goldenPath/hg18/vsTetNig1/">Human/Tetraodon (tetNig1)</a></li>
	  <li><a href="/goldenPath/hg18/vsXenTro2/">Human/X. tropicalis (xenTro2)</a></li>
	  <li><a href="/goldenPath/hg18/vsXenTro1/">Human/X. tropicalis (xenTro1)</a></li>
	  <li><a href="/goldenPath/hg18/vsDanRer5/">Human/Zebrafish (danRer5)</a></li>
	  <li><a href="/goldenPath/hg18/vsDanRer4/">Human/Zebrafish (danRer4)</a></li> 
	  <li><a href="/goldenPath/hg18/vsDanRer3/">Human/Zebrafish (danRer3)</a></li> 
	</ul>
	</div>
      </li>
      <li class="collapse-section">
	<a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg18mult>Multiple 
	alignments</a>
	<div class="collapse" id="hg18mult"> 
	<ul> 
	  <li>
	  <a href="/goldenPath/hg18/multiz44way/">Multiple alignments of 43 vertebrate genomes with 
	  human</a></li>
	  <li>
	  <a href="/goldenPath/hg18/phastCons44way/">Conservation scores for alignments of 43 vertebrate
	  genomes with human</a></li>
	  <li>
	  <a href="/goldenPath/hg18/phyloP44way/">Basewise conservation scores (phyloP) of 43 vertebrate
	  genomes with human</a></li>
	  <li>
	  <a href="/goldenPath/hg18/multiz44way/alignments/">FASTA alignments of 43 vertebrate genomes 
	  with human for CDS regions</a></li>
	  <li>
	  <a href="/goldenPath/hg18/multiz28way/">Multiple alignments of 27 vertebrate genomes with 
	  human</a></li>
	  <li>
	  <a href="/goldenPath/hg18/phastCons28way/">Conservation scores for alignments of 27 vertebrate
	  genomes with human</a></li>
	  <li>
	  <a href="/goldenPath/hg18/phyloP28way/">Basewise conservation scores (phyloP) of 27 vertebrate
	  genomes with human</a></li>
	  <li>
	  <a href="/goldenPath/hg18/multiz28way/alignments/">FASTA alignments of 27 vertebrate genomes 
	  with human for CDS regions</a></li>
	  <li>
	  <a href="/goldenPath/hg18/multiz17way/">Multiple alignments of 16 vertebrate genomes with 
	  human</a></li>
	  <li>
	  <a href="/goldenPath/hg18/phastCons17way/">Conservation scores for alignments of 16 vertebrate
	  genomes with human</a></li>
	  <li>
	  <a href="/goldenPath/hg18/encode/MSA/DEC-2007/">Multiple alignments of 35 vertebrate genomes 
	  with human in ENCODE regions</a></li>
	</ul>
	</div>
      </li> 
    </ul>	
  </li>
</ul>

<!-- hg17 -->
<h3>May 2004 (NCBI35/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=hg17"
target="_blank">hg17</a>)</h3> 
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg17Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg17Downloads">
    <ul> 
      <li><a href="/goldenPath/hg17/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/hg17/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg17/database/">Annotation database</a></li>
      <li><a href="/goldenPath/hg17/affyTxnPhase3/">Affymetrix transcriptome phase 3 data</a></li>
      <li><a href="/goldenPath/hg17/affyTxnPhase2/">Affymetrix transcriptome phase 2 data</a></li>
      <li><a href="/goldenPath/hg17/regPotential/5X">Regulatory potential data</a></li>
      <li><a href="/goldenPath/hg17/gc5Base/">GC percent data</a></li>
      <li><a href="/goldenPath/hg17/liftOver/">LiftOver files</a> </li>
      <li class="collapse-section">
	<a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg17pairwise>Pairwise
	alignments (mammals)</a>
	<div class="collapse" id="hg17pairwise">
	<ul>
	  <li><a href="/goldenPath/hg17/vsSelf/">Human self alignments</a></li>
	  <li><a href="/goldenPath/hg17/vsPanTro2/">Human/Chimp (panTro2)</a></li>
	  <li><a href="/goldenPath/hg17/vsPanTro1/">Human/Chimp (panTro1)</a></li>
	  <li><a href="/goldenPath/hg17/vsBosTau2/">Human/Cow (bosTau2)</a></li>
	  <li><a href="/goldenPath/hg17/vsBosTau1/">Human/Cow (bosTau1)</a></li>
	  <li><a href="/goldenPath/hg17/vsCanFam2/">Human/Dog (canFam2)</a></li>
    <!--
	  <li><a href="/goldenPath/hg17/vsCanFam1/">Human/Dog (canFam1) </a></li>
    -->
	  <li><a href="/goldenPath/hg17/vsMm7/">Human/Mouse (mm7)</a></li>
	  <li><a href="/goldenPath/hg17/vsMm6/">Human/Mouse (mm6)</a></li>
	  <li><a href="/goldenPath/hg17/vsMm5/">Human/Mouse (mm5)</a></li>
	  <li><a href="/goldenPath/hg17/vsMonDom1/">Human/Opossum (monDom1)</a></li>
	  <li><a href="/goldenPath/hg17/vsRn4/">Human/Rat (rn4)</a></li>
	  <li><a href="/goldenPath/hg17/vsRn3/">Human/Rat (rn3)</a></li>
	  <li><a href="/goldenPath/hg17/vsRheMac2/">Human/Rhesus (rheMac2)</a></li>
	  <li><a href="/goldenPath/hg17/vsEchTel1/">Human/Tenrec (echTel1)</a></li>
	  </ul>
	</div>
      </li> 
      <li class="collapse-section">
	<a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg17pairwise-vert>Pairwise
	alignments (other vertebrates)</a>
	<div class="collapse" id="hg17pairwise-vert">
	<ul>
	  <li><a href="/goldenPath/hg17/vsGalGal2/">Human/Chicken (galGal2)</a></li>
	  <li><a href="/goldenPath/hg17/vsFr1/">Human/Fugu (fr1)</a></li>
	  <li><a href="/goldenPath/hg17/vsTetNig1/">Human/Tetraodon (tetNig1)</a></li>
	  <li><a href="/goldenPath/hg17/vsDanRer3/">Human/Zebrafish (danRer3)</a> </li>
	  <li><a href="/goldenPath/hg17/vsDanRer2/">Human/Zebrafish (danRer2)</a></li>
    <!--            
	  <li><a href="/goldenPath/hg17/vsDanRer1/">Human/Zebrafish (danRer1) </a></li>
    -->
	</ul>
	</div>
      </li>
      <li class="collapse-section">
	<a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg17mult>Multiple
	alignments</a>
	<div class="collapse" id="hg17mult">
	<ul>
	  <li>
	  <a href="/goldenPath/hg17/multiz17way/">Multiple alignments of 16 vertebrate genomes with 
	  Human</a></li>
	  <li>
	  <a href="/goldenPath/hg17/phastCons17way/">Conservation scores for alignments of 16 vertebrate
	  genomes with Human</a></li>
	  <li>
	  <a href="/goldenPath/hg17/multiz8way/">Multiple alignments of 8 vertebrate genomes with 
	  Human</a></li>
	  <li>
	  <a href="/goldenPath/hg17/phastCons/mzPt1Mm5Rn3Cf1Gg2Fr1Dr1/">Conservation scores for 
	  alignments of 8 vertebrate genomes with Human</a></li>
	</ul>	
	</div>
      </li>
    </ul>	
  </li>
</ul>

<!-- hg16 -->
<h3>Jul. 2003 (NCBI34/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=hg16"
target="_blank">hg16</a>)</h3> 
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg16Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg16Downloads">
    <ul> 
      <li><a href="/goldenPath/hg16/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/hg16/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg16/database/">Annotation database</a></li>
      <li><a href="/goldenPath/hg16/regPotential/">Regulatory potential data</a></li>
      <li><a href="/goldenPath/hg16/liftOver/">LiftOver files</a></li>
      <li class="collapse-section">
	<a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg16pairwise>Pairwise 
	alignments</a>
	<div class="collapse" id="hg16pairwise">
	<ul>
	  <li><a href="/goldenPath/hg16/vsSelf/">Human self alignments</a></li>
	  <li><a href="/goldenPath/hg16/vsPanTro1/">Human/Chimp (PanTro1)</a></li>
	  <li><a href="/goldenPath/hg16/vsMm6/">Human/Mouse (mm6)</a></li>
	  <li><a href="/goldenPath/hg16/vsMm4/">Human/Mouse (mm4)</a></li>
	  <li><a href="/goldenPath/hg16/vsRn3/">Human/Rat (rn3)</a></li>
	</ul>
	</div>
      </li>
      <li class="collapse-section">
	<a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg16mult>Multiple 
	alignments</a>
	<div class="collapse" id="hg16mult">
	<ul>
	  <li>
	  <a href="/goldenPath/hg16/humorMm3Rn3/">Humor multiple alignments of 
	  Human/Mouse/Rat (mm3/rn3)</a></li>
	  <li>
	  <a href="/goldenPath/hg16/mzPt1Mm3Rn3Gg2/">Multiple alignments of 4 vertebrate genomes with 
	  Human</a></li>
	  <li>
	  <a href="/goldenPath/hg16/phastCons/hg16mm3rn3panTro1galGal2/">Conservation scores for 
	  alignments of 4 vertebrate genomes with Human</a></li>
	</ul>	
	</div>
      </li>
    </ul>	
  </li>
</ul>

<!-- hg15 -->
<h3>Apr. 10, 2003 (hg15)</h3> 
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg15Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg15Downloads">
    <ul> 
      <li><a href="/goldenPath/10april2003/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/10april2003/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/10april2003/database/">Annotation database</a></li>
      <li><a href="/goldenPath/10april2003/liftOver/">LiftOver files</a></li>
      <li class="collapse-section">
	<a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg15mult>Multiple 
	alignments</a>
	<div class="collapse" id="hg15mult">
	<ul> 
	  <li>
	  <a href="/goldenPath/10april2003/maf/">Multiple alignments of Human/Mouse/Rat (mm3/rn2)</a>
	  </li>
	</ul>	
	</div>
      </li>
    </ul>	
  </li>
</ul>

<!-- hg13 --> 
<h3>Nov. 14, 2002 (hg13)</h3> 
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg13Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg13Downloads">
    <ul> 
      <li><a href="/goldenPath/hg13/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/hg13/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg13/database/">Annotation database</a></li>
      <li><a href="/goldenPath/hg13/liftOver/">LiftOver files</a></li>
      <li><a href="/goldenPath/hg13/vsMm2/">Human/Mouse (mm2)</a></li>
    </ul>
  </li>
</ul>

<!-- hg12 --> 
<h3>Jun. 28, 2002 (hg12)</h3>
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg12Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg12Downloads">
    <ul>
      <li><a href="/goldenPath/hg12/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/hg12/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg12/database/">Annotation database</a></li>
      <li><a href="/goldenPath/hg12/liftOver/">LiftOver files</a></li>
      <li><a href="/goldenPath/hg12/vsMm2/">Human/Mouse (mm2)</a></li>
    </ul>
  </li>
</ul>

<!-- hg11 -->
<h3>Apr. 5, 2002 (hg11)</h3>
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg11Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg11Downloads">
    <ul>
      <li><a href="/goldenPath/hg11/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/hg11/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg11/database/">Annotation database</a></li>
    </ul>
  </li>
</ul>

<!-- hg10 -->
<h3>Dec. 22, 2001 (hg10)</h3>
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg10Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg10Downloads">
    <ul>
      <li><a href="/goldenPath/hg10/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/hg10/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg10/database/">Annotation database</a></li>
      <li><a href="/goldenPath/hg10/liftOver/">LiftOver files</a></li>
    </ul>
  </li>
</ul>

<!-- hg8 -->
<h3>Aug. 6, 2001 (hg8)</h3>
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg8Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg8Downloads">
    <ul>
      <li><a href="/goldenPath/hg8/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/hg8/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg8/database/">Annotation database</a></li>
    </ul>
  </li>
</ul>

<!-- hg7 --> 
<h3>Apr. 1, 2001 (hg7)</h3>
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg7Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg7Downloads">
    <ul>
      <li><a href="/goldenPath/hg7/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/hg7/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg7/database/">Annotation database</a></li>
    </ul>
  </li>
</ul>

<!-- hg6 -->
<h3>Dec. 12, 2000 (hg6)</h3>
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg6Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg6Downloads">
    <ul>
      <li><a href="/goldenPath/hg6/bigZips/">Full data set (Centromeres fixed)</a></li>
      <li><a href="/goldenPath/hg6/chromosomes/">Data set by chromosome (Centromeres fixed)</a></li>
      <li><a href="/goldenPath/hg6/database/">Annotation database</a></li>
    </ul>
  </li>
</ul>

<!-- hg5 -->
<h3>Oct. 7, 2000 (hg5)</h3>
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg5Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg5Downloads">
    <ul>
      <li><a href="/goldenPath/hg5/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/hg5/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg5/database/">Annotation database</a></li>
    </ul> 
  </li>
</ul>

<h3>Oct. 7, 2000 preliminary version</h3>
<ul>
  <li><a href="/goldenPath/hg07oct2000_oo21/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/hg07oct2000_oo21/chromosomes/">Data set by chromosome</a></li>
</ul>

<!-- hg4 -->
<h3>Sep. 5, 2000 (hg4)</h3>
<ul> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hg4Downloads>Data
    and annotations</a>
    <div class="collapse" id="hg4Downloads">
    <ul>
      <li><a href="/goldenPath/hg4/bigZips/">Full data set</a></li>
      <li><a href="/goldenPath/hg4/chromosomes/">Data set by chromosome</a></li>
      <li><a href="/goldenPath/hg4/database/">Annotation database</a></li>
    </ul>
  </li>
</ul>

<h3>Jun. 15, 2000</h3>
<ul>
  <li><a href="/goldenPath/hg15june2000/bigZips/">Full data set</a></li>
</ul>

<h3>May 24, 2000</h3>
<ul>
  <li><a href="/goldenPath/hg24may2000/bigZips/">Full data set</a></li>
</ul>

<h3>Older human data and documentation</h3>
<ul>
  <li><a href="/goldenPath/human/hgArchivedDocs/">Documents from the early instances of the Genome 
  Browser</a></li>
  <li><a href="/goldenPath/human/mapPlots/">Map plots</a></li>
  <li><a href="/goldenPath/human/chromReports/">Chromosome reports for early builds</a></li>
</ul>

<!-- Alpaca Downloads -->

<a name="alpaca"></a>
<h2>Alpaca genome</h2>
<h3>Mar. 2013 (Vicugna_pacos-2.0.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=vicPac2" 
    target="_blank">vicPac2</a>)</h3>
<ul> 
  <li><a href="/goldenPath/vicPac2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/vicPac2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/vicPac2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#vicPac2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="vicPac2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/vicPac2/vsHg38/">Alpaca/Human (hg38)</a></li> 
        <li><a href="/goldenPath/vicPac2/vsHg19/">Alpaca/Human (hg19)</a></li> 
        <li><a href="/goldenPath/vicPac2/vsMm10/">Alpaca/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2008 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=vicPac1" 
    target="_blank">vicPac1</a>)</h3>
<ul> 
  <li><a href="/goldenPath/vicPac1/bigZips/">Full data set</a> </li>
  <li><a href="/goldenPath/vicPac1/database/">Annotation database</a> </li>
  <li><a href="/goldenPath/vicPac1/liftOver/">LiftOver files</a> </li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#vicPac1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="vicPac1pairwise">
      <ul> 
        <li><a href="/goldenPath/vicPac1/vsBosTau7/">Alpaca/Cow (bosTau7)</a></li> 
        <li><a href="/goldenPath/vicPac1/vsMm10/">Alpaca/Mouse (mm10)</a></li> 
      </ul> 
    </div>
  </li>
</ul>



<a name="african_clawed_frog"></a>
<h2>African clawed frog</h2>
<h3>Aug. 2016 (Xenopus_laevis_v2)/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=xenLae2" 
    target="_blank">xenLae2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/xenLae2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/xenLae2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/xenLae2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#xenLae2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="xenLae2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/xenLae2/vsHg38/">African clawed frog/Human (hg38)</a></li> 
        <li><a href="/goldenPath/xenLae2/vsMm10/">African clawed frog/Mouse (mm10)</a></li> 
        <li><a href="/goldenPath/xenLae2/vsXenTro9/">African clawed frog/Tropical clawed frog 
        (xenTro9)</a></li> 
      </ul>
    </div>
  </li> 
</ul>


<!-- American alligator Downloads -->
<a name="american_alligator"></a>
<h2>American alligator</h2>
<h3>Aug. 2012 (allMis0.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=allMis1" 
    target="_blank">allMis1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/allMis1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/allMis1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/allMis1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#allMis1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="allMis1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/allMis1/vsHg19/">American alligator/Human (hg19)</a></li> 
      </ul>
    </div>
  </li> 
</ul>

<!-- Armadillo Downloads -->
<a name="armadillo"></a>
<h2>Armadillo genome</h2>
<h3>Dec. 2011 (Baylor/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=dasNov3" 
    target="_blank">dasNov3</a>)</h3>
<ul> 
  <li><a href="/goldenPath/dasNov3/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/dasNov3/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/dasNov3/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dasNov3pairwise>Pairwise 
    alignments</a>
    <div class="collapse" id="dasNov3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/dasNov3/vsMm10/">Armadillo/Mouse (mm10)</a></li> 
      </ul> 
    </div>
  </li>
</ul>

<!-- Atlantic cod Downloads -->
<a name="atlantic_cod"></a>
<h2>Atlantic cod genome</h2>
<h3>May 2010 (Genofisk GadMor_May2010/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=gadMor1" 
    target="_blank">gadMor1</a>)</h3>
<ul> 
  <li><a href="/goldenPath/gadMor1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/gadMor1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/gadMor1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#gadMor1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="gadMor1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/gadMor1/vsFr3/">Atlantic cod/Fugu (fr3)</a></li>
        <li><a href="/goldenPath/gadMor1/vsMm10/">Atlantic cod/Mouse (mm10)</a></li> 
      </ul>
    </div>
  </li>
</ul>

<!-- Baboon Downloads -->
<a name="baboon"></a>
<h2>Baboon genome</h2>
<h3>Apr. 2017 (Panu_3.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=papAnu4" 
    target="_blank">papAnu4</a>)</h3>
<ul>
  <li><a href="/goldenPath/papAnu4/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/papAnu4/database/">Annotation database</a></li>
  <li><a href="/goldenPath/papAnu4/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#papAnu4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="papAnu4pairwise">
      <ul>
        <li><a href="/goldenPath/papAnu4/vsHg38/">Baboon/Human (hg38)</a></li>
        <li><a href="/goldenPath/papAnu4/vsMm10/">Baboon/Mouse (mm10)</a></li>
      </ul> 
    </div>
  </li>
</ul>
<h3>Mar. 2012 (Baylor Panu_2.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=papAnu2" 
    target="_blank">papAnu2</a>)</h3>
<ul> 
  <li><a href="/goldenPath/papAnu2/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/papAnu2/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/papAnu2/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#papAnu2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="papAnu2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/papAnu2/vsHg38/">Baboon/Human (hg38)</a></li>    
        <li><a href="/goldenPath/papAnu2/vsHg19/">Baboon/Human (hg19)</a></li>
        <li><a href="/goldenPath/papAnu2/vsTarSyr2/">Baboon/Tarsier (tarSyr2)</a></li>   
      </ul>
    </div>
  </li> 
</ul>
<h3>Nov. 2008 (Baylor Pham_1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=papHam1" 
    target="_blank">papHam1</a>)</h3>
<ul> 
  <li><a href="/goldenPath/papHam1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/papHam1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/papHam1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#papHam1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="papHam1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/papHam1/vsMacFas5/">Baboon/Crab-eating macaque (macFas5)</a></li>
        <li><a href="/goldenPath/papHam1/vsGorGor3/">Baboon/Gorilla (gorGor3)</a></li> 
        <li><a href="/goldenPath/papHam1/vsCalJac3/">Baboon/Marmoset (calJac3)</a></li> 
        <li><a href="/goldenPath/papHam1/vsMm10/">Baboon/Mouse (mm10)</a></li> 
      </ul>
    </div>
  </li>
</ul>

<!-- Bison Downloads -->
<a name="bison"></a>
<h2>Bison genome</h2> 
<h3>Oct. 2014 (Bison_UMD1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=bisBis1" 
    target="_blank">bisBis1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/bisBis1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/bisBis1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/bisBis1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#bisBis1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="bisBis1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/bisBis1/vsHg38/">Bison/Human (hg38)</a></li> 
        <li><a href="/goldenPath/bisBis1/vsMm10/">Bison/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Bonobo Downloads -->
<a name="bonobo"></a>
<h2>Bonobo genome</h2>
<h3>Aug. 2015 (MPI-EVA panpan1.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=panPan2"
    target="_blank">panPan2</a>)</h3>
<ul>
  <li><a href="/goldenPath/panPan2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/panPan2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/panPan2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#panPan2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="panPan2pairwise">
      <ul>
        <li><a href="/goldenPath/panPan2/vsPanTro4/">Bonobo/Chimp (panTro4)</a></li>
        <li><a href="/goldenPath/panPan2/vsGorGor5/">Bonobo/Gorilla (gorGor5)</a></li>
        <li><a href="/goldenPath/panPan2/vsHg38/">Bonobo/Human (hg38)</a></li>
        <li><a href="/goldenPath/panPan2/vsMm10/">Bonobo/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li>
</ul>
<h3>May 2012 (Max-Planck/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=panPan1" 
    target="_blank">panPan1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/panPan1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/panPan1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/panPan1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#panPan1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="panPan1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/panPan1/vsHg38/">Bonobo/Human (hg38)</a></li>
        <li><a href="/goldenPath/panPan1/vsTarSyr2/">Bonobo/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Brown kiwi Downloads -->
<a name="brown_kiwi"></a>
<h2>Brown kiwi genome</h2> 
<h3>Jun. 2015 (MPI-EVA AptMant0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=aptMan1" 
    target="_blank">aptMan1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/aptMan1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/aptMan1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/aptMan1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#aptMan1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="aptMan1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/aptMan1/vsHg38/">Brown kiwi/Human (hg38)</a></li> 
      </ul>
    </div>
  </li> 
</ul>

<!-- Budgerigar Downloads -->
<a name="budgerigar"></a>
<h2>Budgerigar genome</h2>
<h3>Sep. 2011 (WUSTL v6.3/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=melUnd1" 
    target="_blank">melUnd1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/melUnd1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/melUnd1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/melUnd1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#melUnd1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="melUnd1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/melUnd1/vsGeoFor1/">Budgerigar/Medium ground finch 
        (geoFor1)</a></li>
        <li><a href="/goldenPath/melUnd1/vsMm10/">Budgerigar/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li>
</ul>

<!-- Bushbaby Downloads -->
<a name="bushbaby"></a>
<h2>Bushbaby genome</h2>
<h3>Mar. 2011 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=otoGar3" 
    target="_blank">otoGar3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/otoGar3/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/otoGar3/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/otoGar3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#otoGar3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="otoGar3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/otoGar3/vsMm10/">Bushbaby/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/otoGar3/vsTarSyr2/">Bushbaby/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li>
</ul>

<!-- Cat Downloads -->
<a name="cat"></a>
<h2>Cat genome</h2>

<h3>Nov. 2017 (Felis_catus_9.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=felCat9" 
    target="_blank">felCat9</a>)</h3>
<ul>
  <li><a href="/goldenPath/felCat9/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/felCat9/database/">Annotation database</a></li>
  <li><a href="/goldenPath/felCat9/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#felCat9pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="felCat9pairwise">
      <ul>
        <li><a href="/goldenPath/felCat9/vsHg38/">Cat/Human (hg38)</a></li>
        <li><a href="/goldenPath/felCat9/vsMm10/">Cat/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li>
</ul>



<h3>Nov. 2014 (ICGSC Felis_catus_8.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=felCat8" 
    target="_blank">felCat8</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/felCat8/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/felCat8/database/">Annotation database</a></li>
  <li><a href="/goldenPath/felCat8/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#felCat8pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="felCat8pairwise"> 
      <ul>
        <li><a href="/goldenPath/felCat8/vsHg38/">Cat/Human (hg38)</a></li>
        <li><a href="/goldenPath/felCat8/vsCanFam3/">Cat/Dog (canFam3)</a></li>
        <li><a href="/goldenPath/felCat8/vsRn6/">Cat/Rat (rn6)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Sep. 2011 (ICGSC Felis_catus 6.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=felCat5" 
    target="_blank">felCat5</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/felCat5/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/felCat5/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/felCat5/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#felCat5pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="felCat5pairwise"> 
      <ul> 
        <li><a href="/goldenPath/felCat5/vsHg19/">Cat/Human (hg19)</a></li> 
        <li><a href="/goldenPath/felCat5/vsMm10/">Cat/Mouse (mm10)</a></li> 
      </ul>
    </div>
  </li>
</ul>
<h3>Dec. 2008 (NHGRI/GTB V17/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=felCat4" 
    target="_blank">felCat4</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/felCat4/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/felCat4/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/felCat4/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#felCat4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="felCat4pairwise"> 
      <ul> 
        <li><a href="/goldenPath/felCat4/vsHg19/">Cat/Human (hg19)</a></li> 
        <li><a href="/goldenPath/felCat4/vsCanFam2/">Cat/Dog (canFam2)</a></li> 
        <li><a href="/goldenPath/felCat4/vsMm9/">Cat/Mouse (mm9)</a></li> 
        <li><a href="/goldenPath/felCat4/vsMonDom5/">Cat/Opossum (monDom5)</a></li> 
        <li><a href="/goldenPath/felCat4/vsAilMel1/">Cat/Panda (ailMel1)</a></li> 
      </ul>
    </div>
  </li>
</ul>
<h3>Mar. 2006 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=felCat3" 
    target="_blank">felCat3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/felCat3/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/felCat3/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/felCat3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#felCat3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="felCat3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/felCat3/vsHg19/">Cat/Human (hg19)</a></li> 
        <li><a href="/goldenPath/felCat3/vsHg18/">Cat/Human (hg18)</a></li> 
        <li><a href="/goldenPath/felCat3/vsCanFam2/">Cat/Dog (canFam2)</a></li> 
        <li><a href="/goldenPath/felCat3/vsMm8/">Cat/Mouse (mm8)</a></li> 
      </ul> 
    </div>
  </li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#felCat3mult>Multiple
    alignments</a>
    <div class="collapse" id="felCat3mult"> 
      <ul> 
        <li><a href="/goldenPath/felCat3/multiz4way/">Multiple alignments of 3 vertebrate genomes 
        with Cat</a></li> 
        <li><a href="/goldenPath/felCat3/phastCons4way/">Conservation scores for alignments of 3 
        vertebrate genomes with Cat</a></li> 
      </ul>
    </div>
  </li>
</ul>

<!-- Chicken Downloads -->
<a name="chicken"></a>
<h2>Chicken genome</h2>

<h3>Mar. 2018 (Gallus_gallus-6.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=galGal6" 
    target="_blank">galGal6</a>)</h3>
<ul>
  <li><a href="/goldenPath/galGal6/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/galGal6/database/">Annotation database</a></li>
  <li><a href="/goldenPath/galGal6/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#galGal6pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="galGal6pairwise">
      <ul>
        <li><a href="/goldenPath/galGal6/vsHg38/">Chicken/Human (hg38)</a></li>
        <li><a href="/goldenPath/galGal6/vsMm10/">Chicken/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li>
 <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#galGal6mult>Multiple
    alignments</a>
    <div class="collapse" id="galGal6mult">
      <ul>
        <li><a href="/goldenPath/galGal6/multiz77way/">Multiple alignments of 77 vertebrate genomes with Chicken</a></li>
        <li><a href="/goldenPath/galGal6/phastCons77way/">Conservation scores for alignments of 77 vertebrate genomes with Chicken</a></li>
	<li><a href="/goldenPath/galGal6/phyloP77way/">Basewise conservation scores (phyloP) of 77 vertebrate genomes with Chicken</a></li>
      </ul>
    </div>
 </li>
</ul>

<h3>Dec. 2015 (Gallus_gallus-5.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=galGal5" 
    target="_blank">galGal5</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/galGal5/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/galGal5/database/">Annotation database</a></li>
  <li><a href="/goldenPath/galGal5/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#galGal5pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="galGal5pairwise"> 
      <UL>
        <li><a href="/goldenPath/galGal5/vsHg38/">Chicken/Human (hg38)</a></li>
        <li><a href="/goldenPath/galGal5/vsAquChr2/">Chicken/Golden eagle (aquChr2)</a></li>
        <li><a href="/goldenPath/galGal5/vsMm10/">Chicken/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/galGal5/vsRn6/">Chicken/Rat (rn6)</a></li>
        <li><a href="/goldenPath/galGal5/vsDanRer10/">Chicken/Zebrafish (danRer10)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Nov. 2011 (ICGSC Gallus_gallus-4.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=galGal4"
    target="_blank">galGal4</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/galGal4/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/galGal4/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/galGal4/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#galGal4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="galGal4pairwise"> 
      <ul> 
        <li><a href="/goldenPath/galGal4/vsHg19/">Chicken/Human (hg19)</a></li>
        <li><a href="/goldenPath/galGal4/vsPetMar2/">Chicken/Lamprey (petMar2)</a></li>
        <li><a href="/goldenPath/galGal4/vsGeoFor1/">Chicken/Medium ground finch (geoFor1)</a></li> 
        <li><a href="/goldenPath/galGal4/vsMm10/">Chicken/Mouse (mm10)</a></li> 
        <li><a href="/goldenPath/galGal4/vsTaeGut2/">Chicken/Zebra finch (taeGut2)</a></li>
      </ul>
    </div>
  </li>
</ul>
<h3>May 2006 ((WUGSC 2.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=galGal3" 
    target="_blank">galGal3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/galGal3/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/galGal3/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/galGal3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/galGal3/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#galGal3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="galGal3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/galGal3/vsHg19/">Chicken/Human (hg19)</a></li> 
        <li><a href="/goldenPath/galGal3/vsHg18/">Chicken/Human (hg18)</a></li> 
        <li><a href="/goldenPath/galGal3/vsPanTro3/">Chicken/Chimp (panTro3)</a></li> 
        <li><a href="/goldenPath/galGal3/vsFr2/">Chicken/Fugu (fr2)</a></li>
        <li><a href="/goldenPath/galGal3/vsCavPor3/">Chicken/Guinea pig (cavPor3)</a></li> 
        <li><a href="/goldenPath/galGal3/vsEquCab2/">Chicken/Horse (equCab2)</a></li> 
        <li><a href="/goldenPath/galGal3/vsEquCab1/">Chicken/Horse (equCab1)</a></li> 
        <li><a href="/goldenPath/galGal3/vsPetMar1/">Chicken/Lamprey (petMar1)</a></li>
        <li><a href="/goldenPath/galGal3/vsBraFlo1/">Chicken/Lancelet (braFlo1)</a></li>
        <li><a href="/goldenPath/galGal3/vsAnoCar2/">Chicken/Lizard (anoCar2)</a></li>
        <li><a href="/goldenPath/galGal3/vsAnoCar1/">Chicken/Lizard (anoCar1)</a></li>
        <li><a href="/goldenPath/galGal3/vsMm9/">Chicken/Mouse (mm9)</a></li> 
        <li><a href="/goldenPath/galGal3/vsMm8/">Chicken/Mouse (mm8)</a></li> 
        <li><a href="/goldenPath/galGal3/vsMonDom5/">Chicken/Opossum (monDom5)</a></li> 
        <li><a href="/goldenPath/galGal3/vsMonDom4/">Chicken/Opossum (monDom4)</a></li> 
        <li><a href="/goldenPath/galGal3/vsPonAbe2/">Chicken/Orangutan (ponAbe2)</a></li> 
        <li><a href="/goldenPath/galGal3/vsOrnAna1/">Chicken/Platypus (ornAna1)</a></li>
        <li><a href="/goldenPath/galGal3/vsRn4/">Chicken/Rat (rn4)</a></li> 
<!--
        <li><a href="/goldenPath/galGal3/vsTetNig1/">Chicken/Tetraodon (tetNig1) </a></li>
-->
        <li><a href="/goldenPath/galGal3/vsMelGal1/">Chicken/Turkey (melGal1)</a></li>
        <li><a href="/goldenPath/galGal3/vsXenTro3/">Chicken/X. tropicalis 
        (xenTro3)</a></li>
        <li><a href="/goldenPath/galGal3/vsXenTro2/">Chicken/X. tropicalis 
        (xenTro2)</a></li>
        <li><a href="/goldenPath/galGal3/vsDanRer4/">Chicken/Zebrafish (danRer4)</a></li>
      </ul>
    </div>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#galGal3mult>Multiple
    alignments</a>
    <div class="collapse" id="galGal3mult"> 
      <ul> 
        <li><a href="/goldenPath/galGal3/multiz7way/">Multiple alignments of 6 vertebrate genomes 
        with chicken</a></li>
        <li><a href="/goldenPath/galGal3/phastCons7way/">Conservation scores for alignments of 6 
        vertebrate genomes with chicken</a></li>
      </ul>
    </div>
  </li>
</ul>
<h3>Feb. 2004 (WUGSC 1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=galGal2" 
    target="_blank">galGal2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/galGal2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/galGal2/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/galGal2/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/galGal2/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#galGal2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="galGal2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/galGal2/vsHg18/">Chicken/Human (hg18)</a></li> 
        <li><a href="/goldenPath/galGal2/vsHg17/">Chicken/Human (hg17)</a></li> 
        <li><a href="/goldenPath/galGal2/vsHg16/">Chicken/Human (hg16)</a></li> 
        <li><a href="/goldenPath/galGal2/vsMm8/">Chicken/Mouse (mm8)</a></li> 
        <li><a href="/goldenPath/galGal2/vsMm7/">Chicken/Mouse (mm7)</a></li> 
        <li><a href="/goldenPath/galGal2/vsMm6/">Chicken/Mouse (mm6)</a></li> 
        <li><a href="/goldenPath/galGal2/vsMonDom1/">Chicken/Opossum (monDom1)</a></li> 
        <li><a href="/goldenPath/galGal2/vsRn4/">Chicken/Rat (rn4)</a></li> 
        <li><a href="/goldenPath/galGal2/vsTetNig1/">Chicken/Tetraodon (tetNig1)</a></li>
        <li><a href="/goldenPath/galGal2/vsXenTro1/">Chicken/X. tropicalis 
        (xenTro1)</a></li> 
        <li><a href="/goldenPath/galGal2/vsDanRer2/">Chicken/Zebrafish (danRer2)</a></li> 
      </ul>
    </div>
  </li> 
  <li><a href="/goldenPath/galGal2/multiz7way/">Multiple alignments of 6 vertebrate genomes with 
  chicken</a></li> 
</ul>

<!-- Chimp Downloads -->
<a name="chimp"></a>
<h2>Chimpanzee genome</h2>
<h3>Jan. 2018 (Clint_PTRv2/<a href="../cgi-bin/hgGateway?db=panTro6" 
    target="_blank">panTro6</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/panTro6/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/panTro6/database/">Annotation database</a></li>
  <li><a href="/goldenPath/panTro6/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#panTro6pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="panTro6pairwise"> 
      <ul> 
        <li><a href="/goldenPath/panTro6/vsHg38/">Chimp/Human (hg38)</a></li>
        <li><a href="/goldenPath/panTro6/vsHg19/">Chimp/Human (hg19)</a></li>
        <li><a href="/goldenPath/panTro6/vsMm10/">Chimp/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<h3>May. 2016 (Pan_tro 3.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=panTro5" 
    target="_blank">panTro5</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/panTro5/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/panTro5/database/">Annotation database</a></li>
  <li><a href="/goldenPath/panTro5/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#panTro5pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="panTro5pairwise"> 
      <ul> 
        <li><a href="/goldenPath/panTro5/vsHg38/">Chimp/Human (hg38)</a></li>
        <li><a href="/goldenPath/panTro5/vsHg19/">Chimp/Human (hg19)</a></li>
        <li><a href="/goldenPath/panTro5/vsGorGor5/">Chimp/Gorilla (gorGor5)</a></li>
        <li><a href="/goldenPath/panTro5/vsCalJac3/">Chimp/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/panTro5/vsMm10/">Chimp/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/panTro5/vsRn6/">Chimp/Rat (rn6)</a></li>
        <li><a href="/goldenPath/panTro5/vsRheMac8/">Chimp/Rhesus (rheMac8)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Feb. 2011 (SAC 2.1.4/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=panTro4" 
    target="_blank">panTro4</a></h3>
<ul> 
  <li><a href="/goldenPath/panTro4/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/panTro4/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/panTro4/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#panTro4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="panTro4pairwise"> 
      <ul> 
        <li><a href="/goldenPath/panTro4/vsHg19/">Chimp/Human (hg19)</a></li> 
        <li><a href="/goldenPath/panTro4/vsGorGor5/">Chimp/Gorilla (gorGor5)</a></li> 
        <li><a href="/goldenPath/panTro4/vsMm10/">Chimp/Mouse (mm10)</a></li> 
        <li><a href="/goldenPath/panTro4/vsRn5/">Chimp/Rat (rn5)</a></li>
        <li><a href="/goldenPath/panTro4/vsRheMac3/">Chimp/Rhesus (rheMac3)</a></li> 
        <li><a href="/goldenPath/panTro4/vsTarSyr2/">Chimp/Tarsier (tarSyr2)</a></li> 
      </ul>
    </div>
  </li> 
</ul>
<h3>Oct. 2010 (CGSC 2.1.3/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=panTro3" 
    target="_blank">panTro3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/panTro3/bigZips/">Full data set</a></li> 
  <!-- 
  <li><a href="/goldenPath/panTro3/chromosomes/">Data set by chromosome</a></li>        
  -->
  <li><a href="/goldenPath/panTro3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/panTro3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#panTro3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="panTro3pairwise"> 
      <ul>
        <li><a href="/goldenPath/panTro3/vsHg19/">Chimp/Human (hg19)</a></li>
        <li><a href="/goldenPath/panTro3/vsGorGor3/">Chimp/Gorilla (gorGor3)</a></li>
        <li><a href="/goldenPath/panTro3/vsPonAbe2/">Chimp/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/panTro3/vsRheMac2/">Chimp/Rhesus (rheMac2)</a></li>
        <li><a href="/goldenPath/panTro3/vsCalJac3/">Chimp/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/panTro3/vsMm9/">Chimp/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/panTro3/vsCanFam2/">Chimp/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/panTro3/vsEquCab2/">Chimp/Horse (equCab2)</a></li>
        <li><a href="/goldenPath/panTro3/vsMonDom5/">Chimp/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/panTro3/vsGalGal3/">Chimp/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/panTro3/vsDanRer7/">Chimp/Zebrafish (danRer7)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Mar. 2006 (CGSC 2.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=panTro2  
    "target="_blank">panTro2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/panTro2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/panTro2/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/panTro2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/panTro2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#panTro2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="panTro2pairwise"> 
      <ul>
        <li><a href="/goldenPath/panTro2/vsHg19/">Chimp/Human (hg19)</a></li>
        <li><a href="/goldenPath/panTro2/vsHg18/">Chimp/Human (hg18)</a></li>
        <li><a href="/goldenPath/panTro2/vsHg17/">Chimp/Human (hg17)</a></li>
        <li><a href="/goldenPath/panTro2/vsGalGal2/">Chimp/Chicken (galGal2)</a></li>
        <li><a href="/goldenPath/panTro2/vsCanFam2/">Chimp/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/panTro2/vsEquCab1/">Chimp/Horse (equCab1)</a></li>
        <li><a href="/goldenPath/panTro2/vsCalJac3/">Chimp/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/panTro2/vsCalJac1/">Chimp/Marmoset (calJac1)</a></li>
        <li><a href="/goldenPath/panTro2/vsMm8/">Chimp/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/panTro2/vsMm9/">Chimp/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/panTro2/vsMonDom5/">Chimp/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/panTro2/vsMonDom4/">Chimp/Opossum (monDom4)</a></li>
        <li><a href="/goldenPath/panTro2/vsPonAbe2/">Chimp/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/panTro2/vsRn4/">Chimp/Rat (rn4)</a></li>
        <li><a href="/goldenPath/panTro2/vsRheMac2/">Chimp/Rhesus (rheMac2)</a></li>
        <li><a href="/goldenPath/panTro2/vsDanRer4/">Chimp/Zebrafish (danRer4)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Nov. 2003 (CGSC 1.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=panTro1" 
    target="_blank">panTro1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/panTro1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/panTro1/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/panTro1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/panTro1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#panTro1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="panTro1pairwise"> 
      <ul>
        <li><a href="/goldenPath/panTro1/vsHg18/">Chimp/Human (hg18)</a></li>
        <li><a href="/goldenPath/panTro1/vsHg17/">Chimp/Human (hg17)</a></li>
        <li><a href="/goldenPath/panTro1/vsHg16/">Chimp/Human (hg16)</a></li>
        <li><a href="/goldenPath/panTro1/vsMm8/">Chimp/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/panTro1/vsMm7/">Chimp/Mouse (mm7)</a></li>
        <li><a href="/goldenPath/panTro1/vsMm6/">Chimp/Mouse (mm6)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Chinese hamster Downloads -->
<a name="chinese_hamster"></a>
<h2>Chinese hamster genome</h2>
<h3>Jun. 2017 (CHOK1S_HZDv1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=criGriChoV2" 
    target="_blank">criGriChoV2</a>)</h3>
<ul>
  <li><a href="/goldenPath/criGriChoV2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/criGriChoV2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/criGriChoV2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#criGriChoV2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="criGriChoV2pairwise">
      <ul>
        <li><a href="/goldenPath/criGriChoV2/vsHg38/">CHO K1 cell line (criGriChoV2)/Human (hg38)</a></li>
        <li><a href="/goldenPath/criGriChoV2/vsMm10/">CHO K1 cell line (criGriChoV2)/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li>
</ul>



<h3>Jul. 2013 (C_griseus_v1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=criGri1" 
    target="_blank">criGri1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/criGri1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/criGri1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/criGri1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#criGri1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="criGri1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/criGri1/vsHg19/">Chinese hamster/Human (hg19)</a></li>
        <li><a href="/goldenPath/criGri1/vsCriGriChoV1/">Chinese hamster/CHO K1 cell line
        (criGriChoV1)</a></li> 
      </ul>
    </div>
  </li> 
</ul>

<h3>Aug. 2011 (CriGri_1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=criGriChoV1" 
    target="_blank">criGriChoV1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/criGriChoV1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/criGriChoV1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/criGriChoV1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#criGriChoV1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="criGriChoV1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/criGriChoV1/vsCriGri1/">CHO K1 cell line/Chinese hamster</a></li> 
        <li><a href="/goldenPath/criGriChoV1/vsHg38/">CHO K1 cell line/Human (hg38)</a></li>
        <li><a href="/goldenPath/criGriChoV1/vsMm10/">CHO K1 cell line/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Chinese pangolin Downloads -->
<a name="chinese_pangolin"></a>
<h2>Chinese pangolin genome</h2> 
<h3>Aug. 2014 (M_pentadactyla-1.1.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=manPen1" 
    target="_blank">manPen1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/manPen1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/manPen1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/manPen1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#manPen1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="manPen1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/manPen1/vsHg38/">Chinese pangolin/Human (hg38)</a></li> 
        <li><a href="/goldenPath/manPen1/vsMm10/">Chinese pangolin/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>


<!-- Coelacanth Downloads -->
<a name="coelacanth"></a>
<h2>Coelacanth genome</h2>
<h3>Aug. 2011 (LatCha1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=latCha1" 
    target="_blank">latCha1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/latCha1/bigZips/">Full data set</a>
  <li><a href="/goldenPath/latCha1/database/">Annotation database</a>
  <li><a href="/goldenPath/latCha1/liftOver/">LiftOver files</a>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#latCha1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="latCha1pairwise"> 
      <ul>
        <li><a href="/goldenPath/latCha1/vsFr3/">Coelacanth/Fugu (fr3)</a>
        <li><a href="/goldenPath/latCha1/vsMm10/">Coelacanth/Mouse (mm10)</a>
      </ul>
    </div>
  </li> 
</ul>

<!-- Cow Downloads -->
<a name="cow"></a>
<h2>Cow genome</h2>
<h3>Apr. 2018 (ARS-UCD1.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=bosTau9" 
    target="_blank">bosTau9</a>)</h3>
<ul>
  <li><a href="/goldenPath/bosTau9/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/bosTau9/database/">Annotation database</a></li>
  <li><a href="/goldenPath/bosTau9/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#bosTau9pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="bosTau9pairwise">
      <ul>
        <li><a href="/goldenPath/bosTau9/vsHg38/">Cow/Human (hg38)</a></li>
        <li><a href="/goldenPath/bosTau9/vsMm10/">Cow/Mouse (mm10)</a></li>
      </ul> 
    </div>
  </li>
</ul>
<h3>Jun. 2014 (UMD_3.1.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=bosTau8" 
    target="_blank">bosTau8</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/bosTau8/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/bosTau8/database/">Annotation database</a></li>
  <li><a href="/goldenPath/bosTau8/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#bosTau8pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="bosTau8pairwise"> 
      <ul>
        <li><a href="/goldenPath/bosTau8/vsHg38/">Cow/Human (hg38)</a></li>
        <li><a href="/goldenPath/bosTau8/vsMm10/">Cow/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/bosTau8/vsRn6/">Cow/Rat (rn6)</a></li>
        <li><a href="/goldenPath/bosTau8/vsEquCab2/">Cow/Horse (equCab2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Oct. 2011 (Btau_4.6.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=bosTau7" 
    target="_blank">bosTau7</a>)</h3> 
<ul>
  <li><a href="/goldenPath/bosTau7/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/bosTau7/database/">Annotation database</a></li>
  <li><a href="/goldenPath/bosTau7/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#bosTau7pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="bosTau7pairwise"> 
      <ul>
      <li><a href="/goldenPath/bosTau7/vsHg19/">Cow/Human (hg19)</a></li>
      <li><a href="/goldenPath/bosTau7/vsVicPac1/">Cow/Alpaca (vicPac1)</a></li>
      <li><a href="/goldenPath/bosTau7/vsCanFam2/">Cow/Dog (canFam2)</a></li>
      <li><a href="/goldenPath/bosTau7/vsTurTru2/">Cow/Dolphin (turTru2)</a></li>
      <li><a href="/goldenPath/bosTau7/vsMm10/">Cow/Mouse (mm10)</a></li>
      <li><a href="/goldenPath/bosTau7/vsMm9/">Cow/Mouse (mm9)</a></li>
      <li><a href="/goldenPath/bosTau7/vsMonDom5/">Cow/Opossum (monDom5)</a></li>
      <li><a href="/goldenPath/bosTau7/vsSusScr2/">Cow/Pig (susScr2)</a></li>
      <li><a href="/goldenPath/bosTau7/vsOrnAna1/">Cow/Platypus (ornAna1)</a></li>
      <li><a href="/goldenPath/bosTau7/vsRn4/">Cow/Rat (rn4)</a></li>
      <li><a href="/goldenPath/bosTau7/vsCerSim1/">Cow/White rhinoceros (cerSim1)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Nov. 2009 (Bos_taurus_UMD_3.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=bosTau6" 
    target="_blank">bosTau6</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/bosTau6/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/bosTau6/database/">Annotation database</a></li>
  <li><a href="/goldenPath/bosTau6/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#bosTau6pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="bosTau6pairwise"> 
    <ul> 
      <li><a href="/goldenPath/bosTau6/vsHg19/">Cow/Human (hg19)</a></li>
      <li><a href="/goldenPath/bosTau6/vsCanFam2/">Cow/Dog (canFam2)</a></li>
      <li><a href="/goldenPath/bosTau6/vsMm9/">Cow/Mouse (mm9)</a></li>
      <li><a href="/goldenPath/bosTau6/vsMonDom5/">Cow/Opossum (monDom5)</a></li>
      <li><a href="/goldenPath/bosTau6/vsSusScr2/">Cow/Pig (susScr2)</a></li>
      <li><a href="/goldenPath/bosTau6/vsOrnAna1/">Cow/Platypus (ornAna1)</a></li>
      <li><a href="/goldenPath/bosTau6/vsRn4/">Cow/Rat (rn4)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Apr. 2009 (bosTau5)</h3> 
<ul> 
  <li><a href="/goldenPath/bosTau5/liftOver/">LiftOver files</a></li>
</ul>
<h3>Oct. 2007 (Baylor 4.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=bosTau4"
target="_blank">bosTau4</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/bosTau4/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/bosTau4/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/bosTau4/database/">Annotation database</a></li>
  <li><a href="/goldenPath/bosTau4/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#bosTau4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="bosTau4pairwise"> 
      <ul>
        <li><a href="/goldenPath/bosTau4/vsHg19/">Cow/Human (hg19)</a></li>
        <li><a href="/goldenPath/bosTau4/vsHg18/">Cow/Human (hg18)</a></li>
        <li><a href="/goldenPath/bosTau4/vsCanFam2/">Cow/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/bosTau4/vsMm9/">Cow/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/bosTau4/vsSusScr2/">Cow/Pig (susScr2)</a></li>
        <li><a href="/goldenPath/bosTau4/vsOrnAna1/">Cow/Platypus (ornAna1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#bosTau4mult>Multiple
    alignments</a>
    <div class="collapse" id="bosTau4mult"> 
      <ul> 
        <li><a href="/goldenPath/bosTau4/multiz5way/">Multiple alignments of 4 vertebrate genomes 
        with Cow</a></li>
        <li><a href="/goldenPath/bosTau4/phastCons5way/">Conservation scores for alignments of 4 
        vertebrate genomes with Cow</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Aug. 2009 (bosTauMd3)</h3> 
<ul> 
  <li><a href="/goldenPath/bosTauMd3/liftOver/">LiftOver files</a></li>
</ul>
<h3>Aug. 2006 (Baylor 3.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=bosTau3"
target="_blank">bosTau3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/bosTau3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/bosTau3/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/bosTau3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/bosTau3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#bosTau3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="bosTau3pairwise"> 
      <ul>
        <li><a href="/goldenPath/bosTau3/vsHg18/">Cow/Human (hg18)</a></li>
        <li><a href="/goldenPath/bosTau3/vsMm8/">Cow/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/bosTau3/vsMm9/">Cow/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/bosTau3/vsRn4/">Cow/Rat (rn4)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Mar. 2005 (Baylor 2.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=bosTau2"
target="_blank">bosTau2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/bosTau2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/bosTau2/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/bosTau2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/bosTau2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#bosTau2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="bosTau2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/bosTau2/vsHg18/">Cow/Human (hg18)</a></li>
        <li><a href="/goldenPath/bosTau2/vsMm8/">Cow/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/bosTau2/vsMm7/">Cow/Mouse (mm7)</a></li>
        <li><a href="/goldenPath/bosTau2/vsRn4/">Cow/Rat (rn4)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>May 2004 (bosTau1)</h3> 
<ul> 
  <li><a href="/goldenPath/bosTau1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/bosTau1/database/">Annotation database</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#bosTau1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="bosTau1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/bosTau1/vsHg17/">Cow/Human (hg17)</a></li>
        <li><a href="/goldenPath/bosTau1/vsMm6/">Cow/Mouse (mm6)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Crab-eating macaque Downloads -->
<a name="crab-eating_macaque"></a>
<h2>Crab-eating macaque genome</h2>
<h3>Jun. 2013 (Macaca_fascicularis_5.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=macFas5"
    target="_blank">macFas5</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/macFas5/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/macFas5/database/">Annotation database</a></li>
  <li><a href="/goldenPath/macFas5/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#macFas5pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="macFas5pairwise"> 
      <ul> 
        <li><a href="/goldenPath/macFas5/vsHg38/">Crab-eating macaque/Human (hg38)</a></li>
        <li><a href="/goldenPath/macFas5/vsHg19/">Crab-eating macaque/Human (hg19)</a></li>
        <li><a href="/goldenPath/macFas5/vsPapHam1/">Crab-eating macaque/Baboon (papHam1)</a></li>
        <li><a href="/goldenPath/macFas5/vsTarSyr2/">Crab-eating macaque/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Dog Downloads -->
<a name="dog"></a>
<h2>Dog genome</h2>
<h3>Sep. 2011 (Broad CanFam3.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=canFam3" 
    target="_blank">canFam3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/canFam3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/canFam3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/canFam3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#canFam3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="canFam3pairwise"> 
      <ul>
        <li><a href="/goldenPath/canFam3/vsHg19/">Dog/Human (hg19)</a></li>
        <li><a href="/goldenPath/canFam3/vsMm10/">Dog/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>May 2005 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=canFam2" 
    target="_blank">canFam2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/canFam2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/canFam2/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/canFam2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/canFam2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#canFam2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="canFam2pairwise"> 
      <ul>
        <li><a href="/goldenPath/canFam2/vsSelf/">Dog self alignments</a></li>
        <li><a href="/goldenPath/canFam2/vsHg19/">Dog/Human (hg19)</a></li>
        <li><a href="/goldenPath/canFam2/vsHg18/">Dog/Human (hg18)</a></li>
        <li><a href="/goldenPath/canFam2/vsHg17/">Dog/Human (hg17)</a></li>
        <li><a href="/goldenPath/canFam2/vsFelCat3/">Dog/Cat (felCat3)</a></li>
        <li><a href="/goldenPath/canFam2/vsPanTro3/">Dog/Chimp (panTro3)</a></li>
        <li><a href="/goldenPath/canFam2/vsBosTau7/">Dog/Cow (bosTau7)</a></li>
        <li><a href="/goldenPath/canFam2/vsBosTau6/">Dog/Cow (bosTau6)</a></li>
        <li><a href="/goldenPath/canFam2/vsBosTau4/">Dog/Cow (bosTau4)</a></li>
        <li><a href="/goldenPath/canFam2/vsEquCab2/">Dog/Horse (equCab2)</a></li>
        <li><a href="/goldenPath/canFam2/vsEquCab1/">Dog/Horse (equCab1)</a></li>
        <li><a href="/goldenPath/canFam2/vsCalJac3/">Dog/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/canFam2/vsCalJac1/">Dog/Marmoset (calJac1)</a></li>
        <li><a href="/goldenPath/canFam2/vsMm9/">Dog/Mouse (mm9)</a></li>                
        <li><a href="/goldenPath/canFam2/vsMm8/">Dog/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/canFam2/vsMm7/">Dog/Mouse (mm7)</a></li>
        <li><a href="/goldenPath/canFam2/vsMm6/">Dog/Mouse (mm6)</a></li>
        <li><a href="/goldenPath/canFam2/vsMonDom5/">Dog/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/canFam2/vsAilMel1/">Dog/Panda (ailMel1)</a></li>
        <li><a href="/goldenPath/canFam2/vsRn4/">Dog/Rat (rn4)</a></li>
        <li><a href="/goldenPath/canFam2/vsRn3/">Dog/Rat (rn3)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#canFam2mult>Multiple
    alignments</a>
    <div class="collapse" id="canFam2mult"> 
      <ul>
        <li><a href="/goldenPath/canFam2/multiz4way/">Multiple alignments of 3 vertebrate genomes 
        with Dog</a></li>
        <li><a href="/goldenPath/canFam2/phastCons4way/">Conservation scores for alignments of 3 
        vertebrate genomes with Dog</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2004 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=canFam1" 
    target="_blank">canFam1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/canFam1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/canFam1/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/canFam1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/canFam1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#canFam1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="canFam1pairwise"> 
      <ul>
        <li><a href="/goldenPath/canFam1/vsHg17/">Dog/Human (hg17)</a></li>
        <li><a href="/goldenPath/canFam1/vsMm6/">Dog/Mouse (mm6)</a></li>
      </ul>
    </div>
  </li> 
  <li><a href="/goldenPath/canFam1/multizHg17Mm5/">Multiple alignments of Dog/Human/Mouse 
  (hg17/mm5)</a></li> 
</ul>

<!-- Dolphin Downloads -->
<a name="dolphin"></a>
<h2>Dolphin genome</h2>
<h3>Oct. 2011 (Baylor Ttru_1.4/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=turTru2" 
    target="_blank">turTru2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/turTru2/bigZips/">Full data set</a>
  <li><a href="/goldenPath/turTru2/database/">Annotation database</a>
  <li><a href="/goldenPath/turTru2/liftOver/">LiftOver files</a>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#turTru2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="turTru2pairwise"> 
      <ul>
        <li><a href="/goldenPath/turTru2/vsHg38/">Dolphin/Human (hg38)</a>
        <li><a href="/goldenPath/turTru2/vsBosTau7/">Dolphin/Cow (bosTau7)</a>
        <li><a href="/goldenPath/turTru2/vsMm10/">Dolphin/Mouse (mm10)</a>
      </ul>
    </div>
  </li> 
</ul>

<!-- Elephant Downloads -->
<a name="elephant"></a>
<h2>Elephant genome</h2>
<h3>Jul. 2009 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=loxAfr3" 
    target="_blank">loxAfr3</a>)</h3> 
<ul>
  <li><a href="/goldenPath/loxAfr3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/loxAfr3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/loxAfr3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#loxAfr3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="loxAfr3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/loxAfr3/vsHg19/">Elephant/Human (hg19)</a></li> 
        <li><a href="/goldenPath/loxAfr3/vsMm9/">Elephant/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/loxAfr3/vsMonDom5/">Elephant/Opossum (monDom5)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>May 2005 (loxAfr1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=loxAfr1" 
    target="_blank">loxAfr1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/loxAfr1/bigZips/">Full data set</a>
</ul>	

<!-- Elephant shark Downloads -->
<a name="elephant_shark"></a>
<h2>Elephant shark genome</h2>
<h3>Dec. 2013 (Callorhinchus_milii-6.1.3/<a 
    href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=calMil1" target="_blank">calMil1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/calMil1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/calMil1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/calMil1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#calMil1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="calMil1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/calMil1/vsRn6/">Elephant shark/Rat (rn6)</a></li> 
      </ul>
    </div>
  </li> 
</ul>

<!-- Ferret Downloads -->
<a name="ferret"></a>
<h2>Ferret genome</h2>
<h3>Apr. 2011 (MusPutFur1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=musFur1" 
    target="_blank">musFur1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/musFur1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/musFur1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/musFur1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#musFur1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="musFur1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/musFur1/vsHg38/">Ferret/Human (hg38)</a></li> 
      </ul>
    </div>
  </li> 
</ul>

<!-- Fugu Downloads ======================================================== -->
<a name="fugu"></a>
<h2>Fugu genome</h2>
<h3>Oct. 2011 (fugu5/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=fr3" 
    target="_blank">fr3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/fr3/bigZips/">Full data set</a></li>  
<!--
  <li><a href="/goldenPath/fr3/chromosomes/">Data set by chromosome</a></li> 
-->
  <li><a href="/goldenPath/fr3/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/fr3/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#fr3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="fr3pairwise"> 
      <ul>
        <li><a href="/goldenPath/fr3/vsGadMor1/">Fugu/Atlantic cod (gadMor1)</a></li>
        <li><a href="/goldenPath/fr3/vsLatCha1/">Fugu/Coelacanth (latCha1)</a></li>
        <li><a href="/goldenPath/fr3/vsOryLat2/">Fugu/Medaka (oryLat2)</a></li>
        <li><a href="/goldenPath/fr3/vsOreNil1/">Fugu/Nile tilapia (oreNil1)</a></li>
        <li><a href="/goldenPath/fr3/vsGasAcu1/">Fugu/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/fr3/vsTetNig2/">Fugu/Tetraodon (tetNig2)</a></li>
        <li><a href="/goldenPath/fr3/vsDanRer7/">Fugu/Zebrafish (danRer7)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#fr3mult>Multiple
    alignments</a>
    <div class="collapse" id="fr3mult"> 
      <ul> 
        <li><a href="/goldenPath/fr3/multiz8way/">Multiple alignments of 7 vertebrate genomes with 
        Fugu</a></li>
        <li><a href="/goldenPath/fr3/phastCons8way/">Conservation scores for alignments of 7 
        vertebrate genomes with Fugu</a></li>	
      </ul>
    </div>
  </li> 
</ul>
<h3>Oct. 2004 (JGI V4.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=fr2"
    target="_blank">fr2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/fr2/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/fr2/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/fr2/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/fr2/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#fr2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="fr2pairwise"> 
      <ul>
        <li><a href="/goldenPath/fr2/vsHg19/">Fugu/Human (hg19)</a></li>
        <li><a href="/goldenPath/fr2/vsHg18/">Fugu/Human (hg18)</a></li>
        <li><a href="/goldenPath/fr2/vsGalGal3/">Fugu/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/fr2/vsOryLat2/">Fugu/Medaka (oryLat2)</a></li> 
        <li><a href="/goldenPath/fr2/vsMm9/">Fugu/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/fr2/vsGasAcu1/">Fugu/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/fr2/vsTetNig2/">Fugu/Tetraodon (tetNig2)</a></li>
        <li><a href="/goldenPath/fr2/vsTetNig1/">Fugu/Tetraodon (tetNig1)</a></li>
        <li><a href="/goldenPath/fr2/vsDanRer7/">Fugu/Zebrafish (danRer7)</a></li>
        <li><a href="/goldenPath/fr2/vsDanRer5/">Fugu/Zebrafish (danRer5)</a></li>
        <li><a href="/goldenPath/fr2/vsDanRer4/">Fugu/Zebrafish (danRer4)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#fr2mult>Multiple
    alignments</a>
    <div class="collapse" id="fr2mult"> 
      <ul>
        <li><a href="/goldenPath/fr2/multiz5way/">Multiple alignments of 4 vertebrate genomes with 
        Fugu</a></li>
        <li><a href="/goldenPath/fr2/phastCons5Scores/">Conservation scores for alignments of 4 
        vertebrate genomes with Fugu</a></li>	
      </ul>
    </div>
  </li> 
</ul>
<h3>Aug. 2002 (fugu/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=fr1" 
    target="_blank">fr1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/fr1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/fr1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/fr1/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#fr1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="fr1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/fr1/vsHg18/">Fugu/Human (hg18)</a></li>
        <li><a href="/goldenPath/fr1/vsMm7/">Fugu/Mouse (mm7)</a></li>
        <li><a href="/goldenPath/fr1/vsMm6/">Fugu/Mouse (mm6)</a></li>
        <li><a href="/goldenPath/fr1/vsDanRer4/">Fugu/Zebrafish (danRer4)</a></li>
        <li><a href="/goldenPath/fr1/vsDanRer3/">Fugu/Zebrafish (danRer3)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Gibbon Downloads -->
<a name="gibbon"></a>
<h2>Gibbon genome</h2>
<h3>Oct. 2012 (Nleu_3.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=nomLeu3" 
    target="_blank">nomLeu3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/nomLeu3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/nomLeu3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/nomLeu3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#nomLeu3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="nomLeu3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/nomLeu3/vsHg19/">Gibbon/Human (hg19)</a></li>
        <li><a href="/goldenPath/nomLeu3/vsTarSyr2/">Gibbon/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3> Jun. 2011 (iGGSC Nleu1.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=nomLeu2"
target="_blank">nomLeu2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/nomLeu2/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/nomLeu2/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/nomLeu2/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#nomLeu2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="nomLeu2pairwise"> 
      <ul>
        <li><a href="/goldenPath/nomLeu2/vsMm10/">Gibbon/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jan. 2010 (GGSC Nleu1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=nomLeu1" 
    target="_blank">nomLeu1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/nomLeu1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/nomLeu1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/nomLeu1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#nomLeu1>Pairwise
    alignments</a>
    <div class="collapse" id="nomLeu1"> 
      <ul>
        <li><a href="/goldenPath/nomLeu1/vsHg19/">Gibbon/Human (hg19)</a></li> 
        <li><a href="/goldenPath/nomLeu1/vsGorGor3/">Gibbon/Gorilla (gorGor3)</a></li> 
        <li><a href="/goldenPath/nomLeu1/vsMm10/">Gibbon/Mouse (mm10)</a></li> 
      </ul>
    </div>
  </li> 
</ul>

<!-- Garter snake Downloads -->
<a name="garter_snake"></a>
<h2>Garter snake genome</h2>
<h3>Jun. 2015 (Thanmnophis_sirtalis-6.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=thaSir" 
    target="_blank">thaSir1</a>)</h3>
<ul>
  <li><a href="/goldenPath/thaSir1/bigZips/">Full data set</a>
  <li><a href="/goldenPath/thaSir1/database/">Annotation database</a>
  <li><a href="/goldenPath/thaSir1/liftOver/">LiftOver files</a>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#thaSir1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="thaSir1pairwise">
      <ul>
        <li><a href="/goldenPath/thaSir1/vsHg38/">Garter snake/Human (hg38)</a></li>
        <li><a href="/goldenPath/thaSir1/vsMm10/">Garter snake/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li>
</ul>
<!-- Golden eagle Downloads -->
<a name="golden_eagle"></a>
<h2>Golden eagle genome</h2> 
<h3>Oct. 2014 (aquChr-1.0.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=aquChr2" 
    target="_blank">aquChr2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/aquChr2/bigZips/">Full data set</a> 
  <li><a href="/goldenPath/aquChr2/database/">Annotation database</a> 
  <li><a href="/goldenPath/aquChr2/liftOver/">LiftOver files</a> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#aquChr2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="aquChr2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/aquChr2/vsHg38/">Golden eagle/Human (hg38)</a></li>
        <li><a href="/goldenPath/aquChr2/vsGalGal5/">Golden eagle/Chicken (galGal5)</a></li>
        <li><a href="/goldenPath/aquChr2/vsMm10/">Golden eagle/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Golden snub-nosed monkey Downloads -->
<a name="golden_snub-nosed_monkey"></a>
<h2>Golden snub-nosed monkey genome</h2> 
<h3>Oct. 2014 (Rrox_v1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=rhiRox1" 
    target="_blank">rhiRox1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/rhiRox1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/rhiRox1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/rhiRox1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rhiRox1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="rhiRox1pairwise"> 
      <ul>
        <li><a href="/goldenPath/rhiRox1/vsHg38/">Golden snub-nosed monkey/Human (hg38)</a></li>
        <li><a href="/goldenPath/rhiRox1/vsTarSyr2/">Golden snub-nosed monkey/Tarsier 
        (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Gorilla Downloads -->
<a name="gorilla"></a>
<h2>Gorilla genome</h2> 
<h3>Mar. 2016 (GMSRT3/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=gorGor5" 
    target="_blank">gorGor5</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/gorGor5/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/gorGor5/database/">Annotation database</a></li>
  <li><a href="/goldenPath/gorGor5/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#gorGor5pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="gorGor5pairwise"> 
      <ul> 
        <li><a href="/goldenPath/gorGor5/vsHg38/">Gorilla/Human (hg38)</a></li>
        <li><a href="/goldenPath/gorGor5/vsHg19/">Gorilla/Human (hg19)</a></li>
        <li><a href="/goldenPath/gorGor5/vsPanTro5/">Gorilla/Chimp (panTro5)</a></li>
        <li><a href="/goldenPath/gorGor5/vsPanTro4/">Gorilla/Chimp (panTro4)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Dec 2014 (gorGor4.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=gorGor4" 
    target="_blank">gorGor4</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/gorGor4/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/gorGor4/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/gorGor4/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#gorGor4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="gorGor4pairwise"> 
      <ul> 
        <li><a href="/goldenPath/gorGor4/vsHg38/">Gorilla/Human (hg38)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>May 2010 (gorGor3.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=gorGor3" 
    target="_blank">gorGor3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/gorGor3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/gorGor3/database/">Annotation database</a></li>
   <li><a href="/goldenPath/gorGor3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#gorGor3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="gorGor3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/gorGor3/vsHg19/">Gorilla/Human (hg19)</a></li> 
        <li><a href="/goldenPath/gorGor3/vsPapHam1/">Gorilla/Baboon (papHam1)</a></li>
        <li><a href="/goldenPath/gorGor3/vsPanTro3/">Gorilla/Chimp (panTro3)</a></li> 
        <li><a href="/goldenPath/gorGor3/vsNomLeu1/">Gorilla/Gibbon (nomLeu1)</a></li>
        <li><a href="/goldenPath/gorGor3/vsCalJac3/">Gorilla/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/gorGor3/vsMm10/">Gorilla/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/gorGor3/vsMicMur1/">Gorilla/Mouse lemur (micMur1)</a></li>
        <li><a href="/goldenPath/gorGor3/vsPonAbe2/">Gorilla/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/gorGor3/vsRheMac3/">Gorilla/Rhesus (rheMac3)</a></li>
        <li><a href="/goldenPath/gorGor3/vsRheMac2/">Gorilla/Rhesus (rheMac2)</a></li>
        <li><a href="/goldenPath/gorGor3/vsTarSyr2/">Gorilla/Tarsier (tarSyr2)</a></li>
        <li><a href="/goldenPath/gorGor3/vsTarSyr1/">Gorilla/Tarsier (tarSyr1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#gorGor3mult>Multiple
    alignments</a>
    <div class="collapse" id="gorGor3mult"> 
      <ul> 
        <li><a href="/goldenPath/gorGor3/multiz11way/">Multiple alignments of 11 vertebrate genomes 
        with Gorilla</a></li> 
        <li><a href="/goldenPath/gorGor3/phastCons11way/">Conservation scores for alignments of 11 
        vertebrate genomes with Gorilla</a></li> 
      </ul>
    </div>
  </li> 
</ul>

<!-- Green monkey Downloads -->
<a name="green_monkey"></a>
<h2>Green monkey genome</h2> 
<h3>Mar 2014 (Chlorocebus_sabeus 1.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=chlSab2" 
    target="_blank">chlSab2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/chlSab2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/chlSab2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/chlSab2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#chlSab2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="chlSab2pairwise"> 
      <ul>
        <li><a href="/goldenPath/chlSab2/vsHg38/">Green monkey/Human (hg38)</a></li>
        <li><a href="/goldenPath/chlSab2/vsRheMac8/">Green monkey/Rhesus (rheMac8)</a></li>
        <li><a href="/goldenPath/chlSab2/vsTarSyr2/">Green monkey/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Guinea pig Downloads -->
<a name="guinea_pig"></a>
<h2>Guinea pig genome</h2>
<h3>Feb. 2008 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=cavPor3" 
    target="_blank">cavPor3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/cavPor3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/cavPor3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/cavPor3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#cavPor3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="cavPor3pairwise"> 
      <ul>
<!-- missing 
        <li><a href="/goldenPath/cavPor3/vsHg19/">Guinea pig/Human (hg19)</a></li>
-->
        <li><a href="/goldenPath/cavPor3/vsHg18/">Guinea pig/Human (hg18)</a></li>
        <li><a href="/goldenPath/cavPor3/vsGalGal3/">Guinea pig/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/cavPor3/vsGalVar1/">Guinea pig/Malayan flying lemur
        (galVar1)</a></li>
        <li><a href="/goldenPath/cavPor3/vsMm10/">Guinea pig/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/cavPor3/vsMm9/">Guinea pig/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/cavPor3/vsMonDom4/">Guinea pig/Opossum (monDom4)</a></li>
        <li><a href="/goldenPath/cavPor3/vsOryCun2/">Guinea pig/Rabbit (oryCun2)</a></li>
        <Li><a href="/goldenPath/cavPor3/vsRn6/">Guinea pig/Rat (rn6)</a></li>
        <li><a href="/goldenPath/cavPor3/vsRn4/">Guinea pig/Rat (rn4)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Oct. 2005 (cavPor2) (sequence only)</h3> 
<ul> 
  <li><a href="/goldenPath/cavPor2/bigZips/">Sequence data set</a></li>
</ul>	

<!-- Hedgehog Downloads -->
<a name="hedgehog"></a>
<h2>Hedgehog genome</h2> 
<h3>May 2012 (EriEur2.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=eriEur2" 
    target="_blank">eriEur2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/eriEur2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/eriEur2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/eriEur2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#eriEur2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="eriEur2pairwise"> 
      <ul> 
       <li><a href="/goldenPath/eriEur2/vsHg19/">Hedgehog/Human (hg19)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jun. 2006 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=eriEur1" 
    target="_blank">eriEur1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/eriEur1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/eriEur1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/eriEur1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#eriEur1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="eriEur1pairwise"> 
      <ul>
        <li><a href="/goldenPath/eriEur1/vsMm10/">Hedgehog/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Horse Downloads ==================================================== -->
<a name="horse"></a>
<h2>Horse genome</h2>
<h3>Sep. 2007 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=equCab2" 
    target="_blank">equCab2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/equCab2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/equCab2/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/equCab2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/equCab2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#equCab2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="equCab2pairwise"> 
      <ul>
        <li><a href="/goldenPath/equCab2/vsHg38/">Horse/Human (hg38)</a></li>
        <li><a href="/goldenPath/equCab2/vsHg19/">Horse/Human (hg19)</a></li>
<!--
        <li><a href="/goldenPath/equCab2/vsHg18/">Horse/Human (hg18) pairwise alignments</a>
-->
        <li><a href="/goldenPath/equCab2/vsGalGal3/">Horse/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/equCab2/vsPanTro3/">Horse/Chimp (panTro3)</a></li>
        <li><a href="/goldenPath/equCab2/vsBosTau8/">Horse/Cow (bosTau8)</a></li>
        <li><a href="/goldenPath/equCab2/vsCanFam2/">Horse/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/equCab2/vsMm10/">Horse/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/equCab2/vsMm9/">Horse/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/equCab2/vsMonDom5/">Horse/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/equCab2/vsOrnAna1/">Horse/Platypus (ornAna1)</a></li>
      </ul>
    </div>
  </li> 
<!--
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#equCab2mult>Multiple
    alignments</a>
    <div class="collapse" id="equCab2mult"> 
      <ul>
        <li>Multiple alignments of 5 vertebrate genomes with horse - release pending</li>
        <li>Conservation scores for alignments of 5 vertebrate genomes with horse - release 
        pending</li>
        <li><a href="/goldenPath/equCab2/multiz6way/">Multiple alignments of 5 vertebrate genomes 
        with horse</li></a>
        <li><a href="/goldenPath/equCab2/phastCons6way/">Conservation scores for alignments of 5 
        vertebrate genomes with horse</li></a>
      </ul>
    </div>
  </li> 
-->
</ul>
<h3>Jan. 2007 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=equCab1" 
    target="_blank">equCab1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/equCab1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/equCab1/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/equCab1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/equCab1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#equCab1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="equCab1pairwise"> 
      <ul>
        <li><a href="/goldenPath/equCab1/vsHg18/">Horse/Human (hg18)</a></li>
        <li><a href="/goldenPath/equCab1/vsGalGal3/">Horse/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/equCab1/vsPanTro2/">Horse/Chimp (panTro2)</a></li>
        <li><a href="/goldenPath/equCab1/vsCanFam2/">Horse/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/equCab1/vsMm9/">Horse/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/equCab1/vsMm8/">Horse/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/equCab1/vsRn4/">Horse/Rat (rn4)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Kangaroo rat Downloads -->
<a name="kangaroo_rat"></a>
<h2>Kangaroo rat genome</h2>
<h3>Jul. 2008 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=dipOrd1" 
    target="_blank">dipOrd1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/dipOrd1/bigZips/">Full data set</a>
  <li><a href="/goldenPath/dipOrd1/database/">Annotation database</a>
  <li><a href="/goldenPath/dipOrd1/liftOver/">LiftOver files</a>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dipOrd1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="dipOrd1pairwise"> 
      <ul>
        <li><a href="/goldenPath/dipOrd1/vsMm10/">Kangaroo rat/Mouse (mm10)</a>
      </ul>
    </div>
  </li> 
</ul>

<!-- Lamprey Downloads -->
<a name="lamprey"></a>
<h2>Lamprey genome</h2>
<h3>Dec. 2017 (Pmar_germline 1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=petMar3"
    target="_blank">petMar3</a>)</h3>
<ul>
  <li><a href="/goldenPath/petMar3/bigZips/" target="_blank">Full data set</a></li>
  <li><a href="/goldenPath/petMar3/database/" target="_blank">Annotation database</a></li>
  <li><a href="/goldenPath/petMar3/liftOver/" target="_blank">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse"
    href=#petMar3pairwise>Pairwise alignments</a>
    <div class="collapse" id="petMar3pairwise">
      <ul>
        <li><a href="/goldenPath/petMar3/vsHg38/" target="_blank">Lamprey/Human (hg38)</a></li>
        <li><a href="/goldenPath/petMar3/vsMm10/" target="_blank">Lamprey/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li>
</ul>

<h3>Sep. 2010 (WUGSC 7.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=petMar2" 
    target="_blank">petMar2</a>)</h3> 
<ul>
  <li><a href="/goldenPath/petMar2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/petMar2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/petMar2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#petMar2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="petMar2pairwise"> 
      <ul>
        <li><a href="/goldenPath/petMar2/vsHg19/">Lamprey/Human (hg19)</a></li>
        <li><a href="/goldenPath/petMar2/vsGalGal4/">Lamprey/Chicken (galGal4)</a>
        <li><a href="/goldenPath/petMar2/vsBraFlo2/">Lamprey/Lancelet (braFlo2)</a>
        <li><a href="/goldenPath/petMar2/vsOryLat2/">Lamprey/Medaka (oryLat2)</a>
        <li><a href="/goldenPath/petMar2/vsMm10/">Lamprey/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/petMar2/vsMonDom5/">Lamprey/Opossum (monDom5)</a>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#petMar2mult>Multiple
    alignments</a>
    <div class="collapse" id="petMar2mult"> 
      <ul>
        <li><a href="/goldenPath/petMar2/multiz7way/">Multiple alignments of 6 genomes with Lamprey</a>
        <li><a href="/goldenPath/petMar2/phastCons7way/">Conservation scores for alignments of 6 genomes with Lamprey</a>
      </ul>
    </div>
  </li> 
</ul>
<h3> Mar. 2007 (WUGSC 3.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=petMar1" 
     target="_blank">petMar1</a>)</h3>
<ul> 
  <li><a href="/goldenPath/petMar1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/petMar1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/petMar1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#petMar1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="petMar1pairwise"> 
      <ul>
        <li><a href="/goldenPath/petMar1/vsHg19/">Lamprey/Human (hg19)</a></li>
        <li><a href="/goldenPath/petMar1/vsHg18/">Lamprey/Human (hg18)</a></li>
        <li><a href="/goldenPath/petMar1/vsGalGal3/">Lamprey/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/petMar1/vsBraFlo1/">Lamprey/Lancelet (braFlo1)</a></li>
        <li><a href="/goldenPath/petMar1/vsOryLat2/">Lamprey/Medaka (oryLat2)</a></li>
        <li><a href="/goldenPath/petMar1/vsMm9/">Lamprey/Mouse (mm9)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#petMar1mult>Multiple
    alignments</a>
    <div class="collapse" id="petMar1mult"> 
      <ul>
        <li><a href="/goldenPath/petMar1/multiz6way/">Multiple alignments of 5 genomes with 
        Lamprey</a>
        <li><a href="/goldenPath/petMar1/phastCons6way/">Conservation scores for alignments of 5 
        genomes with Lamprey</a>
      </ul>
    </div>
  </li> 
</ul>

<!-- Lancelet Downloads -->
<a name="lancelet"></a>
<h2>Lancelet genome</h2>
<h3>Mar. 2006 (JGI 1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=braFlo1" 
    target="_blank">braFlo1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/braFlo1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/braFlo1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/braFlo1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#braFlo1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="braFlo1pairwise"> 
    <ul>
      <li><a href="/goldenPath/braFlo1/vsHg18/">Lancelet/Human (hg18)</a></li>
      <li><a href="/goldenPath/braFlo1/vsGalGal3/">Lancelet/Chicken (galGal3)</a></li>
      <li><a href="/goldenPath/braFlo1/vsPetMar1/">Lancelet/Lamprey (petMar1)</a></li>
      <li><a href="/goldenPath/braFlo1/vsMm9/">Lancelet/Mouse (mm9)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#braFlo1mult>Multiple
    alignments</a>
    <div class="collapse" id="braFlo1mult"> 
      <ul> 
        <li><a href="/goldenPath/braFlo1/multiz5way/">Multiple alignments of 4 genomes with 
        Lancelet</a></li>
        <li><a href="/goldenPath/braFlo1/phastCons5way/">Conservation scores for alignments of 4 
        genomes with Lancelet</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Lizard Downloads -->
<a name="lizard"></a>
<h2>Lizard genome</h2>
<h3>May 2010 (Broad AnoCar2.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=anoCar2" 
    target="_blank">anoCar2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/anoCar2/bigZips/">Full data set</a>
  <li><a href="/goldenPath/anoCar2/database/">Annotation database</a>
  <li><a href="/goldenPath/anoCar2/liftOver/">LiftOver files</a> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#anoCar2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="anoCar2pairwise"> 
      <ul>
        <li><a href="/goldenPath/anoCar2/vsHg19/">Lizard/Human (hg19)</a></li>
        <li><a href="/goldenPath/anoCar2/vsGalGal3/">Lizard/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/anoCar2/vsMm10/">Lizard/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/anoCar2/vsMm9/">Lizard/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/anoCar2/vsOrnAna1/">Lizard/Platypus (ornAna1)</a></li>
        <li><a href="/goldenPath/anoCar2/vsGasAcu1/">Lizard/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/anoCar2/vsXenTro3/">Lizard/X. tropicalis (xenTro3)</a></li>
        <li><a href="/goldenPath/anoCar2/vsXenTro2/">Lizard/X. tropicalis (xenTro2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jan. 2007 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=anoCar1" 
    target="_blank">anoCar1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/anoCar1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/anoCar1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/anoCar1/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#anoCar1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="anoCar1pairwise"> 
      <ul>
        <li><a href="/goldenPath/anoCar1/vsHg19/">Lizard/Human (hg19)</a></li>
        <li><a href="/goldenPath/anoCar1/vsHg18/">Lizard/Human (hg18)</a></li>
        <li><a href="/goldenPath/anoCar1/vsGalGal3/">Lizard/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/anoCar1/vsMm8/">Lizard/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/anoCar1/vsMm9/">Lizard/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/anoCar1/vsOrnAna1/">Lizard/Platypus (ornAna1)</a></li>
        <li><a href="/goldenPath/anoCar1/vsGasAcu1/">Lizard/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/anoCar1/vsXenTro2/">Lizard/X. tropicalis
(xenTro2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Malayan flying lemur Downloads -->
<a name="malayan_flying_lemur"></a>
<h2>Malayan flying lemur genome</h2>
<h3>Jun. 2014 (G_variegatus-3.0.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=galVar1"
    target="_blank">galVar1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/galVar1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/galVar1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/galVar1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#galVar1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="galVar1pairwise"> 
      <UL>
        <li><a href="/goldenPath/galVar1/vsHg38/">Malayan flying lemur/Human (hg38)</a></li> 
        <li><a href="/goldenPath/galVar1/vsCavPor3/">Malayan flying lemur/Guinea pig (cavPor3)</a></li>
        <li><a href="/goldenPath/galVar1/vsMm10/">Malayan flying lemur/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/galVar1/vsTupBel1/">Malayan flying lemur/Tree shrew (tupBel1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#galVar1mult>Multiple
    alignments</a>
    <div class="collapse" id="galVar1mult"> 
      <ul> 
        <li><a href="/goldenPath/galVar1/multiz5way/">Multiple alignments of 5 vertebrate genomes 
        with Malayan flying lemur</a></li> 
        <li><a href="/goldenPath/galVar1/phyloP5way/">Conservation scores for alignments of 5 
        vertebrate genomes with Malyan flying lemur</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Manatee Downloads -->
<a name="manatee"></a>
<h2>Manatee genome</h2>
<h3>Oct. 2011 (Broad v1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=triMan1"
target="_blank">triMan1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/triMan1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/triMan1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/triMan1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#triMan1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="triMan1pairwise"> 
      <ul>
        <li><a href="/goldenPath/triMan1/vsHg38/">Manatee/Human (hg38)</a></li>
        <li><a href="/goldenPath/triMan1/vsMm10/">Manatee/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Marmoset Downloads -->
<a name="marmoset"></a>
<h2>Marmoset genome</h2>
<h3>Mar. 2009 (WUGSC 3.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=calJac3"
target="_blank">calJac3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/calJac3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/calJac3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/calJac3/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#calJac3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="calJac3pairwise"> 
      <ul>
        <li><a href="/goldenPath/calJac3/vsHg19/">Marmoset/Human (hg19)</a></li>
        <li><a href="/goldenPath/calJac3/vsHg38/">Marmoset/Human (hg38)</a></li>
        <li><a href="/goldenPath/calJac3/vsPapHam1/">Marmoset/Baboon (papHam1)</a></li>
        <li><a href="/goldenPath/calJac3/vsPanTro5/">Marmoset/Chimpanzee (panTro5)</a></li>	
        <li><a href="/goldenPath/calJac3/vsPanTro3/">Marmoset/Chimpanzee (panTro3)</a></li>	
        <li><a href="/goldenPath/calJac3/vsPanTro2/">Marmoset/Chimpanzee (panTro2)</a></li>
        <li><a href="/goldenPath/calJac3/vsCanFam2/">Marmoset/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/calJac3/vsGorGor3/">Marmoset/Gorilla (gorGor3)</a></li>
        <li><a href="/goldenPath/calJac3/vsMicMur1/">Marmoset/Mouse lemur (micMur1)</a></li>
        <li><a href="/goldenPath/calJac3/vsMm10/">Marmoset/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/calJac3/vsMm9/">Marmoset/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/calJac3/vsMonDom5/">Marmoset/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/calJac3/vsPonAbe2/">Marmoset/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/calJac3/vsRheMac8/">Marmoset/Rhesus (rheMac8)</a></li>
        <li><a href="/goldenPath/calJac3/vsRheMac3/">Marmoset/Rhesus (rheMac3)</a></li>
        <li><a href="/goldenPath/calJac3/vsRheMac2/">Marmoset/Rhesus (rheMac2)</a></li>
        <li><a href="/goldenPath/calJac3/vsTarSyr2/">Marmoset/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
<!--
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#calJac3mult>Multiple
    alignments</a>
    <div class="collapse" id="calJac3mult"> 
      <ul> 
        <li><a href="/goldenPath/calJac1/multiz9way/">Multiple alignments of 8 vertebrate genomes 
        with Marmoset</a></li>
        <li><a href="/goldenPath/calJac1/phastCons9way/">Conservation scores for alignments of 8 
        vertebrate genomes with Marmoset</a></li>
      </ul>
    </div>
  </li> 
-->
</ul>
<h3>Jun. 2007 (WUGSC 2.0.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=calJac1"
    target="_blank">calJac1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/calJac1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/calJac1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/calJac1/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#calJac1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="calJac1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/calJac1/vsHg19/">Marmoset/Human (hg19)</a></li>
        <li><a href="/goldenPath/calJac1/vsHg18/">Marmoset/Human (hg18)</a></li>
        <li><a href="/goldenPath/calJac1/vsPanTro2/">Marmoset/Chimpanzee (panTro2)</a></li>
        <li><a href="/goldenPath/calJac1/vsCanFam2/">Marmoset/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/calJac1/vsMm9/">Marmoset/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/calJac1/vsMonDom4/">Marmoset/Opossum (monDom4)</a></li>
        <li><a href="/goldenPath/calJac1/vsPonAbe2/">Marmoset/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/calJac1/vsOrnAna1/">Marmoset/Platypus (ornAna1)</a></li>
        <li><a href="/goldenPath/calJac1/vsRheMac2/">Marmoset/Rhesus (rheMac2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#calJac1mult>Multiple
    alignments</a>
    <div class="collapse" id="calJac1mult"> 
      <ul> 
        <li><a href="/goldenPath/calJac1/multiz9way/">Multiple alignments of 8 vertebrate genomes 
        with Marmoset</a></li>
        <li><a href="/goldenPath/calJac1/phastCons9way/">Conservation scores for alignments of 8 
        vertebrate genomes with Marmoset</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Medaka Downloads ================================================= -->
<a name="medaka"></a>
<h2>Medaka genome</h2>
<h3>Oct. 2005 (NIG/UT MEDAKA1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=oryLat2"
target="_blank">oryLat2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/oryLat2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/oryLat2/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/oryLat2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/oryLat2/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#oryLat2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="oryLat2pairwise"> 
      <ul>
        <li><a href="/goldenPath/oryLat2/vsHg19/">Medaka/Human (hg19)</a></li>
        <li><a href="/goldenPath/oryLat2/vsHg18/">Medaka/Human (hg18)</a></li>
        <li><a href="/goldenPath/oryLat2/vsFr3/">Medaka/Fugu (fr3)</a></li>
        <li><a href="/goldenPath/oryLat2/vsFr2/">Medaka/Fugu (fr2)</a></li>
        <li><a href="/goldenPath/oryLat2/vsPetMar2/">Medaka/Lamprey (petMar2)</a></li>
        <li><a href="/goldenPath/oryLat2/vsPetMar1/">Medaka/Lamprey (petMar1)</a></li>
        <li><a href="/goldenPath/oryLat2/vsMm10/">Medaka/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/oryLat2/vsMm9/">Medaka/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/oryLat2/vsGasAcu1/">Medaka/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/oryLat2/vsTetNig2/">Medaka/Tetraodon (tetNig2)</a></li>
        <li><a href="/goldenPath/oryLat2/vsTetNig1/">Medaka/Tetraodon (tetNig1)</a></li>
        <li><a href="/goldenPath/oryLat2/vsDanRer6/">Medaka/Zebrafish (danRer6)</a></li>
        <li><a href="/goldenPath/oryLat2/vsDanRer5/">Medaka/Zebrafish (danRer5)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#oryLat2mult>Multiple
    alignments</a>
    <div class="collapse" id="oryLat2mult"> 
      <ul> 
        <li><a href="/goldenPath/oryLat2/multiz5way/">Multiple alignments of 4 vertebrate genomes 
        with Medaka</a></li>
        <li><a href="/goldenPath/oryLat2/phastCons5way/">Conservation scores for alignments of 4 
        vertebrate genomes with Medaka</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Medium ground finch Downloads -->
<a name="medium_ground_finch"></a>
<h2>Medium ground finch genome</h2>
<h3>Apr. 2012 (GeoFor_1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=geoFor1"
    target="_blank">geoFor1</a>)</h3>
<ul> 
  <li><a href="/goldenPath/geoFor1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/geoFor1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/geoFor1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#geoFor1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="geoFor1pairwise"> 
      <ul>
        <li><a href="/goldenPath/geoFor1/vsHg19/">Medium ground finch/Human (hg19)</a></li>
        <li><a href="/goldenPath/geoFor1/vsMelUnd1/">Medium ground finch/Budgerigar (melUnd1)</a></li>
        <li><a href="/goldenPath/geoFor1/vsGalGal4/">Medium ground finch/Chicken (galGal4)</a></li>
        <li><a href="/goldenPath/geoFor1/vsMm10/">Medium ground finch/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/geoFor1/vsMelGal1/">Medium ground finch/Turkey (melGal1)</a></li>
        <li><a href="/goldenPath/geoFor1/vsTaeGut1/">Medium ground finch/Zebra finch (taeGut1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#geoFor1mult>Multiple
    alignments</a>
    <div class="collapse" id="geoFor1mult"> 
      <ul> 
        <li><a href="/goldenPath/geoFor1/multiz7way/">Multiple alignments of 6 vertebrate genomes 
        with the Medium ground finch</a>
        <li><a href="/goldenPath/geoFor1/phastCons7way/">Conservation scores for alignments of 6 
        vertebrate genomes with the Medium ground finch</a>
        <li><a href="/goldenPath/geoFor1/phyloP7way/">Basewise conservation scores (phyloP) of 6 
        vertebrate genomes with the Medium ground finch</a>
      </ul>
    </div>
  </li> 
</ul>

<!-- Megabat Downloads -->
<a name="megabat"></a>
<h2>Megabat genome</h2>
<h3>Jul. 2008 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=pteVam1"
target="_blank">pteVam1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/pteVam1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/pteVam1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/pteVam1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#pteVam1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="pteVam1pairwise"> 
      <ul>
        <li><a href="/goldenPath/pteVam1/vsMm10/">Megabat/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Microbat Downloads -->
<a name="microbat"></a>
<h2>Microbat genome</h2>
<h3>Jul. 2010 (Myoluc2.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=myoLuc2"
    target="_blank">myoLuc2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/myoLuc2/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/myoLuc2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/myoLuc2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#myoLuc2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="myoLuc2pairwise"> 
      <ul>
        <li><a href="/goldenPath/myoLuc2/vsHg38/">Microbat/Human (hg38)</a></li>
        <li><a href="/goldenPath/myoLuc2/vsMm10/">Microbat/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Minke whale Downloads -->
<a name="minke_whale"></a>
<h2>Minke whale genome</h2>
<h3>Oct. 2013 (KORDI BalAcu1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=balAcu1" 
    target="_blank">balAcu1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/balAcu1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/balAcu1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/balAcu1/liftOver/">Liftover files</a></li>
</ul>

<!-- Mouse Downloads -->
<a name="mouse"></a>
<h2>Mouse genome</h2>
<h3>Dec. 2011 (GRCm38/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=mm10"
target="_blank">mm10</a>)</h3> 
<ul>
  <li><a href="/goldenPath/mm10/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/mm10/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/mm10/database/">Annotation database</a></li>
  <li><a href="/goldenPath/mm10/snp142Mask/">SNP142-masked FASTA files</a></li>
  <li><a href="/goldenPath/mm10/liftOver/">LiftOver files</a></li>
  <li><a href="/goldenPath/proteinDB/proteins180404/database/">Protein database for mm10</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm10pairwise-primate>Pairwise
    alignments (primates)</a>
    <div class="collapse" id="mm10pairwise-primate">  
      <ul>
        <li><a href="/goldenPath/mm10/vsHg38/">Mouse/Human (hg38)</a></li>
        <li><a href="/goldenPath/mm10/vsHg19/">Mouse/Human (hg19)</a></li>
        <li><a href="/goldenPath/mm10/vsPapAnu4/">Mouse/Baboon (papAnu4)</a></li>
        <li><a href="/goldenPath/mm10/vsPapHam1/">Mouse/Baboon (papHam1)</a></li>
        <li><a href="/goldenPath/mm10/vsOtoGar3/">Mouse/Bushbaby (otoGar3)</a></li>
        <li><a href="/goldenPath/mm10/vsPanTro6/">Mouse/Chimp (panTro6)</a></li>
        <li><a href="/goldenPath/mm10/vsPanTro5/">Mouse/Chimp (panTro5)</a></li>
        <li><a href="/goldenPath/mm10/vsPanTro4/">Mouse/Chimp (panTro4)</a></li>
        <li><a href="/goldenPath/mm10/vsNomLeu2/">Mouse/Gibbon (nomLeu2)</a></li>
        <li><a href="/goldenPath/mm10/vsNomLeu1/">Mouse/Gibbon (nomLeu1)</a></li>
        <li><a href="/goldenPath/mm10/vsGorGor3/">Mouse/Gorilla (gorGor3)</a></li>
        <li><a href="/goldenPath/mm10/vsPetMar3/">Mouse/Lamprey (petMar3)</a></li>
        <li><a href="/goldenPath/mm10/vsCalJac3/">Mouse/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/mm10/vsMicMur1/">Mouse/Mouse lemur (micMur1)</a></li>
        <li><a href="/goldenPath/mm10/vsPonAbe2/">Mouse/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/mm10/vsPonAbe3/">Mouse/Orangutan (ponAbe3)</a></li>
        <li><a href="/goldenPath/mm10/vsRheMac8/">Mouse/Rhesus (rheMac8)</a></li>
        <li><a href="/goldenPath/mm10/vsRheMac3/">Mouse/Rhesus (rheMac3)</a></li>
        <li><a href="/goldenPath/mm10/vsTarSyr2/">Mouse/Tarsier (tarSyr2)</a></li>
        <li><a href="/goldenPath/mm10/vsTarSyr1/">Mouse/Tarsier (tarSyr1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm10pairwise-mammal>Pairwise
    alignments (mammals)</a>
    <div class="collapse" id="mm10pairwise-mammal">  
      <ul>
        <li><a href="/goldenPath/mm10/vsVicPac2/">Mouse/Alpaca (vicPac2)</a></li>
        <li><a href="/goldenPath/mm10/vsVicPac1/">Mouse/Alpaca (vicPac1)</a></li>
        <li><a href="/goldenPath/mm10/vsDasNov3/">Mouse/Armadillo (dasNov3)</a></li>
        <li><a href="/goldenPath/mm10/vsFelCat5/">Mouse/Cat (felCat5)</a></li>
        <li><a href="/goldenPath/mm10/vsFelCat9/">Mouse/Cat (felCat9)</a></li>
        <li><a href="/goldenPath/mm10/vsManPen1/">Mouse/Chinese Pangolin (manPen1)</a></li>
        <li><a href="/goldenPath/mm10/vsCriGriChoV2/">Human/CHO K1 cell line (criGriChoV2)</a></li>
        <li><a href="/goldenPath/mm10/vsBosTau9/">Mouse/Cow (bosTau9)</a></li>
        <li><a href="/goldenPath/mm10/vsBosTau8/">Mouse/Cow (bosTau8)</a></li>
        <li><a href="/goldenPath/mm10/vsBosTau7/">Mouse/Cow (bosTau7)</a></li>
        <li><a href="/goldenPath/mm10/vsCanFam3/">Mouse/Dog (canFam3)</a></li>
        <li><a href="/goldenPath/mm10/vsTurTru2/">Mouse/Dolphin (turTru2)</a></li>
        <li><a href="/goldenPath/mm10/vsLoxAfr3/">Mouse/Elephant (loxAfr3)</a></li>
        <li><a href="/goldenPath/mm10/vsCavPor3/">Mouse/Guinea pig (cavPor3)</a></li>
        <li><a href="/goldenPath/mm10/vsEriEur1/">Mouse/Hedgehog (eriEur1)</a></li>
        <li><a href="/goldenPath/mm10/vsEquCab2/">Mouse/Horse (equCab2)</a></li>
        <li><a href="/goldenPath/mm10/vsDipOrd1/">Mouse/Kangaroo rat (dipOrd1)</a></li>
        <li><a href="/goldenPath/mm10/vsGalVar1/">Mouse/Malayan flying lemur (galVar1)</a></li>
        <li><a href="/goldenPath/mm10/vsTriMan1/">Mouse/Manatee (triMan1)</a></li>
        <li><a href="/goldenPath/mm10/vsPteVam1/">Mouse/Megabat (pteVam1)</a></li>
        <li><a href="/goldenPath/mm10/vsMyoLuc2/">Mouse/Microbat (myoLuc2)</a></li>
        <li><a href="/goldenPath/mm10/vsHetGla2/">Mouse/Naked mole-rat (hetGla2)</a></li>
        <li><a href="/goldenPath/mm10/vsHetGla1/">Mouse/Naked mole-rat (hetGla1)</a></li>
        <li><a href="/goldenPath/mm10/vsMonDom5/">Mouse/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/mm10/vsAilMel1/">Mouse/Panda (ailMel1)</a></li>  
        <li><a href="/goldenPath/mm10/vsSusScr11/">Mouse/Pig (susScr11)</a></li>        
        <li><a href="/goldenPath/mm10/vsSusScr3/">Mouse/Pig (susScr3)</a></li>
        <li><a href="/goldenPath/mm10/vsOchPri2/">Mouse/Pika (ochPri2)</a></li>
        <li><a href="/goldenPath/mm10/vsOrnAna1/">Mouse/Platypus (ornAna1)</a></li>
        <li><a href="/goldenPath/mm10/vsOryCun2/">Mouse/Rabbit (oryCun2)</a></li>
        <li><a href="/goldenPath/mm10/vsRn6/">Mouse/Rat (rn6)</a></li>
        <li><a href="/goldenPath/mm10/vsRn5/">Mouse/Rat (rn5)</a></li>
        <li><a href="/goldenPath/mm10/vsProCap1/">Mouse/Rock hyrax (proCap1)</a></li>
        <li><a href="/goldenPath/mm10/vsOviAri1/">Mouse/Sheep (oviAri1)</a></li>
        <li><a href="/goldenPath/mm10/vsOviAri4/">Human/Sheep (oviAri4)</a></li>
        <li><a href="/goldenPath/mm10/vsSorAra2/">Mouse/Shrew (sorAra2)</a></li>
        <li><a href="/goldenPath/mm10/vsSorAra1/">Mouse/Shrew (sorAra1)</a></li>
        <li><a href="/goldenPath/mm10/vsChoHof1/">Mouse/Sloth (choHof1)</a></li>
        <li><a href="/goldenPath/mm10/vsSpeTri2/">Mouse/Squirrel (speTri2)</a></li>
        <li><a href="/goldenPath/mm10/vsSaiBol1/">Mouse/Squirrel monkey (saiBol1)</a></li>
        <li><a href="/goldenPath/mm10/vsSarHar1/">Mouse/Tasmanian devil (sarHar1)</a></li>
        <li><a href="/goldenPath/mm10/vsEchTel2/">Mouse/Tenrec (echTel2)</a></li>
        <li><a href="/goldenPath/mm10/vsEchTel1/">Mouse/Tenrec (echTel1)</a></li>
        <li><a href="/goldenPath/mm10/vsTupBel1/">Mouse/Tree shrew (tupBel1)</a></li>
        <li><a href="/goldenPath/mm10/vsMacEug2/">Mouse/Wallaby (macEug2)</a></li>
        <li><a href="/goldenPath/mm10/vsCerSim1/">Mouse/White rhinoceros (cerSim1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm10pairwise-vert>Pairwise
    alignments (other vertebrates)</a>
    <div class="collapse" id="mm10pairwise-vert">  
      <ul>
        <li><a href="/goldenPath/mm10/vsXenLae2/">Mouse/African clawed frog (xenLae2)</a></li> 
        <li><a href="/goldenPath/mm10/vsGadMor1/">Mouse/Atlantic cod (gadMor1)</a></li>
        <li><a href="/goldenPath/mm10/vsMelUnd1/">Mouse/Budgerigar (melUnd1)</a></li>
        <li><a href="/goldenPath/mm10/vsGalGal6/">Mouse/Chicken (galGal6)</a></li>
        <li><a href="/goldenPath/mm10/vsGalGal5/">Mouse/Chicken (galGal5)</a></li>
        <li><a href="/goldenPath/mm10/vsGalGal4/">Mouse/Chicken (galGal4)</a></li>
        <li><a href="/goldenPath/mm10/vsCriGriChoV1/">Mouse/Chinese hamster ovary (CHO) K1 cell line
        (criGriChoV1)</a></li> 
        <li><a href="/goldenPath/mm10/vsLatCha1/">Mouse/Coelacanth (latCha1)</a></li>
        <li><a href="/goldenPath/mm10/vsFr3/">Mouse/Fugu (fr3)</a></li>
        <li><a href="/goldenPath/mm10/vsthaSir1/">Mouse/Garter snake (thaSir1)</a></li>
        <li><a href="/goldenPath/mm10/vsAquChr2/">Mouse/Golden eagle (aquChr2)</a></li>
        <li><a href="/goldenPath/mm10/vsPetMar2/">Mouse/Lamprey (petMar2)</a></li>
        <li><a href="/goldenPath/mm10/vsAnoCar2/">Mouse/Lizard (anoCar2)</a></li>
        <li><a href="/goldenPath/mm10/vsOryLat2/">Mouse/Medaka (oryLat2)</a></li>
        <li><a href="/goldenPath/mm10/vsOreNil2/">Mouse/Nile tilapia (oreNil2)</a></li>
        <li><a href="/goldenPath/mm10/vsChrPic1/">Mouse/Painted turtle (chrPic1)</a></li>
        <li><a href="/goldenPath/mm10/vsGasAcu1/">Mouse/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/mm10/vsTetNig2/">Mouse/Tetraodon (tetNig2)</a></li>
        <li><a href="/goldenPath/mm10/vsMelGal1/">Mouse/Turkey (melGal1)</a></li>
        <li><a href="/goldenPath/mm10/vsXenTro3/">Mouse/X.  tropicalis (xenTro3)</a></li>
        <li><a href="/goldenPath/mm10/vsTaeGut1/">Mouse/Zebra finch (taeGut1)</a></li>
        <li><a href="/goldenPath/mm10/vsDanRer10/">Mouse/Zebrafish (danRer10)</a></li>
        <li><a href="/goldenPath/mm10/vsDanRer7/">Mouse/Zebrafish (danRer7)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm10mult>Multiple
    alignments</a>
    <div class="collapse" id="mm10mult"> 
      <ul> 
        <li><a href="/goldenPath/mm10/multiz60way/">Multiple alignments of 59 vertebrate genomes 
        with Mouse</a></li>
        <li><a href="/goldenPath/mm10/phastCons60way/">Conservation scores for alignments of 59 
        vertebrate genomes with Mouse</a></li>
        <li><a href="/goldenPath/mm10/phyloP60way/">Basewise conservation scores (phyloP) of 59 
        vertebrate genomes with Mouse</a></li>
        <li><a href="/goldenPath/mm10/multiz60way/alignments/">FASTA alignments of 59 vertebrate 
        genomes with Mouse for CDS regions</a></li>
      </ul>
    </div>
  </li> 
  <li><a href="/goldenPath/mm10/mm10Patch4/">GRCm38 Patch 4 - Full data set</a></li>
</ul>
<h3>Jul. 2007 (NCBI37/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=mm9" 
    target="_blank">mm9</a>)</h3>
<ul> 
  <li><a href="/goldenPath/mm9/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/mm9/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/mm9/database/">Annotation database</a></li>
  <li><a href="/goldenPath/mm9/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm9pairwise-primate>Pairwise
    alignments (primates)</a>
    <div class="collapse" id="mm9pairwise-primate"> 
      <ul>
        <li><a href="/goldenPath/mm9/vsHg19/">Mouse/Human (hg19)</a></li>
        <li><a href="/goldenPath/mm9/vsHg18/">Mouse/Human (hg18)</a></li>
        <li><a href="/goldenPath/mm9/vsPanTro3/">Mouse/Chimp (panTro3)</a></li>
        <li><a href="/goldenPath/mm9/vsPanTro2/">Mouse/Chimp (panTro2)</a></li>
        <li><a href="/goldenPath/mm9/vsCalJac3/">Mouse/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/mm9/vsCalJac1/">Mouse/Marmoset (calJac1)</a></li>
        <li><a href="/goldenPath/mm9/vsPonAbe2/">Mouse/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/mm9/vsRheMac2/">Mouse/Rhesus (rheMac2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm9pairwise-mammal>Pairwise
    alignments (mammals)</a>
    <div class="collapse" id="mm9pairwise-mammal"> 
      <ul>
        <li><a href="/goldenPath/mm9/vsDasNov1/">Mouse/Armadillo (dasNov1)</a></li>
        <li><a href="/goldenPath/mm9/vsBosTau7/">Mouse/Cow (bosTau7)</a></li>
        <li><a href="/goldenPath/mm9/vsBosTau6/">Mouse/Cow (bosTau6)</a></li>
        <li><a href="/goldenPath/mm9/vsBosTau4/">Mouse/Cow (bosTau4)</a></li>
        <li><a href="/goldenPath/mm9/vsBosTau3/">Mouse/Cow (bosTau3)</a></li>
        <li><a href="/goldenPath/mm9/vsCanFam2/">Mouse/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/mm9/vsLoxAfr1/">Mouse/Elephant (loxAfr1)</a></li>
        <li><a href="/goldenPath/mm9/vsCavPor3/">Mouse/Guinea pig (cavPor3)</a></li>
        <li><a href="/goldenPath/mm9/vsCavPor2/">Mouse/Guinea pig (cavPor2)</a></li>
        <li><a href="/goldenPath/mm9/vsEriEur1/">Mouse/Hedgehog (eriEur1)</a></li>
        <li><a href="/goldenPath/mm9/vsEquCab2/">Mouse/Horse (equCab2)</a></li>
        <li><a href="/goldenPath/mm9/vsEquCab1/">Mouse/Horse (equCab1)</a></li>
        <li><a href="/goldenPath/mm9/vsMonDom5/">Mouse/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/mm9/vsMonDom4/">Mouse/Opossum (monDom4)</a></li>
        <li><a href="/goldenPath/mm9/vsAilMel1/">Mouse/Panda (ailMel1)</a></li>
        <li><a href="/goldenPath/mm9/vsSusScr2/">Mouse/Pig (susScr2)</a></li>
        <li><a href="/goldenPath/mm9/vsOrnAna1/">Mouse/Platypus (ornAna1)</a></li>
        <li><a href="/goldenPath/mm9/vsOryCun2/">Mouse/Rabbit (oriCun2)</a></li>
        <li><a href="/goldenPath/mm9/vsRn5/">Mouse/Rat (rn5)</a></li>
        <li><a href="/goldenPath/mm9/vsRn4/">Mouse/Rat (rn4)</a></li>
        <li><a href="/goldenPath/mm9/vsEchTel1/">Mouse/Tenrec (echTel1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm9pairwise-vert>Pairwise
    alignments (other vertebrates)</a>
    <div class="collapse" id="mm9pairwise-vert">  
      <ul>
        <li><a href="/goldenPath/mm9/vsGalGal3/">Mouse/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/mm9/vsFr2/">Mouse/Fugu (fr2)</a></li>
        <li><a href="/goldenPath/mm9/vsPetMar1/">Mouse/Lamprey (petMar1)</a></li>
        <li><a href="/goldenPath/mm9/vsBraFlo1/">Mouse/Lancelet (braFlo1)</a></li>
        <li><a href="/goldenPath/mm9/vsAnoCar2/">Mouse/Lizard (anoCar2)</a></li>
        <li><a href="/goldenPath/mm9/vsAnoCar1/">Mouse/Lizard (anoCar1)</a></li>
        <li><a href="/goldenPath/mm9/vsOryLat2/">Mouse/Medaka (oryLat2)</a></li>
        <li><a href="/goldenPath/mm9/vsGasAcu1/">Mouse/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/mm9/vsTetNig2/">Mouse/Tetraodon (tetNig2)</a></li>
        <li><a href="/goldenPath/mm9/vsTetNig1/">Mouse/Tetraodon (tetNig1)</a></li>
        <li><a href="/goldenPath/mm9/vsMelGal1/">Mouse/Turkey (melGal1)</a></li>
        <li><a href="/goldenPath/mm9/vsXenTro3/">Mouse/X. tropicalis (xenTro3)</a></li>
        <li><a href="/goldenPath/mm9/vsXenTro2/">Mouse/X. tropicalis (xenTro2)</a></li>
        <li><a href="/goldenPath/mm9/vsDanRer7/">Mouse/Zebrafish (danRer7)</a></li>
        <li><a href="/goldenPath/mm9/vsDanRer6/">Mouse/Zebrafish (danRer6)</a></li>
        <li><a href="/goldenPath/mm9/vsDanRer5/">Mouse/Zebrafish (danRer5)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm9mult>Multiple
    alignments</a>
    <div class="collapse" id="mm9mult"> 
      <ul> 
        <li><a href="/goldenPath/mm9/multiz30way/">Multiple alignments of 29 vertebrate genomes with
        Mouse</a></li>
        <li><a href="/goldenPath/mm9/phastCons30way/">Conservation scores for alignments of 29 
        vertebrate genomes with Mouse</a></li>
        <li><a href="/goldenPath/mm9/phyloP30way/">Basewise conservation scores (phyloP) of 29 
        vertebrate genomes with Mouse</a></li>
        <li><a href="/goldenPath/mm9/multiz30way/alignments/">FASTA alignments of 29 vertebrate 
        genomes with Mouse for CDS regions</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Mar. 2006 (NCBI36/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=mm8" 
    target="_blank">mm8</a>)</h3>
<ul> 
  <li><a href="/goldenPath/mm8/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/mm8/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/mm8/database/">Annotation database</a></li>
  <li><a href="/goldenPath/mm8/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm8pairwise>Pairwise
    alignments (mammals)</a>
    <div class="collapse" id="mm8pairwise"> 
      <ul>
        <li><a href="/goldenPath/mm8/vsHg18/">Mouse/Human (hg18)</a></li>
        <li><a href="/goldenPath/mm8/vsFelCat3/">Mouse/Cat (felCat3)</a></li>
        <li><a href="/goldenPath/mm8/vsPanTro2/">Mouse/Chimp (panTro2)</a></li>
        <li><a href="/goldenPath/mm8/vsPanTro1/">Mouse/Chimp (panTro1)</a></li>
        <li><a href="/goldenPath/mm8/vsBosTau3/">Mouse/Cow (bosTau3)</a></li>
        <li><a href="/goldenPath/mm8/vsBosTau2/">Mouse/Cow (bosTau2)</a></li>
        <li><a href="/goldenPath/mm8/vsCanFam2/">Mouse/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/mm8/vsEquCab1/">Mouse/Horse (equCab1)</a></li>
        <li><a href="/goldenPath/mm8/vsMonDom4/">Mouse/Opossum (monDom4)</a></li>
        <li><a href="/goldenPath/mm8/vsOrnAna1/">Mouse/Platypus (ornAna1)</a></li>
        <li><a href="/goldenPath/mm8/vsRn4/">Mouse/Rat (rn4)</a></li>
        <li><a href="/goldenPath/mm8/vsRheMac2/">Mouse/Rhesus (rheMac2)</a></li>
        <li><a href="/goldenPath/mm8/vsEchTel1/">Mouse/Tenrec (echTel1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm8pairwise-vert>Pairwise
    alignments (other vertebrates)</a>
    <div class="collapse" id="mm8pairwise-vert">  
      <ul>
        <li><a href="/goldenPath/mm8/vsGalGal3/">Mouse/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/mm8/vsGalGal2/">Mouse/Chicken (galGal2)</a></li>
        <li><a href="/goldenPath/mm8/vsFr1/">Mouse/Fugu (fr1)</a></li>
        <li><a href="/goldenPath/mm8/vsAnoCar1/">Mouse/Lizard (anoCar1)</a></li>
        <li><a href="/goldenPath/mm8/vsTetNig1/">Mouse/Tetraodon (tetNig1)</a></li>
        <li><a href="/goldenPath/mm8/vsXenTro1/">Mouse/X. tropicalis (xenTro1)</a></li>
        <li><a href="/goldenPath/mm8/vsDanRer4/">Mouse/Zebrafish (danRer4)</a></li>
        <li><a href="/goldenPath/mm8/vsDanRer3/">Mouse/Zebrafish (danRer3)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm8mult>Multiple
    alignments</a>
    <div class="collapse" id="mm8mult"> 
      <ul> 
        <li><a href="/goldenPath/mm8/multiz17way/">Multiple alignments of 16 vertebrate genomes with
         Mouse</a></li>
        <li><a href="/goldenPath/mm8/phastCons17Scores/">Conservation scores for alignments of 16 
        vertebrate genomes with Mouse</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Aug. 2005 (NCBI35/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=mm7" 
    target="_blank">mm7</a>)</h3>
<ul> 
  <li><a href="/goldenPath/mm7/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/mm7/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/mm7/database/">Annotation database</a></li>
  <li><a href="/goldenPath/mm7/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm7pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="mm7pairwise"> 
      <ul>
        <li><a href="/goldenPath/mm7/vsHg18/">Mouse/Human (hg18)</a></li>
        <li><a href="/goldenPath/mm7/vsHg17/">Mouse/Human (hg17)</a></li>
        <li><a href="/goldenPath/mm7/vsGalGal2/">Mouse/Chicken (galGal2)</a></li>
        <li><a href="/goldenPath/mm7/vsPanTro1/">Mouse/Chimp (panTro1)</a></li>
        <li><a href="/goldenPath/mm7/vsBosTau2/">Mouse/Cow (bosTau2)</a></li>
        <li><a href="/goldenPath/mm7/vsFr1/">Mouse/Fugu (fr1)</a></li>
        <li><a href="/goldenPath/mm7/vsRn3/">Mouse/Rat (rn3)</a></li>
        <li><a href="/goldenPath/mm7/vsRheMac1/">Mouse/Rhesus (rheMac1)</a></li>
        <li><a href="/goldenPath/mm7/vsEchTel1/">Mouse/Tenrec (echTel1)</a></li>
        <li><a href="/goldenPath/mm7/vsTetNig1/">Mouse/Tetraodon (tetNig1)</a></li>
        <li><a href="/goldenPath/mm7/vsXenTro1/">Mouse/X. tropicalis (xenTro1)</a></li>
        <li><a href="/goldenPath/mm7/vsDanRer3/">Mouse/Zebrafish (danRer3)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm7mult>Multiple
    alignments</a>
    <div class="collapse" id="mm7mult"> 
      <ul> 
        <li><a href="/goldenPath/mm7/multiz17way/">Multiple alignments of 16 vertebrate genomes with
        Mouse</a></li>
        <li><a href="/goldenPath/mm7/phastCons17Scores/">Conservation scores for alignments of 16 
        vertebrate genomes with Mouse</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Mar. 2005 (mm6)</h3> 
<ul> 
  <li><a href="/goldenPath/mm6/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/mm6/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/mm6/database/">Annotation database</a></li>
  <li><a href="/goldenPath/mm6/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm6pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="mm6pairwise"> 
      <ul>
        <li><a href="/goldenPath/mm6/vsSelf/">Mouse self alignments</a></li>
        <li><a href="/goldenPath/mm6/vsHg17/">Mouse/Human (hg17)</a></li>
        <li><a href="/goldenPath/mm6/vsHg16/">Mouse/Human (hg16)</a></li>
        <li><a href="/goldenPath/mm6/vsGalGal2/">Mouse/Chicken (galGal2)</a></li>
        <li><a href="/goldenPath/mm6/vsPanTro1/">Mouse/Chimp (panTro1)</a></li>
        <li><a href="/goldenPath/mm6/vsBosTau1/">Mouse/Cow (bosTau1)</a></li>
        <li><a href="/goldenPath/mm6/vsCanFam1/">Mouse/Dog (canFam1)</a></li>
        <li><a href="/goldenPath/mm6/vsFr1/">Mouse/Fugu (fr1)</a></li>
        <li><a href="/goldenPath/mm6/vsMonDom1/">Mouse/Opossum (monDom1)</a></li>
        <li><a href="/goldenPath/mm6/vsRn3/">Mouse/Rat (rn3)</a></li>
        <li><a href="/goldenPath/mm6/vsTetNig1/">Mouse/Tetraodon (tetNig1)</a></li>
        <li><a href="/goldenPath/mm6/vsXenTro1/">Mouse/X. tropicalis (xenTro1)</a></li>
        <li><a href="/goldenPath/mm6/vsDanRer2/">Mouse/Zebrafish (danRer2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm6mult>Multiple
    alignments</a>
    <div class="collapse" id="mm6mult"> 
      <ul> 
        <li><a href="/goldenPath/mm6/multiz10way/">Multiple alignments of 9 vertebrate genomes with 
        Mouse</a></li>
        <li><a href="/goldenPath/mm6/phastCons10Scores/">Conservation scores for alignments of 9 
        vertebrate genomes with Mouse</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>May 2004 (mm5)</h3>
<ul> 
  <li><a href="/goldenPath/mm5/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/mm5/chromosomes/">Data set by chromosome</a></li>        
  <li><a href="/goldenPath/mm5/database/">Annotation database</a></li>
  <li><a href="/goldenPath/mm5/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm5pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="mm5pairwise"> 
      <ul>
        <li><a href="/goldenPath/mm5/vsHg17/">Mouse/Human (hg17)</a></li>
        <li><a href="/goldenPath/mm5/vsRn3/">Mouse/Rat (rn3)</a></li>
        <li><a href="/goldenPath/mm5/vsGalGal2/">Mouse/Chicken (galGal2)</a></li> 
        <li><a href="/goldenPath/mm5/vsCanFam1/">Mouse/Dog (canFam1)</a></li>
        <li><a href="/goldenPath/mm5/vsMonDom1/">Mouse/Opossum (monDom1)</a></li>
        <li><a href="/goldenPath/mm5/vsTetNig1/">Mouse/Tetraodon (tetNig1)</a></li>
        <li><a href="/goldenPath/mm5/vsDanRer2/">Mouse/Zebrafish (danRer2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm5mult>Multiple
    alignments</a>
    <div class="collapse" id="mm5mult"> 
      <ul> 
        <li><a href="/goldenPath/mm5/multiz5way/">Multiple alignments of 4 vertebrate genomes with 
        Mouse</a></li>
        <li><a href="/goldenPath/mm5/phastCons/mzRn3Hg17Cf1Gg2/">Conservation scores for alignments 
        of 4 vertebrate genomes with Mouse</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Oct. 2003 (mm4)</h3>
<ul>
  <li><a href="/goldenPath/mm4/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/mm4/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/mm4/database/">Annotation database</a></li>
  <li><a href="/goldenPath/mm4/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="mm4pairwise"> 
      <ul>
        <li><a href="/goldenPath/mm4/vsHg16/">Mouse/Human (hg16)</a></li>
        <li><a href="/goldenPath/mm4/vsRn3/">Mouse/Rat (rn3)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Feb. 2003 (mm3)</h3> 
<ul>
  <li><a href="/goldenPath/mm3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/mm3/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/mm3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/mm3/liftOver/">LiftOver files</a></li>
  <li><a href="/goldenPath/mm3/mm3Hg15L/">Human Cons data</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#mm3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="mm3pairwise"> 
      <ul>
        <li><a href="/goldenPath/mm3/alignments/vsHg15">Mouse/Human (hg15)</a></li>
        <li><a href="/goldenPath/mm3/alignments/vsRn2">Mouse/Rat (rn2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Feb. 2002 (mm2)</h3> 
<ul>
  <li><a href="/goldenPath/mm2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/mm2/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/mm2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/mm2/liftOver/">LiftOver files</a></li>
</ul>
<h3>Nov. 2001 (mm1)</h3> 
<ul>
  <li><a href="/goldenPath/mm1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/mm1/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/mm1/database/">Annotation database</a></li>
</ul>

<!-- Mouse lemur Downloads -->
<a name="mouse_lemur"></a>
<h2>Mouse lemur genome</h2>
<h3>May 2015 (Mouse lemur/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=micMur2"
    target="_blank">micMur2</a>)</h3> 
<ul>
  <li><a href="/goldenPath/micMur2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/micMur2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/micMur2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#micMur2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="micMur2pairwise"> 
      <ul>
        <li><a href="/goldenPath/micMur2/vsHg38/">Mouse lemur/Human (hg38)</a></li>
        <li><a href="/goldenPath/micMur2/vsTarSyr2/">Mouse lemur/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2007 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=micMur1"
    target="_blank">micMur1</a>)</h3>
<ul>
  <li><a href="/goldenPath/micMur1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/micMur1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/micMur1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#micMur1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="micMur1pairwise"> 
      <ul>
        <li><a href="/goldenPath/micMur1/vsHg19/">Mouse lemur/Human (hg19)</a></li>
        <li><a href="/goldenPath/micMur1/vsGorGor3/">Mouse lemur/Gorilla (gorGor3)</a></li>
        <li><a href="/goldenPath/micMur1/vsCalJac3/">Mouse lemur/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/micMur1/vsMm10/">Mouse lemur/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/micMur1/vsTarSyr2/">Mouse lemur/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Naked mole-rat Downloads -->
<a name="naked_mole-rat"></a>
<h2>Naked mole-rat genome</h2>
<h3>Jan. 2012 (Broad HetGla_female_1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=hetGla2"
     target="_blank">hetGla2</a>)</h3> 
<ul>
  <li><a href="/goldenPath/hetGla2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/hetGla2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/hetGla2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hetGla2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="hetGla2pairwise"> 
    <ul>
      <li><a href="/goldenPath/hetGla2/vsMm10/">Naked mole-rat/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2011 (HetGla_1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=hetGla1"
target="_blank">hetGla1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/hetGla1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/hetGla1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/hetGla1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#hetGla1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="hetGla1pairwise"> 
      <ul>
        <li><a href="/goldenPath/hetGla1/vsMm10/">Naked mole-rat/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Nile tilapia Downloads -->
<a name="nile_tilapia"></a>
<h2>Nile tilapia genome</h2>
<h3>Jan. 2011 (Nile tilapia/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=oreNil2"
    target="_blank">oreNil2</a>)</h3> 
<ul>
  <li><a href="/goldenPath/oreNil2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/oreNil2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/oreNil2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#oreNil2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="oreNil2pairwise"> 
      <ul>
        <li><a href="/goldenPath/oreNil2/vsMm10/">Nile tilapia/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Opossum Downloads -->
<a name="opossum"></a>
<h2>Opossum genome</h2>
<h3>Oct. 2006 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=monDom5"
    target="_blank">monDom5</a>)</h3>
<ul> 
  <li><a href="/goldenPath/monDom5/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/monDom5/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/monDom5/database/">Annotation database</a></li>
  <li><a href="/goldenPath/monDom5/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#monDom5pairwise>Pairwise
    alignments (mammals)</a>
    <div class="collapse" id="monDom5pairwise"> 
      <ul> 
        <li><a href="/goldenPath/monDom5/vsHg19/">Opossum/Human (hg19)</a></li>
        <li><a href="/goldenPath/monDom5/vsHg18/">Opossum/Human (hg18)</a></li>
        <li><a href="/goldenPath/monDom5/vsPanTro3/">Opossum/Chimp (panTro3)</a></li>
        <li><a href="/goldenPath/monDom5/vsPanTro2/">Opossum/Chimp (panTro2)</a></li>
        <li><a href="/goldenPath/monDom5/vsBosTau7/">Opossum/Cow (bosTau7)</a></li>
        <li><a href="/goldenPath/monDom5/vsBosTau6/">Opossum/Cow (bosTau6)</a></li>
        <li><a href="/goldenPath/monDom5/vsBosTau4/">Opossum/Cow (bosTau4)</a></li>
        <li><a href="/goldenPath/monDom5/vsCanFam2/">Opossum/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/monDom5/vsLoxAfr3/">Opossum/Elephant (loxAfr3)</a></li>
        <li><a href="/goldenPath/monDom5/vsEquCab2/">Opossum/Horse (equCab2)</a></li>
        <li><a href="/goldenPath/monDom5/vsCalJac3/">Opossum/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/monDom5/vsMm10/">Opossum/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/monDom5/vsMm9/">Opossum/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/monDom5/vsPonAbe2/">Opossum/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/monDom5/vsAilMel1/">Opossum/Panda (ailMel1)</a></li>
        <li><a href="/goldenPath/monDom5/vsSusScr2/">Opossum/Pig (susScr2)</a></li>
        <li><a href="/goldenPath/monDom5/vsOrnAna1/">Opossum/Platypus (ornAna1)</a></li>
        <li><a href="/goldenPath/monDom5/vsOryCun2/">Opossum/Rabbit (oryCun2)</a></li>
        <li><a href="/goldenPath/monDom5/vsRn6/">Opossum/Rat (rn6)</a></li>
        <li><a href="/goldenPath/monDom5/vsRheMac3/">Opossum/Rhesus (rheMac3)</a></li>
        <li><a href="/goldenPath/monDom5/vsRheMac2/">Opossum/Rhesus (rheMac2)</a></li>
        <li><a href="/goldenPath/monDom5/vsCerSim1/">Opossum/White rhinoceros (cerSim1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#monDom5pairwise-vert>Pairwise
    alignments (other vertebrates)</a>
    <div class="collapse" id="monDom5pairwise-vert"> 
      <ul> 
        <li><a href="/goldenPath/monDom5/vsGalGal3/">Opossum/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/monDom5/vsPetMar2/">Opossum/Lamprey (petMar2)</a></li>
        <li><a href="/goldenPath/monDom5/vsMelGal1/">Opossum/Turkey (melGal1)</a></li>
        <li><a href="/goldenPath/monDom5/vsXenTro3/">Opossum/X. tropicalis (xenTro3)</a></li>
        <li><a href="/goldenPath/monDom5/vsXenTro2/">Opossum/X. tropicalis</EM> (xenTro2)</a></li>
        <li><a href="/goldenPath/monDom5/vsDanRer5/">Opossum/Zebrafish (danRer5)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#monDom5mult>Multiple
    alignments</a>
    <div class="collapse" id="monDom5mult"> 
      <ul> 
        <li><a href="/goldenPath/monDom5/multiz9way/">Multiple alignments of 8 vertebrate genomes 
        with Opossum</a></li>
        <li><a href="/goldenPath/monDom5/phastCons9way/">Conservation scores for alignments of 8 
        vertebrate genomes with Opossum</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jan. 2006 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=monDom4"
    target="_blank">monDom4</a>)</h3>
<ul> 
  <li><a href="/goldenPath/monDom4/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/monDom4/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/monDom4/database/">Annotation database</a></li>
  <li><a href="/goldenPath/monDom4/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#monDom4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="monDom4pairwise"> 
      <ul>
        <li><a href="/goldenPath/monDom4/vsHg18/">Opossum/Human (hg18)</a></li>
        <li><a href="/goldenPath/monDom4/vsGalGal3/">Opossum/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/monDom4/vsGalGal2/">Opossum/Chicken (galGal2)</a></li>
        <li><a href="/goldenPath/monDom4/vsPanTro2/">Opossum/Chimp (panTro2)</a></li>
        <li><a href="/goldenPath/monDom4/vsCavPor3/">Opossum/Guinea pig (cavPor3)</a></li>
        <li><a href="/goldenPath/monDom4/vsCalJac1/">Opossum/Marmoset (calJac1)</a></li>
        <li><a href="/goldenPath/monDom4/vsMm9/">Opossum/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/monDom4/vsMm8/">Opossum/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/monDom4/vsPonAbe2/">Opossum/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/monDom4/vsOrnAna1/">Opossum/Platypus (ornAna1)</a></li>
        <li><a href="/goldenPath/monDom4/vsRn4/">Opossum/Rat (rn4)</a></li>
        <li><a href="/goldenPath/monDom4/vsXenTro2/">Opossum/X. tropicalis (xenTro2)</a></li>
        <li><a href="/goldenPath/monDom4/vsDanRer4/">Opossum/Zebrafish (danRer4)</a></li>
        <li><a href="/goldenPath/monDom4/vsDanRer3/">Opossum/Zebrafish (danRer3)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#monDom4mult>Multiple
    alignments</a>
    <div class="collapse" id="monDom4mult"> 
      <ul> 
        <li><a href="/goldenPath/monDom4/multiz7way/">Multiple alignments of 6 vertebrate genomes 
        with Opossum</a></li>
        <li><a href="/goldenPath/monDom4/phastCons7Scores/">Conservation scores for alignments of 6 
        vertebrate genomes with Opossum</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jun. 2005 (Broad/monDom2)</h3>
<ul> 
  <li><a href="/goldenPath/monDom2/bigZips/">Full data set (.2bit format)</a></li>
</ul> 
<h3>Oct. 2004 (Broad prelim/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=monDom1"
target="_blank">monDom1</a>)</h3>
<ul> 
  <li><a href="/goldenPath/monDom1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/monDom1/database/">Annotation database</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#monDom1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="monDom1pairwise"> 
      <ul>
        <li><a href="/goldenPath/monDom1/vsHg17/">Opossum/Human (hg17)</a></li>
        <li><a href="/goldenPath/monDom1/vsGalGal2/">Opossum/Chicken (galGal2)</a></li>
        <li><a href="/goldenPath/monDom1/vsMm6/">Opossum/Mouse (mm6)</a></li>
        <li><a href="/goldenPath/monDom1/vsDanRer2/">Opossum/Zebrafish (vsDanRer2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Orangutan Downloads -->
<a name="orangutan"></a>
<h2>Orangutan genome</h2>

<h3>Jan. 2018 (Susie_PABv2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ponAbe3"
target="_blank">ponAbe3</a>)</h3>
<ul>
  <li><a href="/goldenPath/ponAbe3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ponAbe3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/ponAbe3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ponAbe3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ponAbe3pairwise">
      <ul>
        <li><a href="/goldenPath/ponAbe3/vsHg38/">Orangutan/Human (hg38)</a></li>
        <li><a href="/goldenPath/ponAbe3/vsMm10/">Orangutan/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li>
</ul>



<h3>Jul. 2007 (WUGSC 2.0.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ponAbe2"
target="_blank">ponAbe2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/ponAbe2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ponAbe2/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/ponAbe2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/ponAbe2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ponAbe2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ponAbe2pairwise"> 
      <ul>
        <li><a href="/goldenPath/ponAbe2/vsHg19/">Orangutan/Human (hg19)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsHg18/">Orangutan/Human (hg18)</a></li>
<!--
        <li><a href="/goldenPath/ponAbe2/vsGalGal3/">Orangutan/Chicken (galGal3)</a></li>
-->
        <li><a href="/goldenPath/ponAbe2/vsPanTro3/">Orangutan/Chimp (panTro3)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsPanTro2/">Orangutan/Chimp (panTro2)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsGorGor3/">Orangutan/Gorilla (gorGor3)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsCalJac3/">Orangutan/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsCalJac1/">Orangutan/Marmoset (calJac1)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsMm10/">Orangutan/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsMm9/">Orangutan/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsMonDom5/">Orangutan/Opposum (monDom5)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsMonDom4/">Orangutan/Opposum (monDom4)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsOrnAna1/">Orangutan/Platypus (ornAna1)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsRheMac3/">Orangutan/Rhesus (rheMac3)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsRheMac2/">Orangutan/Rhesus (rheMac2)</a></li>
        <li><a href="/goldenPath/ponAbe2/vsTarSyr2/">Orangutan/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ponAbe2mult>Multiple
    alignments</a>
    <div class="collapse" id="ponAbe2mult"> 
      <ul> 
        <li><a href="/goldenPath/ponAbe2/multiz8way/">Multiple alignments of 7 vertebrate genomes 
        with Orangutan</a></li>
        <li><a href="/goldenPath/ponAbe2/phastCons8way/">Conservation scores for alignments of 7 
        vertebrate genomes with Orangutan</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Painted turtle Downloads -->
<a name="painted_turtle"></a>
<h2>Painted turtle genome</h2>
<h3>Dec. 2011 (v3.0.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=chrPic1"
target="_blank">chrPic1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/chrPic1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/chrPic1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/chrPic1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#chrPic1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="chrPic1pairwise"> 
      <ul>
        <li><a href="/goldenPath/chrPic1/vsMm10/">Painted turtle/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Panda Downloads -->
<a name="panda"></a>
<h2>Panda genome</h2>
<h3>Dec. 2009 (BGI-Shenzhen 1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ailMel1" 
    target="_blank">ailMel1</a>)</h3>
<ul> 
  <li><a href="/goldenPath/ailMel1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ailMel1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/ailMel1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ailMel1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ailMel1pairwise"> 
      <ul>
        <li><a href="/goldenPath/ailMel1/vsHg19/">Panda/Human (hg19)</a></li>
        <li><a href="/goldenPath/ailMel1/vsCanFam2/">Panda/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/ailMel1/vsMm10/">Panda/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/ailMel1/vsMm9/">Panda/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/ailMel1/vsMonDom5/">Panda/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/ailMel1/vsRn6/">Panda/Rat (rn6)</a></li>
        <li><a href="/goldenPath/ailMel1/vsRn4/">Panda/Rat (rn4)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Pig Downloads -->
<a name="pig"></a>
<h2>Pig genome</h2>
<h3>Feb. 2017 (SGSC Sscrofa11.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=susScr11" 
    target="_blank">susScr11</a>)</h3>
<ul>
  <li><a href="/goldenPath/susScr11/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/susScr11/database/">Annotation database</a></li>
  <li><a href="/goldenPath/susScr11/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#susScr11pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="susScr11pairwise"> 
      <ul>
        <li><a href="/goldenPath/susScr11/vsHg38/">Pig/Human (hg38)</a></li>
        <li><a href="/goldenPath/susScr11/vsMm10/">Pig/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Aug. 2011 (SGSC Sscrofa10.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=susScr3" 
    target="_blank">susScr3</a>)</h3>
<ul>
  <li><a href="/goldenPath/susScr3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/susScr3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/susScr3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#susScr3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="susScr3pairwise"> 
      <ul>
        <li><a href="/goldenPath/susScr3/vsMm10/">Pig/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Nov. 2009 (SGSC Sscrofa9.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=susScr2"
   target="_blank">susScr2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/susScr2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/susScr2/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/susScr2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/susScr2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#susScr2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="susScr2pairwise"> 
      <ul>
        <li><a href="/goldenPath/susScr2/vsHg19/">Pig/Human (hg19)</a></li>
        <li><a href="/goldenPath/susScr2/vsBosTau7/">Pig/Cow (bosTau7)</a></li>
        <li><a href="/goldenPath/susScr2/vsBosTau6/">Pig/Cow (bosTau6)</a></li>
        <li><a href="/goldenPath/susScr2/vsBosTau4/">Pig/Cow (bosTau4)</a></li>
        <li><a href="/goldenPath/susScr2/vsMm9/">Pig/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/susScr2/vsMonDom5/">Pig/Opossum (monDom5)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Pika Downloads -->
<a name="pika"></a>
<h2>Pika genome</h2>
<h3>May 2012 (OchPri3.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ochPri3"
    target="_blank">ochPri3</a>)</h3> 
<ul>
  <li><a href="/goldenPath/ochPri3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ochPri3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/ochPri3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ochPri3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ochPri3pairwise"> 
      <uL> 
        <li><a href="/goldenPath/ochPri3/vsHg19/">Pika/Human (hg19)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2008 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ochPri2"
    target="_blank">ochPri2</a>)</h3> 
<ul>
  <li><a href="/goldenPath/ochPri2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ochPri2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/ochPri2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ochPri2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ochPri2pairwise"> 
      <ul>
        <li><a href="/goldenPath/ochPri2/vsMm10/">Pika/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Platypus Downloads -->
<a name="platypus"></a>
<h2>Platypus genome</h2>
<h3>Feb. 2007 (ASM227v2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ornAna2"
    target="_blank">ornAna2</a>)</h3> 
<ul>
  <li><a href="/goldenPath/ornAna2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ornAna2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/ornAna2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ornAna2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ornAna2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/ornAna2/vsHg38/">Platypus/Human (hg38)</a></li>
        <li><a href="/goldenPath/ornAna2/vsRn6/">Platypus/Rat (rn6)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Mar. 2007 (WUGSC 5.0.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ornAna1"
target="_blank">ornAna1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/ornAna1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ornAna1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/ornAna1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ornAna1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ornAna1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/ornAna1/vsHg19/">Platypus/Human (hg19)</a></li>
        <li><a href="/goldenPath/ornAna1/vsHg18/">Platypus/Human (hg18)</a></li>
        <li><a href="/goldenPath/ornAna1/vsGalGal3/">Platypus/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/ornAna1/vsBosTau7/">Platypus/Cow (bosTau7)</a></li>
        <li><a href="/goldenPath/ornAna1/vsBosTau6/">Platypus/Cow (bosTau6)</a></li>
        <li><a href="/goldenPath/ornAna1/vsBosTau4/">Platypus/Cow (bosTau4)</a></li>
        <li><a href="/goldenPath/ornAna1/vsEquCab2/">Platypus/Horse (equCab2)</a></li>
        <li><a href="/goldenPath/ornAna1/vsAnoCar2/">Platypus/Lizard (anoCar2)</a></li>
        <li><a href="/goldenPath/ornAna1/vsAnoCar1/">Platypus/Lizard (anoCar1)</a></li>
        <li><a href="/goldenPath/ornAna1/vsCalJac1/">Platypus/Marmoset (calJac1)</a></li>
        <li><a href="/goldenPath/ornAna1/vsMm9/">Platypus/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/ornAna1/vsMm8/">Platypus/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/ornAna1/vsMonDom5/">Platypus/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/ornAna1/vsMonDom4/">Platypus/Opossum (monDom4)</a></li>
        <li><a href="/goldenPath/ornAna1/vsPonAbe2/">Platypus/Orangutan (ponAbe2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ornAna1mult>Multiple
    alignments</a>
    <div class="collapse" id="ornAna1mult"> 
      <ul> 
        <li><a href="/goldenPath/ornAna1/multiz6way/">Multiple alignments of 5 vertebrate genomes 
        with Platypus</a></li>
        <li><a href="/goldenPath/ornAna1/phastCons6way/">Conservation scores for alignments of 5 
       vertebrate genomes with Platypus</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Proboscis monkey Downloads -->
<a name="proboscis_monkey"></a>
<h2>Proboscis monkey genome</h2>
<h3>Nov 2014 (Charlie1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=nasLar1"
    target="_blank">nasLar1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/nasLar1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/nasLar1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/nasLar1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#nasLar1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="nasLar1pairwise"> 
      <ul> 
       <li><a href="/goldenPath/nasLar1/vsHg38/">Proboscis Monkey/Human (hg38)</a></li>
       <li><a href="/goldenPath/nasLar1/vsTarSyr2/">Proboscis Monkey/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Rabbit Downloads -->
<a name="rabbit"></a>
<h2>Rabbit genome</h2>
<h3>Apr. 2009 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=oryCun2"
    target="_blank">oryCun2</a>)</h3>
<ul> 
  <li><a href="/goldenPath/oryCun2/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/oryCun2/database/">Annotation database</a></lia></li> 
  <li><a href="/goldenPath/oryCun2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#oryCun2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="oryCun2pairwise"> 
      <ul>
        <li><a href="/goldenPath/oryCun2/vsHg38/">Rabbit/Human (hg38)</a></li> 
        <li><a href="/goldenPath/oryCun2/vsHg19/">Rabbit/Human (hg19)</a></li> 
        <li><a href="/goldenPath/oryCun2/vsMm9/">Rabbit/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/oryCun2/vsCavPor3/">Rabbit/Guinea Pig (cavPor3)</a></li>
        <li><a href="/goldenPath/oryCun2/vsMonDom5/">Rabbit/Opossum (monDom5)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>May 2005 (oryCun1)</h3>
<ul> 
  <li><a href="/goldenPath/oryCun1/bigZips/">Full data set</a></li>
</ul>

<!-- Rat Downloads -->
<a name="rat"></a>
<h2>Rat genome</h2>
<h3>Jul. 2014 (RGSC 6.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=rn6"
target="_blank">rn6</a>)</h3>
<ul>
  <li><a href="/goldenPath/rn6/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/rn6/database/">Annotation database</a></li>
  <li><a href="/goldenPath/rn6/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rn6pairwise>Pairwise
    alignments (mammals)</a>
    <div class="collapse" id="rn6pairwise"> 
      <ul> 
        <li><a href="/goldenPath/rn6/vsHg38/">Rat/Human (hg38)</a></li>
        <li><a href="/goldenPath/rn6/vsHg19/">Rat/Human (hg19)</a></li>
        <li><a href="/goldenPath/rn6/vsFelCat8/">Rat/Cat (felCat8)</a></li>
        <li><a href="/goldenPath/rn6/vsPanTro5/">Rat/Chimp (panTro5)</a></li>
        <li><a href="/goldenPath/rn6/vsBosTau8/">Rat/Cow (bosTau8)</a></li>
        <li><a href="/goldenPath/rn6/vsCanFam3/">Rat/Dog (canFam3)</a></li>
        <li><a href="/goldenPath/rn6/vsCavPor3/">Rat/Guinea Pig (cavPor3)</a></li>
        <li><a href="/goldenPath/rn6/vsMm10/">Rat/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/rn6/vsMonDom5/">Rat/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/rn6/vsAilMel1/">Rat/Panda (ailMel1)</a></li>
        <li><a href="/goldenPath/rn6/vsOrnAna2/">Rat/Platypus (ornAna2)</a></li>
        <li><a href="/goldenPath/rn6/vsOryCun2/">Rat/Rabbit (oryCun2)</a></li>
        <li><a href="/goldenPath/rn6/vsRheMac8/">Rat/Rhesus (rheMac8)</a></li>
        <li><a href="/goldenPath/rn6/vsTarSyr2/">Rat/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rn6pairwise-vert>Pairwise
    alignments (other vertebrates)</a>
    <div class="collapse" id="rn6pairwise-vert"> 
      <ul> 
        <li><a href="/goldenPath/rn6/vsGalGal5/">Rat/Chicken (galGal5)</a></li>
        <li><a href="/goldenPath/rn6/vsCalMil1/">Rat/Elephant Shark (calMil1)</a></li>
        <li><a href="/goldenPath/rn6/vsMelGal5/">Rat/Turkey (melGal5)</a></li>
        <li><a href="/goldenPath/rn6/vsXenTro7/">Rat/X. tropicalis (xenTro7)</a></li>
        <li><a href="/goldenPath/rn6/vsDanRer10/">Rat/Zebrafish (danRer10)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rn6mult>Multiple
    alignments</a>
    <div class="collapse" id="rn6mult"> 
      <ul> 
        <li><a href="/goldenPath/rn6/multiz20way">Multiple alignments of 19 vertebrate genomes 
        with Rat</a></li>
        <li><a href="/goldenPath/rn6/phastCons20way">Conservation scores for alignments of 19 
        vertebrate genomes with Rat</a></li>
        <li><a href="/goldenPath/rn6/phyloP20way">Basewise conservation scores (phyloP) of 19 
        vertebrate genomes with Rat</a></li>
        <li><a href="/goldenPath/rn6/multiz20way/alignments">FASTA alignments of 19 vertebrate 
        genomes with Rat</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Mar. 2012 (RGSC 5.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=rn5" 
    target="_blank">rn5</a>)</h3> 
<ul>
  <li><a href="/goldenPath/rn5/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/rn5/database/">Annotation database</a></li>
  <li><a href="/goldenPath/rn5/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rn5pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="rn5pairwise"> 
      <ul> 
        <li><a href="/goldenPath/rn5/vsHg19/">Rat/Human (hg19)</a></li>
        <li><a href="/goldenPath/rn5/vsPanTro4/">Rat/Chimp (panTro4)</a></li>
        <li><a href="/goldenPath/rn5/vsMm10/">Rat/Mouse (mm10)</a></li>   
        <li><a href="/goldenPath/rn5/vsMm9/">Rat/Mouse (mm9)</a></li>   
        <li><a href="/goldenPath/rn5/vsRheMac3/">Rat/Rhesus (rheMac3)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rn5mult>Multiple
    alignments</a>
    <div class="collapse" id="rn5mult"> 
      <ul> 
        <li><a href="/goldenPath/rn5/multiz13way/">Multiple alignments of 12 vertebrate genomes 
        with Rat</a></li>
        <li><a href="/goldenPath/rn5/phastCons13way/">Conservation scores for alignments of 12 
        vertebrate genomes with Rat</a></li>
        <li><a href="/goldenPath/rn5/phyloP13way/">Basewise conservation scores (phyloP) of 12 
        vertebrate genomes with Rat</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Nov. 2004 (Baylor 3.4/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=rn4"
    target="_blank">rn4</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/rn4/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/rn4/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/rn4/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/rn4/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rn4pairwise>Pairwise
    alignments (mammals)</a>
    <div class="collapse" id="rn4pairwise"> 
      <ul> 
        <li><a href="/goldenPath/rn4/vsHg19/">Rat/Human (hg19)</a></li>
        <li><a href="/goldenPath/rn4/vsHg18/">Rat/Human (hg18)</a></li>
        <li><a href="/goldenPath/rn4/vsPanTro2/">Rat/Chimp (panTro2)</a></li>
        <li><a href="/goldenPath/rn4/vsBosTau7/">Rat/Cow (bosTau7)</a></li>
        <li><a href="/goldenPath/rn4/vsBosTau6/">Rat/Cow (bosTau6)</a></li>
        <li><a href="/goldenPath/rn4/vsBosTau3/">Rat/Cow (bosTau3)</a></li>
        <li><a href="/goldenPath/rn4/vsBosTau2/">Rat/Cow (bosTau2)</a></li>
        <li><a href="/goldenPath/rn4/vsCanFam2/">Rat/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/rn4/vsCavPor3/">Rat/Guinea pig (cavPor3)</a></li>
        <li><a href="/goldenPath/rn4/vsEquCab1/">Rat/Horse (equCab1)</a></li>
        <li><a href="/goldenPath/rn4/vsMm9/">Rat/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/rn4/vsMm8/">Rat/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/rn4/vsMonDom4/">Rat/Opossum (monDom4)</a></li>
        <li><a href="/goldenPath/rn4/vsAilMel1/">Rat/Panda (ailMel1)</a></li>
        <li><a href="/goldenPath/rn4/vsRheMac2/">Rat/Rhesus (rheMac2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rn4pairwise-vert>Pairwise
    alignments (other vertebrates)</a>
    <div class="collapse" id="rn4pairwise-vert"> 
      <ul> 
        <li><a href="/goldenPath/rn4/vsGalGal3/">Rat/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/rn4/vsGalGal2/">Rat/Chicken (galGal2)</a></li>
        <li><a href="/goldenPath/rn4/vsMelGal1/">Rat/Turkey (melGal1)</a></li>
        <li><a href="/goldenPath/rn4/vsXenTro3/">Rat/X. tropicalis (xenTro3)</a></li>
        <li><a href="/goldenPath/rn4/vsXenTro2/">Rat/X. tropicalis (xenTro2)</a></li>
        <li><a href="/goldenPath/rn4/vsXenTro1/">Rat/X. tropicalis (xenTro1)</a></li>
        <li><a href="/goldenPath/rn4/vsDanRer4/">Rat/Zebrafish (danRer4)</a></li>
        <li><a href="/goldenPath/rn4/vsDanRer3/">Rat/Zebrafish (danRer3)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rn4mult>Multiple
    alignments</a>
    <div class="collapse" id="rn4mult"> 
      <ul> 
        <li><a href="/goldenPath/rn4/multiz9way/">Multiple alignments of 8 vertebrate genomes with 
        Rat</a></li> 
        <li><a href="/goldenPath/rn4/phastCons9way/">Conservation scores for alignments of 8 
        vertebrate genomes with Rat</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jun. 2003 (Baylor 3.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=rn3"
target="_blank">rn3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/rnJun2003/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/rnJun2003/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/rnJun2003/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/rn3/liftOver/">LiftOver files</a></li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rn3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="rn3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/rnJun2003/vsHg18/">Rat/Human (hg18)</a></li>
        <li><a href="/goldenPath/rnJun2003/vsHg17/">Rat/Human (hg17)</a></li>
        <li><a href="/goldenPath/rnJun2003/vsCanFam2/">Rat/Dog (canFam2)</a></li>
        <li><a href="/goldenPath/rnJun2003/vsMm7/">Rat/Mouse (mm7)</a></li>
        <li><a href="/goldenPath/rnJun2003/vsMm6/">Rat/Mouse (mm6)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jan. 2003 (rn2)</h3> 
<ul> 
  <li><a href="/goldenPath/rnJan2003/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/rnJan2003/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/rnJan2003/database/">Annotation database</a></li>
</ul>
<h3>Nov. 2002 (rn1)</h3> 
<ul>
  <li><a href="/goldenPath/rn1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/rn1/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/rn1/database/">Annotation database</a></li>
</ul>

<!-- Rhesus Downloads -->
<a name="rhesus"></a>
<h2>Rhesus genome</h2>
<h3>Nov. 2015 (BCM Mmul_8.0.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=rheMac8"
    target="_blank">rheMac8</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/rheMac8/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/rheMac8/database/">Annotation database</a></li>
  <li><a href="/goldenPath/rheMac8/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rheMac8pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="rheMac8pairwise"> 
      <ul> 
        <li><a href="/goldenPath/rheMac8/vsPanTro5/">Rhesus/Chimp (panTro5)</a></li>
        <li><a href="/goldenPath/rheMac8/vsHg38/">Rhesus/Human (hg38)</a></li>
        <li><a href="/goldenPath/rheMac8/vsChlSab2/">Rhesus/Green Monkey (chlSab2)</a></li>
        <li><a href="/goldenPath/rheMac8/vsCalJac3/">Rhesus/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/rheMac8/vsMm10/">Rhesus/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/rheMac8/vsRn6/">Rhesus/Rat (rn6)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Oct. 2010 (BGI CR_1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=rheMac3"
    target="_blank">rheMac3</a>)</h3> 
<ul>
  <li><a href="/goldenPath/rheMac3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/rheMac3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/rheMac3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rheMac3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="rheMac3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/rheMac3/vsHg38/">Rhesus/Human (hg38)</a></li>
        <li><a href="/goldenPath/rheMac3/vsHg19/">Rhesus/Human (hg19)</a></li>
        <li><a href="/goldenPath/rheMac3/vsPanTro4/">Rhesus/Chimp (panTro4)</a></li>
        <li><a href="/goldenPath/rheMac3/vsGorGor3/">Rhesus/Gorilla (gorGor3)</a></li>
        <li><a href="/goldenPath/rheMac3/vsCalJac3/">Rhesus/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/rheMac3/vsMm10/">Rhesus/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/rheMac3/vsMonDom5/">Rhesus/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/rheMac3/vsPonAbe2/">Rhesus/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/rheMac3/vsRn5/">Rhesus/Rat (rn5)</a></li>
        <li><a href="/goldenPath/rheMac3/vsTarSyr2/">Rhesus/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jan. 2006 (MGSC Merged 1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=rheMac2"
    target="_blank">rheMac2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/rheMac2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/rheMac2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/rheMac2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rheMac2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="rheMac2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/rheMac2/vsHg19/">Rhesus/Human (hg19)</a></li>
        <li><a href="/goldenPath/rheMac2/vsHg18/">Rhesus/Human (hg18)</a></li>
        <li><a href="/goldenPath/rheMac2/vsPanTro3/">Rhesus/Chimp (panTro3)</a></li>
        <li><a href="/goldenPath/rheMac2/vsPanTro2/">Rhesus/Chimp (panTro2)</a></li>
        <li><a href="/goldenPath/rheMac2/vsGorGor3/">Rhesus/Gorilla (gorGor3)</a></li>
        <li><a href="/goldenPath/rheMac2/vsPonAbe2/">Rhesus/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/rheMac2/vsCalJac3/">Rhesus/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/rheMac2/vsCalJac1/">Rhesus/Marmoset (calJac1)</a></li>
        <li><a href="/goldenPath/rheMac2/vsMm9/">Rhesus/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/rheMac2/vsMm8/">Rhesus/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/rheMac2/vsRn4/">Rhesus/Rat (rn4)</a></li>
        <li><a href="/goldenPath/rheMac2/vsMonDom5/">Rhesus/Opossum (monDom5)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jan. 2005 (Mmul_0.1/rheMac1)</h3> 
<ul> 
  <li><a href="/goldenPath/rheMac1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/rheMac1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/rheMac1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#rheMac1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="rheMac1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/rheMac1/vsMm7/">Rhesus/Mouse (mm7)</a></li> 
      </ul>
    </div>
  </li> 
</ul>

<!-- Rock hyrax Downloads -->
<a name="rock_hyrax"></a>
<h2>Rock hyrax genome</h2>
<h3>Jul. 2008 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=proCap1"
    target="_blank">proCap1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/proCap1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/proCap1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/proCap1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#proCap1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="proCap1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/proCap1/vsMm10/">Rock hyrax/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Sheep Downloads -->
<a name="sheep"></a>
<h2>Sheep genome</h2>
<h3>Nov. 2015 (ISGC Oar_v4.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=oviAri4"
    target="_blank">oviAri4</a>)</h3>
<ul>
  <li><a href="/goldenPath/oviAri4/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/oviAri4/database/">Annotation database</a></li>
  <li><a href="/goldenPath/oviAri4/liftOver/">LiftOver files</a> </li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#oviAri4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="oviAri4pairwise">
      <ul>
        <li><a href="/goldenPath/oviAri4/vsHg38/">Sheep/Human (hg38)</a>
        <li><a href="/goldenPath/oviAri4/vsMm10/">Sheep/Mouse (mm10)</a>
      </ul>
    </div>
  </li>
</ul>
<h3>Aug. 2012 (ISGC Oar_v3.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=oviAri3"
    target="_blank">oviAri3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/oviAri3/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/oviAri3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/oviAri3/liftOver/">LiftOver files</a> </li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#oviAri3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="oviAri3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/oviAri3/vsHg19/">Sheep/Human (hg19)</a>
      </ul>
    </div>
  </li> 
</ul>
<h3>Feb. 2010 (ISGC Ovis_aries_1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=oviAri1" 
    target="_blank">oviAri1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/oviAri1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/oviAri1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/oviAri1/liftOver/">LiftOver files</a> </li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#oviAri1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="oviAri1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/oviAri1/vsBosTau4/">Sheep/Cow (bosTau4)</a></li> 
        <li><a href="/goldenPath/oviAri1/vsEquCab2/">Sheep/Horse (equCab2)</a></li>
        <li><a href="/goldenPath/oviAri1/vsHg19/">Sheep/Human (hg19)</a></li> 
        <li><a href="/goldenPath/oviAri1/vsMm10/">Sheep/Mouse (mm10)</a></li> 
        <li><a href="/goldenPath/oviAri1/vsMm9/">Sheep/Mouse (mm9)</a></li> 
        <li><a href="/goldenPath/oviAri1/vsMonDom5/">Sheep/Opossum (monDom5)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Shrew Downloads -->
<a name="shrew"></a>
<h2>Shrew genome</h2>
<h3>Aug. 2008 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=sorAra2"
    target="_blank">sorAra2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/sorAra2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/sorAra2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/sorAra2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#sorAra2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="sorAra2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/sorAra2/vsHg19/">Shrew/Human (hg19)</a></li>
        <li><a href="/goldenPath/sorAra2/vsMm10/">Shrew/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jun. 2006 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=sorAra1"
    target="_blank">sorAra1</a>)</h3>
<ul>
  <li><a href="/goldenPath/sorAra1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/sorAra1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/sorAra1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#sorAra1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="sorAra1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/sorAra1/vsMm10/">Shrew/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Sloth Downloads -->
<a name="sloth"></a>
<h2>Sloth genome</h2>
<h3>Jul. 2008 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=choHof1" 
    target="_blank">choHof1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/choHof1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/choHof1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/choHof1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#choHof1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="choHof1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/choHof1/vsMm10/">Sloth/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Squirrel Downloads -->
<a name="squirrel"></a>
<h2>Squirrel genome</h2>
<h3>Nov. 2011 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=speTri2" 
target="_blank">speTri2</a>)</h3> 
<ul>
  <li><a href="/goldenPath/speTri2/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/speTri2/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/speTri2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#speTri2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="speTri2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/speTri2/vsMm10/">Squirrel/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Squirrel monkey Downloads -->
<a name="squirrel_monkey"></a>
<h2>Squirrel monkey genome</h2>
<h3>Oct. 2011 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=saiBol1"
    target="_blank">saiBol1</a>)</h3>
<ul>
  <li><a href="/goldenPath/saiBol1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/saiBol1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/saiBol1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#saiBol1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="saiBol1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/saiBol1/vsMm10/">Squirrel monkey/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/saiBol1/vsTarSyr2/">Squirrel monkey/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Stickleback Downloads -->
<a name="stickleback"></a>
<h2>Stickleback genome</h2>
<h3>Feb. 2006 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=gasAcu1" 
    target="_blank">gasAcu1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/gasAcu1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/gasAcu1/chromosomes/">Data set by chromosome</a></li> 
  <li><a href="/goldenPath/gasAcu1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/gasAcu1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#gasAcu1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="gasAcu1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/gasAcu1/vsHg19/">Stickleback/Human (hg19)</a></li>
        <li><a href="/goldenPath/gasAcu1/vsHg18/">Stickleback/Human (hg18)</a></li> 
        <li><a href="/goldenPath/gasAcu1/vsGalGal3/">Stickleback/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/gasAcu1/vsFr3/">Stickleback/Fugu (fr3)</a></li> 
        <li><a href="/goldenPath/gasAcu1/vsFr2/">Stickleback/Fugu (fr2)</a></li> 
        <li><a href="/goldenPath/gasAcu1/vsFr1/">Stickleback/Fugu (fr1)</a></li>
        <li><a href="/goldenPath/gasAcu1/vsAnoCar2/">Stickleback/Lizard (anoCar2)</a></li>
        <li><a href="/goldenPath/gasAcu1/vsAnoCar1/">Stickleback/Lizard (anoCar1)</a></li>
        <li><a href="/goldenPath/gasAcu1/vsOryLat2/">Stickleback/Medaka (oryLat2)</a></li> 
        <li><a href="/goldenPath/gasAcu1/vsMm10/">Stickleback/Mouse (mm10)</a></li> 
        <li><a href="/goldenPath/gasAcu1/vsMm9/">Stickleback/Mouse (mm9)</a></li> 
        <li><a href="/goldenPath/gasAcu1/vsMm8/">Stickleback/Mouse (mm8)</a></li> 
        <li><a href="/goldenPath/gasAcu1/vsTetNig2/">Stickleback/Tetraodon (tetNig2)</a></li> 
        <li><a href="/goldenPath/gasAcu1/vsTetNig1/">Stickleback/Tetraodon (tetNig1)</a></li> 
        <li><a href="/goldenPath/gasAcu1/vsDanRer7/">Stickleback/Zebrafish (danRer7)</a></li>
        <li><a href="/goldenPath/gasAcu1/vsDanRer6/">Stickleback/Zebrafish (danRer6)</a></li>
        <li><a href="/goldenPath/gasAcu1/vsDanRer4/">Stickleback/Zebrafish (danRer4)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#gasAcu1mult>Multiple
    alignments</a>
    <div class="collapse" id="gasAcu1mult"> 
      <ul> 
        <li><a href="/goldenPath/gasAcu1/multiz8way/">Multiple alignments of 8 vertebrate genomes 
        with Stickleback</a></li> 
        <li><a href="/goldenPath/gasAcu1/phastCons8Scores/">Conservation scores for alignments of 8 
        vertebrate genomes with Stickleback</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Tarsier Downloads -->
<a name="tarsier"></a>
<h2>Tarsier genome</h2>
<h3>Sep. 2013 (Tarsius_syrichta-2.0.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=tarSyr2"
    target="_blank">tarSyr2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/tarSyr2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/tarSyr2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/tarSyr2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#tarSyr2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="tarSyr2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/tarSyr2/vsHg38/">Tarsier/Human (hg38)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsPapAnu2/">Tarsier/Baboon (papAnu2)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsPanPan1/">Tarsier/Bonobo (panPan1)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsOtoGar3/">Tarsier/Bushbaby (otoGar3)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsPanTro4/">Tarsier/Chimp (panTro4)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsMacFas5/">Tarsier/Crab-eating macaque (macFas5)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsCanFam3/">Tarsier/Dog (canFam3)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsNomLeu3/">Tarsier/Gibbon (nomLeu3)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsGorGor3/">Tarsier/Gorilla (gorGor3)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsChlSab2/">Tarsier/Green monkey (chlSab2)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsCalJac3/">Tarsier/Marmoset (calJac3)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsMm10/">Tarsier/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsMicMur2/">Tarsier/Mouse lemur (micMur2)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsPonAbe2/">Tarsier/Orangutan (ponAbe2)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsNasLar1/">Tarsier/Proboscis monkey (nasLar1)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsRn6/">Tarsier/Rat (rn6)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsRheMac3/">Tarsier/Rhesus (rheMac3)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsSaiBol1/">Tarsier/Squirrel monkey (saiBol1)</a></li>
        <li><a href="/goldenPath/tarSyr2/vsTupBel1/">Tarsier/Tree shrew (tupBel1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#tarSyr2mult>Multiple
    alignments</a>
    <div class="collapse" id="tarSyr2mult"> 
      <ul> 
        <li><a href="/goldenPath/tarSyr2/multiz20way/">Multiple alignments of 19 mammalian (16 
        primate) genomes with Tariser</a>
        <li><a href="/goldenPath/tarSyr2/phastCons20way/">Conservation scores for alignments of 19 
        mammalian (16 primate) genomes with Tarsier</a>
        <li><a href="/goldenPath/tarSyr2/phyloP20way/">Basewise conservation scores (phyloP) of 19 
        mammalian (16 primate) genomes with Tarsier</a>
        <li><a href="/goldenPath/tarSyr2/multiz20way/alignments/">FASTA alignments of 19 mammalian 
        (16 primate) genomes with Tarsier for CDS regions</a>
      </ul>
    </div>
  </li> 
</ul>
<h3>Aug. 2008 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=tarSyr1"
target="_blank">tarSyr1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/tarSyr1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/tarSyr1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/tarSyr1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#tarSyr1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="tarSyr1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/tarSyr1/vsGorGor3/">Tarsier/Gorilla (gorGor3)</a></li>
        <li><a href="/goldenPath/tarSyr1/vsMm10/">Tarsier/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Tasmanian devil Downloads -->
<a name="tasmanian_devil"></a>
<h2>Tasmanian devil genome</h2>
<h3>Feb. 2011 (WTSI Devil_ref v7.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=sarHar1" 
target="_blank">sarHar1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/sarHar1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/sarHar1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/sarHar1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#sarHar1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="sarHar1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/sarHar1/vsHg19/">Tasmanian devil/Human (hg19)</a></li>
        <li><a href="/goldenPath/sarHar1/vsMm10/">Tasmanian devil/Mouse (mm10)</a></li> 
      </ul>
    </div>
  </li> 
</ul>

<!-- Tenrec Downloads -->
<a name="tenrec"></a>
<h2>Tenrec genome</h2>
<h3>Nov. 2012 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=echTel2"
    target="_blank">echTel2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/echTel2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/echTel2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/echTel2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#echTel2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="echTel2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/echTel2/vsHg19/">Tenrec/Human (hg19)</a></li>
        <li><a href="/goldenPath/echTel2/vsMm10/">Tenrec/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2005 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=echTel1" 
target="_blank">echTel1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/echTel1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/echTel1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/echTel1/liftOver/">LiftOver files</a></lia></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#echTel1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="echTel1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/echTel1/vsHg19/">Tenrec/Human (hg19)</a></li>
        <li><a href="/goldenPath/echTel1/vsHg18/">Tenrec/Human (hg18)</a></li>
        <li><a href="/goldenPath/echTel1/vsMm10/">Tenrec/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/echTel1/vsMm9/">Tenrec/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/echTel1/vsMm7/">Tenrec/Mouse (mm7)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Tetraodon Downloads -->
<a name="tetraodon"></a>
<h2>Tetraodon genome</h2>
<h3>Mar. 2010 (Genoscope 8.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=tetNig2"
target="_blank">tetNig2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/tetNig2/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/tetNig2/chromosomes/">Data set by chromosome</a></li> 
  <li><a href="/goldenPath/tetNig2/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/tetNig2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#tetNig2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="tetNig2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/tetNig2/vsHg19/">Tetraodon/Human (hg19)</a></li> 
        <li><a href="/goldenPath/tetNig2/vsFr3/">Tetraodon/Fugu (fr3)</a></li>
        <li><a href="/goldenPath/tetNig2/vsFr2/">Tetraodon/Fugu (fr2)</a></li>
        <li><a href="/goldenPath/tetNig2/vsOryLat2/">Tetraodon/Medaka (oryLat2)</a></li>
        <li><a href="/goldenPath/tetNig2/vsMm10/">Tetraodon/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/tetNig2/vsMm9/">Tetraodon/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/tetNig2/vsGasAcu1/">Tetraodon/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/tetNig2/vsDanRer7/">Tetraodon/Zebrafish (danRer7)</a></li>
        <li><a href="/goldenPath/tetNig2/vsDanRer6/">Tetraodon/Zebrafish (danRer6)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Feb. 2004 (Genoscope 7/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=tetNig1"
    target="_blank">tetNig1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/tetNig1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/tetNig1/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/tetNig1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/tetNig1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#tetNig1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="tetNig1pairwise"> 
      <ul>
        <li><a href="/goldenPath/tetNig1/vsHg19/">Tetraodon/Human (hg19)</a></li>
        <li><a href="/goldenPath/tetNig1/vsHg18/">Tetraodon/Human (hg18)</a></li>
        <li><a href="/goldenPath/tetNig1/vsHg17/">Tetraodon/Human (hg17)</a></li>
        <li><a href="/goldenPath/tetNig1/vsFr2/">Tetraodon/Fugu (fr2)</a></li>
        <li><a href="/goldenPath/tetNig1/vsFr1/">Tetraodon/Fugu (fr1)</a></li>
        <li><a href="/goldenPath/tetNig1/vsOryLat2/">Tetraodon/Medaka (oryLat2)</a></li>
        <li><a href="/goldenPath/tetNig1/vsMm9/">Tetraodon/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/tetNig1/vsMm8/">Tetraodon/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/tetNig1/vsMm7/">Tetraodon/Mouse (mm7)</a></li>
        <li><a href="/goldenPath/tetNig1/vsMm6/">Tetraodon/Mouse (mm6)</a></li>
        <li><a href="/goldenPath/tetNig1/vsDanRer5/">Tetraodon/Zebrafish (danRer5)</a></li>
        <li><a href="/goldenPath/tetNig1/vsDanRer4/">Tetraodon/Zebrafish (danRer4)</a></li>
        <li><a href="/goldenPath/tetNig1/vsDanRer3/">Tetraodon/Zebrafish (danRer3)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Tibetan frog Downloads -->
<a name="tibetan_frog"></a>
<h2>Tibetan frog genome</h2>
<h3>Mar. 2015 (BGI_ZX_2015/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=nanPar1"
    target="_blank">nanPar1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/nanPar1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/nanPar1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/nanPar1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#nanPar1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="nanPar1pairwise"> 
      <ul>
        <li><a href="/goldenPath/nanPar1/vsXenTro7/">Tibetan frog/X.  tropicalis (xenTro7)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Tree shrew Downloads -->
<a name="tree_shrew"></a> 
<h2>Tree shrew genome</h2>
<h3>Dec. 2006 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=tupBel1" 
    target="_blank">tupBel1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/tupBel1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/tupBel1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/tupBel1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#tupBel1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="tupBel1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/tupBel1/vsGalVar1/">Tree shrew/Malayan flying lemur (galVar1)</a></li>
        <li><a href="/goldenPath/tupBel1/vsMm10/">Tree shrew/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/tupBel1/vsTarSyr2/">Tree shrew/Tarsier (tarSyr2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Turkey Downloads -->
<a name="turkey"></a>
<h2>Turkey genome</h2>
<h3>Nov. 2014 (TGC Turkey_5.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=melGal5"
    target="_blank">melGal5</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/melGal5/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/melGal5/database/">Annotation database</a></li>
  <li><a href="/goldenPath/melGal5/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#melGal5pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="melGal5pairwise"> 
      <ul> 
        <li><a href="/goldenPath/melGal5/vsHg38/">Turkey/Human (hg38)</a></li>
        <li><a href="/goldenPath/melGal5/vsGalGal5/">Turkey/Chicken (galGal5)</a></li>
        <li><a href="/goldenPath/melGal5/vsGeoFor1/">Turkey/Medium ground finch (geoFor1)</a></li>
        <li><a href="/goldenPath/melGal5/vsMm10/">Turkey/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/melGal5/vsMonDom5/">Turkey/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/melGal5/vsFalPer1/">Turkey/Peregrine falcon (falPer1)</a></li>
        <li><a href="/goldenPath/melGal5/vsRn6/">Turkey/Rat (rn6)</a></li>
        <li><a href="/goldenPath/melGal5/vsXenTro7/">Turkey/X. tropicalis (xenTro7)</a></li>
        <li><a href="/goldenPath/melGal5/vsTaeGut2/">Turkey/Zebra finch (taeGut2)</a></li>
        <li><a href="/goldenPath/melGal5/vsDanRer10/">Turkey/Zebrafish (danRer10)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Dec. 2009 (TGC Turkey_2.01/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=melGal1" 
    target="_blank">melGal1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/melGal1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/melGal1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/melGal1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#melGal1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="melGal1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/melGal1/vsHg19/">Turkey/Human (hg19)</a></li>
        <li><a href="/goldenPath/melGal1/vsGalGal3/">Turkey/Chicken (galGal3)</a></li> 
        <li><a href="/goldenPath/melGal1/vsMm9/">Turkey/Mouse (mm9)</a></li> 
        <li><a href="/goldenPath/melGal1/vsMonDom5/">Turkey/Opossum (monDom5)</a></li> 
        <li><a href="/goldenPath/melGal1/vsRn4/">Turkey/Rat (rn4)</a></li> 
        <li><a href="/goldenPath/melGal1/vsXenTro3/">Turkey/X. tropicalis (xenTro3)</a></li> 
        <li><a href="/goldenPath/melGal1/vsXenTro2/">Turkey/X. tropicalis (xenTro2)</a></li> 
        <li><a href="/goldenPath/melGal1/vsTaeGut1/">Turkey/Zebra finch (taeGut1)</a></li> 
        <li><a href="/goldenPath/melGal1/vsDanRer7/">Turkey/Zebrafish (danRer7)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Wallaby Downloads -->
<a name="wallaby"></a>
<h2>Wallaby genome</h2>
<h3>Sep. 2009 (TWGS Meug_1.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=macEug2" 
    target="_blank">macEug2</a>)</h3> 
<ul>
  <li><a href="/goldenPath/macEug2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/macEug2/database/">Annotation database</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#macEug2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="macEug2pairwise"> 
      <ul>
        <li><a href="/goldenPath/macEug2/vsHg38/">Wallaby/Human (hg38)</a></li>
        <li><a href="/goldenPath/macEug2/vsMm10/">Wallaby/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- White rhinoceros Downloads -->
<a name="white_rhinoceros"></a>
<h2>White rhinoceros genome</h2>
<h3>May 2012 (CerSimSim1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=cerSim1"
    target="_blank">cerSim1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/cerSim1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/cerSim1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/cerSim1/liftOver/">LiftOver files</Aa></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#cerSim1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="cerSim1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/cerSim1/vsHg19/">White rhinoceros/Human (hg19)</a></li>
        <li><a href="/goldenPath/cerSim1/vsBosTau7/">White rhinoceros/Cow (bosTau7)</a></li>
        <li><a href="/goldenPath/cerSim1/vsMm10/">White rhinoceros/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/cerSim1/vsMonDom5/">White rhinoceros/Opossum (monDom5)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- X. tropicalis Downloads -->
<a name="xentro"></a>
<h2>X. tropicalis genome</h2>
<h3>Jul. 2016 (JGI 9.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=xenTro9"
    target="_blank">xenTro9</a>)</h3>
<ul>
  <li><a href="/goldenPath/xenTro9/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/xenTro9/database/">Annotation database</a></li>
  <li><a href="/goldenPath/xenTro9/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#xenTro9pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="xenTro9pairwise">
      <ul>
        <li><a href="/goldenPath/xenTro9/vsHg38/">X. tropicalis/Human (hg38)</a></li>
        <li><a href="/goldenPath/xenTro9/vsXenLae2/">X. tropicalis/African Clawed Frog (xenLae2)</a></li>
        <li><a href="/goldenPath/xenTro9/vsNanPar1/">X. tropicalis/Tibetan frog (nanPar1)</a></li>
        <li><a href="/goldenPath/xenTro9/vsGalGal5/">X. tropicalis/Chicken (galGal5)</a></li>
        <li><a href="/goldenPath/xenTro9/vsAnoCar2/">X. tropicalis/Lizard (anoCar2)</a></li>
        <li><a href="/goldenPath/xenTro9/vsMonDom5/">X. tropicalis/Opossum (monDon5)</a></li>
        <li><a href="/goldenPath/xenTro9/vsCanFam3/">X. tropicalis/Dog (canFam3)</a></li>
        <li><a href="/goldenPath/xenTro9/vsMm10/">X. tropicalis/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/xenTro9/vsFr3/">X. tropicalis/Fugu (fr3)</a></li>
      </ul>
    </div>
  </li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href="#xenTro9mult">Multiple Alignment</a>
    <div class="collapse" id="xenTro9mult">
      <ul>
        <li><a href="/goldenPath/xenTro9/multiz11way/">Multiple alignments of 10 vertebrate
        genomes with <em>X. tropicalis</em></a></li>
        <li><a href="/goldenPath/xenTro9/phastCons11way/">Conservation scores for alignments of 10
        vertebrate genomes with <em>X. tropicalis</em></a></li>
        <li><a href="/goldenPath/xenTro9/phyloP11way/">Basewise conservation scores(phyloP) of 10
        vertebrate genomes with <em>X. tropicalis</em></a></li>
        <li><a href="/goldenPath/xenTro9/multiz11way/alignments/">FASTA alignments of 10
        vertebrate genomes with <em>X. tropicalis</em></a></li>
      </ul>
    </div>
  </li>
</ul>
<h3>Sep. 2012 (JGI 7.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=xenTro7"
    target="_blank">xenTro7</a>)</h3> 
<ul>
  <li><a href="/goldenPath/xenTro7/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/xenTro7/database/">Annotation database</a></li>
  <li><a href="/goldenPath/xenTro7/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#xenTro7pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="xenTro7pairwise"> 
      <ul> 
        <li><a href="/goldenPath/xenTro7/vsHg38/">X. tropicalis/Human (hg38)</a></li>
        <li><a href="/goldenPath/xenTro7/vsRn6/">X. tropicalis/Rat (rn6)</a></li>
        <li><a href="/goldenPath/xenTro7/vsNanPar1/">X. tropicalis/Tibetan frog (nanPar1)</a></li>
        <li><a href="/goldenPath/xenTro7/vsMelGal5/">X. tropicalis/Turkey (melGal5)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Nov. 2009 (JGI 4.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=xenTro3" 
    target="_blank">xenTro3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/xenTro3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/xenTro3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/xenTro3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#xenTro3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="xenTro3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/xenTro3/vsHg19/">X. tropicalis/Human (hg19)</a></li>
        <li><a href="/goldenPath/xenTro3/vsGalGal3/">X. tropicalis/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/xenTro3/vsAnoCar2/">X. tropicalis/Lizard (anoCar2)</a></li>
        <li><a href="/goldenPath/xenTro3/vsMm10/">X. tropicalis/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/xenTro3/vsMm9/">X. tropicalis/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/xenTro3/vsMonDom5/">X. tropicalis/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/xenTro3/vsRn4/">X. tropicalis/Rat (rn4)</a></li>
        <li><a href="/goldenPath/xenTro3/vsMelGal1/">X. tropicalis/Turkey (melGal1)</a></li>
        <li><a href="/goldenPath/xenTro3/vsDanRer7/">X. tropicalis/Zebrafish (danRer7)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#xenTro3mult>Multiple
    alignments</a>
    <div class="collapse" id="xenTro3mult"> 
      <ul> 
        <li><a href="/goldenPath/xenTro3/multiz9way/">Multiple alignments of 8 vertebrate genomes 
        with X. tropicalis</a></li>
        <li><a href="/goldenPath/xenTro3/phastCons9way/">Conservation scores for alignments of 8 
        vertebrate genomes with X. tropicalis</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Aug. 2005 (JGI 4.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=xenTro2"
    target="_blank">xenTro2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/xenTro2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/xenTro2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/xenTro2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#xenTro2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="xenTro2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/xenTro2/vsHg19/">X. tropicalis/Human (hg19)</a></li>
        <li><a href="/goldenPath/xenTro2/vsHg18/">X. tropicalis/Human (hg18)</a></li>
        <li><a href="/goldenPath/xenTro2/vsGalGal3/">X. tropicalis/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/xenTro2/vsGalGal2/">X. tropicalis/Chicken (galGal2)</a></li>
        <li><a href="/goldenPath/xenTro2/vsAnoCar2/">X. tropicalis/Lizard (anoCar2)</a></li>
        <li><a href="/goldenPath/xenTro2/vsAnoCar1/">X. tropicalis/Lizard (anoCar1)</a></li>
        <li><a href="/goldenPath/xenTro2/vsMm9/">X. tropicalis/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/xenTro2/vsMm8/">X. tropicalis/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/xenTro2/vsMonDom4/">X. tropicalis/Opossum (monDom4)</a></li>
        <li><a href="/goldenPath/xenTro2/vsRn4/">X. tropicalis/Rat (rn4)</a></li>
        <li><a href="/goldenPath/xenTro2/vsDanRer7/">X. tropicalis/Zebrafish (danRer7)</a></li>
        <li><a href="/goldenPath/xenTro2/vsDanRer6/">X. tropicalis/Zebrafish (danRer6)</a></li>
        <li><a href="/goldenPath/xenTro2/vsDanRer4/">X. tropicalis/Zebrafish (danRer4)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#xenTro2mult>Multiple
    alignments</a>
    <div class="collapse" id="xenTro2mult"> 
      <ul> 
        <li><a href="/goldenPath/xenTro2/multiz7way/">Multiple alignments of 6 vertebrate genomes 
        with X. tropicalis</a></li>
<!--                  
        <li><a href="/goldenPath/xenTro2/phastCons/">Conservation scores for alignments of 6 
        vertebrate genomes with X. tropicalis</a></li>
-->
      </ul>
    </div>
  </li> 
</ul>
<h3>Oct. 2004 (JGI 3.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=xenTro1" 
target="_blank">xenTro1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/xenTro1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/xenTro1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/xenTro1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#xenTro1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="xenTro1pairwise"> 
      <ul>
        <li><a href="/goldenPath/xenTro1/vsHg18/">X. tropicalis/Human (hg18)</a></li>
        <li><a href="/goldenPath/xenTro1/vsHg17/">X. tropicalis/Human (hg17)</a></li>
        <li><a href="/goldenPath/xenTro1/vsGalGal2/">X. tropicalis/Chicken (galGal2)</a></li>
        <li><a href="/goldenPath/xenTro1/vsMm8/">X. tropicalis/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/xenTro1/vsMm7/">X. tropicalis/Mouse (mm7)</a></li>
        <li><a href="/goldenPath/xenTro1/vsMm6/">X. tropicalis/Mouse (mm6)</a></li>
        <li><a href="/goldenPath/xenTro1/vsDanRer1/">X. tropicalis/Zebrafish (danRer1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#xenTro1mult>Multiple
    alignments</a>
    <div class="collapse" id="xenTro1mult"> 
      <ul> 
        <li><a href="/goldenPath/xenTro1/multiz5way/">Multiple alignments of 4 vertebrate genomes 
        with X. tropicalis</a></li>
        <li><a href="/goldenPath/xenTro1/phastCons/">Conservation scores for alignments of 4 
        vertebrate genomes with X. tropicalis</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Zebra finch Downloads -->
<a name="zebra_finch"></a></li>
<h2>Zebra finch genome</h2>
<h3>Feb. 2013 (taeGut2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=taeGut2"
    target="_blank">taeGut2</a>)</h3> 
<ul>
  <li><a href="/goldenPath/taeGut2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/taeGut2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/taeGut2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#taeGut2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="taeGut2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/taeGut2/vsHg19/">Zebra finch/Human (hg19)</a></li>
        <li><a href="/goldenPath/taeGut2/vsGalGal4/">Zebra finch/Chicken (galGal4)</a></li>
        <li><a href="/goldenPath/taeGut2/vsMelGal1/">Zebra finch/Turkey (melGal1)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2008 (WUGSC 3.2.4/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=taeGut1"
    target="_blank">taeGut1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/taeGut1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/taeGut1/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/taeGut1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/taeGut1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#taeGut1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="taeGut1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/taeGut1/vsHg19/">Zebra finch/Human (hg19)</a></li>
        <li><a href="/goldenPath/taeGut1/vsHg18/">Zebra finch/Human (hg18)</a></li>
        <li><a href="/goldenPath/taeGut1/vsGalGal3/">Zebra finch/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/taeGut1/vsMm10/">Zebra finch/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/taeGut1/vsMelGal1/">Zebra finch/Turkey (melGal1)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Zebrafish Downloads -->
<a name="zebrafish"></a>
<h2>Zebrafish genome</h2>
<h3>May 2017 (GRCz11/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=danRer11"
  target="_blank">danRer11</a>)</h3>
<ul>
  <li><a href="/goldenPath/danRer11/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/danRer11/database/">Annotation database</a></li>
  <li><a href="/goldenPath/danRer11/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer11pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="danRer11pairwise">
      <ul>
        <li><a href="/goldenPath/danRer11/vsHg38/">Zebrafish/Human (hg38)</a></li>
        <li><a href="/goldenPath/danRer11/vsMm10/">Zebrafish/Mouse (mm10)</a></li>
      </ul>
    </div>
  </li>
</ul>
<h3>Sep. 2014 (GRCz10/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=danRer10"
    target="_blank">danRer10</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/danRer10/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/danRer10/database/">Annotation database</a></li>
  <li><a href="/goldenPath/danRer10/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer10pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="danRer10pairwise"> 
      <ul> 
        <li><a href="/goldenPath/danRer10/vsHg38/">Zebrafish/Human (hg38)</a></li>
        <li><a href="/goldenPath/danRer10/vsHg19/">Zebrafish/Human (hg19)</a></li>
        <li><a href="/goldenPath/danRer10/vsGalGal5/">Zebrafish/Chicken (galGal5)</a></li>
        <li><a href="/goldenPath/danRer10/vsMm10/">Zebrafish/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/danRer10/vsRn6/">Zebrafish/Rat (rn6)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2010 (Zv9/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=danRer7"
target="_blank">danRer7</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/danRer7/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/danRer7/database/">Annotation database</a></li>
  <li><a href="/goldenPath/danRer7/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer7pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="danRer7pairwise"> 
      <ul> 
<!-- 
        <li><a href="/goldenPath/danRer7/vsSelf/">Zebrafish self alignments</a></li>
-->
        <li><a href="/goldenPath/danRer7/vsHg19/">Zebrafish/Human (hg19)</a></li>
        <li><a href="/goldenPath/danRer7/vsPanTro3/">Zebrafish/Chimp (panTro3)</a></li>
        <li><a href="/goldenPath/danRer7/vsFr3/">Zebrafish/Fugu (fr3)</a></li>
        <li><a href="/goldenPath/danRer7/vsFr2/">Zebrafish/Fugu (fr2)</a></li>
        <li><a href="/goldenPath/danRer7/vsOryLat2/">Zebrafish/Medaka (oryLat2)</a></li>
        <li><a href="/goldenPath/danRer7/vsMm10/">Zebrafish/Mouse (mm10)</a></li>
        <li><a href="/goldenPath/danRer7/vsMm9/">Zebrafish/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/danRer7/vsGasAcu1/">Zebrafish/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/danRer7/vsTetNig2/">Zebrafish/Tetraodon (tetNig2)</a></li>
        <li><a href="/goldenPath/danRer7/vsMelGal1/">Zebrafish/Turkey (melGal1)</a></li>
        <li><a href="/goldenPath/danRer7/vsXenTro3/">Zebrafish/X. tropicalis (xenTro3)</a></li>
        <li><a href="/goldenPath/danRer7/vsXenTro2/">Zebrafish/X. tropicalis (xenTro2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer7mult>Multiple
    alignments</a>
    <div class="collapse" id="danRer7mult"> 
      <ul> 
        <li><a href="/goldenPath/danRer7/multiz8way/">Multiple alignments of 7 genomes with 
        Zebrafish</a></li>
        <li><a href="/goldenPath/danRer7/phastCons8way/">Conservation scores for alignments of 7 
        genomes with Zebrafish</a></li>
        <li><a href="/goldenPath/danRer7/phyloP8way/">Basewise conservation scores (phyloP) of 7 
        genomes with Zebrafish</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Dec. 2008 (Zv8/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=danRer6" 
    target="_blank">danRer6</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/danRer6/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/danRer6/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/danRer6/database/">Annotation database</a></li>
  <li><a href="/goldenPath/danRer6/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer6pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="danRer6pairwise"> 
      <ul> 
        <li><a href="/goldenPath/danRer6/vsSelf/">Zebrafish self alignments</a></li>
        <li><a href="/goldenPath/danRer6/vsHg19/">Zebrafish/Human (hg19)</a></li>
        <li><a href="/goldenPath/danRer6/vsOryLat2/">Zebrafish/Medaka (oryLat2)</a></li>
        <li><a href="/goldenPath/danRer6/vsMm9/">Zebrafish/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/danRer6/vsGasAcu1/">Zebrafish/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/danRer6/vsTetNig2/">Zebrafish/Tetraodon (tetNig2)</a></li>
        <li><a href="/goldenPath/danRer6/vsXenTro2/">Zebrafish/X.Tropicalis (xenTro2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer6mult>Multiple
    alignments</a>
    <div class="collapse" id="danRer6mult"> 
      <ul> 
        <li><a href="/goldenPath/danRer6/multiz6way/">Multiple alignments of 5 vertebrate genomes 
        with Zebrafish</a></li>
        <li><a href="/goldenPath/danRer6/phastCons6way/">Conservation scores for alignments of 5 
        vertebrate genomes with Zebrafish</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2007 (Zv7/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=danRer5"
    target="_blank">danRer5</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/danRer5/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/danRer5/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/danRer5/database/">Annotation database</a></li>
  <li><a href="/goldenPath/danRer5/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer5pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="danRer5pairwise"> 
      <ul> 
        <li><a href="/goldenPath/danRer5/vsHg19/">Zebrafish/Human (hg19)</a></li>
        <li><a href="/goldenPath/danRer5/vsHg18/">Zebrafish/Human (hg18)</a></li>
        <li><a href="/goldenPath/danRer5/vsFr2/">Zebrafish/Fugu (fr2)</a></li>
        <li><a href="/goldenPath/danRer5/vsOryLat2/">Zebrafish/Medaka (oryLat2)</a></li>
        <li><a href="/goldenPath/danRer5/vsMm9/">Zebrafish/Mouse (mm9)</a></li>
        <li><a href="/goldenPath/danRer5/vsMonDom5/">Zebrafish/Opossum (monDom5)</a></li>
        <li><a href="/goldenPath/danRer5/vsTetNig1/">Zebrafish/Tetraodon (tetNig1)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Mar. 2006 (Zv6/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=danRer4"
    target="_blank">danRer4</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/danRer4/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/danRer4/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/danRer4/database/">Annotation database</a></li>
  <li><a href="/goldenPath/danRer4/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="danRer4pairwise"> 
      <ul> 
        <li><a href="/goldenPath/danRer4/vsHg18/">Zebrafish/Human (hg18)</a></li>
        <li><a href="/goldenPath/danRer4/vsGalGal3/">Zebrafish/Chicken (galGal3)</a></li>
        <li><a href="/goldenPath/danRer4/vsFr2/">Zebrafish/Fugu (fr2)</a></li>
        <li><a href="/goldenPath/danRer4/vsFr1/">Zebrafish/Fugu (fr1)</a></li>
        <li><a href="/goldenPath/danRer4/vsMm8/">Zebrafish/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/danRer4/vsMonDom4/">Zebrafish/Opossum (monDom4)</a></li>
        <li><a href="/goldenPath/danRer4/vsRn4/">Zebrafish/Rat (rn4)</a></li>
        <li><a href="/goldenPath/danRer4/vsGasAcu1/">Zebrafish/Stickleback (gasAcu1)</a></li>
        <li><a href="/goldenPath/danRer4/vsTetNig1/">Zebrafish/Tetraodon (tetNig1)</a></li>
        <li><a href="/goldenPath/danRer4/vsXenTro2/">Zebrafish/X. tropicalis (xenTro2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer4mult>Multiple
    alignments</a>
    <div class="collapse" id="danRer4mult"> 
      <ul> 
        <li><a href="/goldenPath/danRer4/multiz7way/">Multiple alignments of 6 vertebrate genomes 
        with Zebrafish</a></li>
        <li><a href="/goldenPath/danRer4/phastCons7wayScores/">Conservation scores for alignments of
        6 vertebrate genomes with Zebrafish</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>May 2005 (Zv5/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=danRer3"
    target="_blank">danRer3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/danRer3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/danRer3/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/danRer3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/danRer3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="danRer3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/danRer3/vsSelf/">Zebrafish self alignments</a></li>
        <li><a href="/goldenPath/danRer3/vsHg18/">Zebrafish/Human (hg18)</a></li>
        <li><a href="/goldenPath/danRer3/vsFr1/">Zebrafish/Fugu (fr1)</a></li>
        <li><a href="/goldenPath/danRer3/vsMm8/">Zebrafish/Mouse (mm8)</a></li>
        <li><a href="/goldenPath/danRer3/vsMm7/">Zebrafish/Mouse (mm7)</a></li>
<!--
        <li><a href="/goldenPath/danRer3/vsMonDom2/">Zebrafish/Opossum (monDom2)</a></li>
-->
        <li><a href="/goldenPath/danRer3/vsTetNig1/">Zebrafish/Tetraodon (tetNig1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer3mult>Multiple
    alignments</a>
    <div class="collapse" id="danRer3mult"> 
      <ul> 
        <li><a href="/goldenPath/danRer3/multiz5way/">Multiple alignments of 4 vertebrate genomes 
        with Zebrafish</a></li>
        <li><a href="/goldenPath/danRer3/phastCons5wayScores/">Conservation scores for alignments of
        4 vertebrate genomes with Zebrafish</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jun. 2004 (danRer2)</h3> 
<ul> 
  <li><a href="/goldenPath/danRer2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/danRer2/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/danRer2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/danRer2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="danRer2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/danRer2/vsHg17/">Zebrafish/Human (hg17)</a></li>
        <li><a href="/goldenPath/danRer2/vsFr1/">Zebrafish/Fugu (fr1)</a></li>
        <li><a href="/goldenPath/danRer2/vsMm6/">Zebrafish/Mouse (mm6)</a></li>
        <li><a href="/goldenPath/danRer2/vsMonDom1/">Zebrafish/Opossum (monDom1)</a></li>
        <li><a href="/goldenPath/danRer2/vsTetNig1/">Zebrafish/Tetraodon (tetNig1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer2mult>Multiple
    alignments</a>
    <div class="collapse" id="danRer2mult"> 
      <ul> 
        <li><a href="/goldenPath/danRer2/multiz6way/">Multiple alignments of 5 vertebrate genomes 
        with Zebrafish</a></li>
        <li><a href="/goldenPath/danRer2/phastCons6wayScores/">Conservation scores for alignments of
        5 vertebrate genomes with Zebrafish</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Nov. 2003 (danRer1)</h3> 
<ul> 
  <li><a href="/goldenPath/danRer1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/danRer1/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/danRer1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/danRer1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#danRer1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="danRer1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/danRer1/vsHg17/">Zebrafish/Human (hg17)</a></li>
        <li><a href="/goldenPath/danRer1/vsFr1/">Zebrafish/Fugu (fr1)</a></li>
        <li><a href="/goldenPath/danRer1/vsMm5/">Zebrafish/Mouse (mm5)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Sequence-Only Downloads -->
<a name="sequence_only"></a>
<h2>Vertebrates (Sequence only)</h2> 
<ul> 
  <li><a href="/goldenPath/dasNov1/bigZips/">Armadillo (May 2005, dasNov1)</a></li>
  <li><a href="/goldenPath/otoGar1/bigZips/">Bushbaby (Dec. 2006, otoGar1)</a></li>
  <li><a href="/goldenPath/venter1/bigZips/">J. Craig Venter (Sep. 2007, venter1)</a></li>
</ul>	
<p>
The following <a href="http://hgdownload.soe.ucsc.edu/gbdb/" 
target="_blank">http://hgdownload.soe.ucsc.edu/gbdb/</a> location has assembly sequences used in 
alignment tracks, such as in the 100-species conservation track. For example, you can find the 
underlying mayZeb1.2bit sequence file for the Zebra Mbuna fish assembly, not yet released but used 
in the <a href="http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg38&g=cons100way" 
target="_blank">hg38 Vertebrate Multiz Alignment & Conservation (100 Species)</a> track, here: 
<a href="http://hgdownload.soe.ucsc.edu/gbdb/mayZeb1" 
target="_blank">http://hgdownload.soe.ucsc.edu/gbdb/mayZeb1/</a>. These links also display under a 
column titled &quot;UCSC version&quot; on the conservation track description page.</p>

<!-- Ciona Downloads -->
<a name="ciona"></a>
<h2>Ciona intestinalis genome</h2>
<h3>Apr. 2011 (Kyoto KH/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ci3"
    target="_blank">ci3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/ci3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ci3/database/">Annotation database</a></li>
</ul>

<h3>Mar. 2005 (JGI 2.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ci2"
    target="_blank">ci2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/ci2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ci2/database/">Annotation database</a></li>
</ul>	
<h3>Dec. 2002 (JGI 1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ci1"
    target="_blank">ci1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/ci1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ci1/database/">Annotation database</a></li>
</ul>	

<!-- Urchin Downloads -->
<a name="urchin"></a>
<h2>S. purpuratus genome</h2>
<h3>Sep. 2006 (Baylor 2.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=strPur2"
target="_blank">strPur2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/strPur2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/strPur2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/strPur2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#strPur2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="strPur2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/strPur2/vsHg18/">S. purpuratus/Human (hg18)</a></li>
        <li><a href="/goldenPath/strPur2/vsCi2/">S. purpuratus/C. intestinalis (ci2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Apr. 2005 (Baylor 1.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=strPur1"
target="_blank">strPur1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/strPur1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/strPur1/database/">Annotation database</a></li>
<!--
  <li><a href="/goldenPath/strPur1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#strPur1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="strPur1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/strPur1/vsHg18/">S. purpuratus/human (hg18)</a></li>
        <li><a href="/goldenPath/strPur1/vsCioSav1/">S. purpuratus/C. savignyi (cioSav1)</a></li>
      </ul>
    </div>
  </li> 
-->
</ul>

<!-- A. gambiae Downloads -->
<a name="anoGam"></a>
<h2>A. gambiae genome</h2>
<h3>Nov. 2006 (ICSAGA P3/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=anoGam3"
target="_blank">anoGam3</a>)</h3>
<ul>
  <li><a href="/goldenPath/anoGam3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/anoGam3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/anoGam3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#anoGam3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="anoGam3pairwise">
      <ul>
        <li><a href="/goldenPath/anoGam3/vsDm6/">A. gambiae/D. melanogaster (dm6)</a></li>
      </ul>
    </div>
  </li>
</ul>
<h3>Feb. 2003 (IAGEC MOZ2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=anoGam1"
target="_blank">anoGam1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/anoGam1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/anoGam1/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/anoGam1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/anoGam1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#anoGam1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="anoGam1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/anoGam1/vsDm3/">A. gambiae/D. melanogaster (dm3)</a></li>
        <li><a href="/goldenPath/anoGam1/vsDm2/">A. gambiae/D. melanogaster (dm2)</a></li>
        <li><a href="/goldenPath/anoGam1/vsDm1/">A. gambiae/D. melanogaster (dm1)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- A. mellifera Downloads -->
<a name="apiMel"></a>
<h2>A. mellifera genome</h2> 
<h3>May 2005 (Baylor HGSC Amel_3.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=apiMel3" 
    target="_blank">apiMel3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/apiMel3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/apiMel3/chromosomes/">Data set by chromosome</a></li>
</ul>	
<h3>Jan. 2005 (Baylor 2.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=apiMel2"
    target="_blank">apiMel2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/apiMel2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/apiMel2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/apiMel2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#apiMel2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="apiMel2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/apiMel2/vsDm2/">A. mellifera/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2004 (Baylor 1.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=apiMel1"
    target="_blank">apiMel1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/apiMel1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/apiMel1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/apiMel1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#apiMel1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="apiMel1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/apiMel1/vsDm2/">A. mellifera/D. melanogaster (dm2)</a></li>
        <li><a href="/goldenPath/apiMel1/vsDm1/">A. mellifera/D. melanogaster (dm1)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. ananassae Downloads -->
<a name="droAna"></a>
<h2>D. ananassae genome</h2>
<h3>Feb. 2006 (droAna3)</h3> 
<ul> 
  <li><a href="/goldenPath/droAna3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droAna3/liftOver/">LiftOver files</a></li>
</ul>	
<h3>Aug. 2005 (Agencourt prelim/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droAna2" 
target="_blank">droAna2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droAna2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droAna2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/droAna2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droAna2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droAna2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droAna2/vsDm3/">D. ananassae/D. melanogaster (dm3)</a></li>
        <li><a href="/goldenPath/droAna2/vsDm2/">D. ananassae/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2004 (TIGR/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droAna1 "
    target="_blank">droAna1 </a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droAna1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droAna1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/droAna1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droAna1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droAna1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droAna1/vsDm2/">D. ananassae/D. melanogaster (dm2)</a></li>
        <li><a href="/goldenPath/droAna1/vsDm1/">D. ananassae/D. melanogaster (dm1)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. erecta Downloads -->
<a name="droEre"></a>
<h2>D. erecta genome</h2>
<h3>Feb. 2006 (droEre2)</h3> 
<ul> 
  <li><a href="/goldenPath/droEre2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droEre2/liftOver/">LiftOver files</a></li>
</ul>	
<h3>Aug. 2005 (Agencourt prelim/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droEre1" 
target="_blank">droEre1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droEre1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droEre1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/droEre1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droEre1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droEre1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droEre1/vsDm3/">D. erecta/D. melanogaster (dm3)</a></li>
        <li><a href="/goldenPath/droEre1/vsDm2/">D. erecta/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. grimshawi Downloads -->
<a name="droGri"></a>
<h2>D. grimshawi genome</h2>
<h3>Feb. 2006 (droGri2)</h3>
<ul> 
  <li><a href="/goldenPath/droGri2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droGri2/liftOver/">LiftOver files</a></li>
</ul>	
<h3>Aug. 2005 (Agencourt prelim/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droGri1" 
    target="_blank">droGri1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droGri1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droGri1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/droGri1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droGri1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droGri1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droGri1/vsDm2/">D. grimshawi/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. melanogaster Downloads -->
<a name="fruitfly"></a>
<h2>D. melanogaster genome</h2>
<h3>Aug. 2014 (BDGP Release 6 + ISO1 MT/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=dm6" 
    target="_blank">dm6</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/dm6/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/dm6/database/">Annotation database</a></li>
  <li><a href="/goldenPath/dm6/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dm6pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="dm6pairwise"> 
      <ul> 
        <li><a href="/goldenPath/dm6/vsDroPer1/">D. melanogaster/D. persimilis (droPer1)</a></li>
        <li><a href="/goldenPath/dm6/vsDroSec1/">D. melanogaster/D. sechellia (droSec1)</a></li>
        <li><a href="/goldenPath/dm6/vsDroSim1/">D. melanogaster/D. simulans (droSim1)</a></li>
        <li><a href="/goldenPath/dm6/vsAnoGam3/">D. melanogaster/A. gambiae (anoGam3)</a></li>
        <li><a href="/goldenPath/dm6/vsAnoGam1/">D. melanogaster/A. gambiae (anoGam1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dm6mult>Multiple
    alignments</a>
    <div class="collapse" id="dm6mult"> 
      <ul> 
        <li><a href="/goldenPath/dm6/multiz27way/">Multiple alignments of 26 insects with D. 
        melanogaster</a></li>
        <li><a href="/goldenPath/dm6/phastCons27way/">Conservation scores for alignments of 26 
        insects with D. melanogaster</a></li>
        <li><a href="/goldenPath/dm6/phyloP27way/">Basewise conservation scores (phyloP) of 26 
        insects with D. melanogaster</a></li>
        <li><a href="/goldenPath/dm6/multiz27way/alignments/">FASTA alignments of 26 insects with D.
        melanogaster for CDS regions</a></li>
	<li><a href="/goldenPath/dm6/multiz124way/">Multiple alignments of 124 insects with D. 
        melanogaster</a></li>
        <li><a href="/goldenPath/dm6/phastCons124way/">Conservation scores for alignments of 124 
        insects with D. melanogaster</a></li>
        <li><a href="/goldenPath/dm6/phyloP124way/">Basewise conservation scores (phyloP) of 124 
        insects with D. melanogaster</a></li>
        <li><a href="/goldenPath/dm6/multiz124way/alignments/">FASTA alignments of 124 insects with
	D. melanogaster for CDS regions</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Apr. 2006 (BDGP R5/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=dm3"
    target="_blank">dm3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/dm3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/dm3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/dm3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dm3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="dm3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/dm3/vsDroAna3/">D. melanogaster/D. ananassae (droAna3)</a></li>
        <li><a href="/goldenPath/dm3/vsDroAna2/">D. melanogaster/D. ananassae (droAna2)</a></li>
        <li><a href="/goldenPath/dm3/vsDroEre2/">D. melanogaster/D. erecta (droEre2)</a></li>
        <li><a href="/goldenPath/dm3/vsDroEre1/">D. melanogaster/D. erecta (droEre1)</a></li>
        <li><a href="/goldenPath/dm3/vsDroGri2/">D. melanogaster/D. grimshawi (droGri2)</a></li>
        <li><a href="/goldenPath/dm3/vsDroMoj3/">D. melanogaster/D. mojavensis (droMoj3)</a></li>
        <li><a href="/goldenPath/dm3/vsDroPer1/">D. melanogaster/D. persimilis (droPer1)</a></li>
        <li><a href="/goldenPath/dm3/vsDp4/">D. melanogaster/D. pseudoobscura (dp4)</a></li>
        <li><a href="/goldenPath/dm3/vsDp3/">D. melanogaster/D. pseudoobscura (dp3)</a></li>
        <li><a href="/goldenPath/dm3/vsDroSec1/">D. melanogaster/D. sechellia (droSec1)</a></li>
        <li><a href="/goldenPath/dm3/vsDroSim1/">D. melanogaster/D. simulans (droSim1)</a></li>
        <li><a href="/goldenPath/dm3/vsDroVir3/">D. melanogaster/D. virilis (droVir3)</a></li>
        <li><a href="/goldenPath/dm3/vsDroWil1/">D. melanogaster/D. willistoni (droWil1)</a></li>
        <li><a href="/goldenPath/dm3/vsDroYak2/">D. melanogaster/D. yakuba (droYak2)</a></li>
        <li><a href="/goldenPath/dm3/vsAnoGam1/">D. melanogaster/A. gambiae (anoGam1)</a></li>
        <li><a href="/goldenPath/dm3/vsApiMel3/">D. melanogaster/A. mellifera (apiMel3)</a></li>
        <li><a href="/goldenPath/dm3/vsTriCas2/">D. melanogaster/T. castaneum (triCas2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dm3mult>Multiple
    alignments</a>
    <div class="collapse" id="dm3mult"> 
      <ul> 
        <li><a href="/goldenPath/dm3/multiz15way/">Multiple alignments of 14 insects with D. 
        melanogaster</a></li> 
        <li><a href="/goldenPath/dm3/phastCons15way/">Conservation scores for alignments of 14 
        insects with D. melanogaster</a></li>
        <li><a href="/goldenPath/dm3/multiz15way/alignments/">FASTA alignments of 14 insects with 
        D. melanogaster for CDS regions</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Apr. 2004 (BDGP R4/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=dm2"
    target="_blank">dm2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/dm2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/dm2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/dm2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dm2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="dm2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/dm2/vsDroAna1/">D. melanogaster/D. ananassae (droAna1)</a></li>
        <li><a href="/goldenPath/dm2/vsDroMoj1/">D. melanogaster/D. mojavensis (droMoj1)</a></li>
        <li><a href="/goldenPath/dm2/vsDp3/">D. melanogaster/D. pseudoobscura (dp3)</a></li>
        <li><a href="/goldenPath/dm2/vsDroSim1/">D. melanogaster/D. simulans (droSim1)</a></li>
        <li><a href="/goldenPath/dm2/vsDroVir1/">D. melanogaster/D. virilis (droVir1)</a></li>
        <li><a href="/goldenPath/dm2/vsDroYak1/">D. melanogaster/D. yakuba (droYak1)</a></li>
         <li><a href="/goldenPath/dm2/vsAnoGam1/">D. melanogaster/A. gambiae (anoGam1)</a></li>
         <li><a href="/goldenPath/dm2/vsApiMel2/">D. melanogaster/A. mellifera (apiMel2)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dm2mult>Multiple
    alignments</a>
    <div class="collapse" id="dm2mult"> 
      <ul> 
        <li><a href="/goldenPath/dm2/multiz15way/">Multiple alignments of 14 insects with D. 
        melanogaster</a></li> 
        <li><a href="/goldenPath/dm2/phastCons15way/">Conservation scores for alignments of 14 
        insects with D. melanogaster</a></li>
        <li><a href="/goldenPath/dm2/multiz9way/">Multiple alignments of 8 insects with D. 
        melanogaster</a></li>
        <li><a href="/goldenPath/dm2/phastCons9way/">Conservation scores for alignments of 8 insects
        with D. melanogaster</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jan. 2003 (BDGP R3/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=dm1"
    target="_blank">dm1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/dm1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/dm1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/dm1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dm1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="dm1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/dm1/vsDp2/">D. melanogaster/D. pseudoobscura (dp2)</a></li>
        <li><a href="/goldenPath/dm1/vsDroYak1/">D. melanogaster/D. yakuba (droYak1)</a></li>
        <li><a href="/goldenPath/dm1/vsAnoGam1/">D. melanogaster/A. gambiae (anoGam1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dm1mult>Multiple
    alignments</a>
    <div class="collapse" id="dm1mult"> 
      <ul> 
        <li><a href="/goldenPath/dm1/multizDroYak1Dp2AnoGam1/">Multiple alignments of 3 insects with
        D. melanogaster</a></li>
        <li><a href="/goldenPath/dm1/phastConsDroYak1Dp2AnoGam1/">Conservation scores for alignments
        of 3 insects with D. melanogaster</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. mojavensis Downloads -->
<a name="droMoj"></a>
<h2>D. mojavensis genome</h2>
<h3>Feb. 2006 (droMoj3)</h3> 
<ul> 
  <li><a href="/goldenPath/droMoj3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droMoj3/liftOver/">LiftOver files</a></li>
</ul>	
<h3>Aug. 2005 (Agencourt prelim/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droMoj2"
    target="_blank">droMoj2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droMoj2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droMoj2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/droMoj2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droMoj2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droMoj2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droMoj2/vsDm2/">D. mojavensis/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Aug. 2004 (Agencourt prelim/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droMoj1"
    target="_blank">droMoj1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droMoj1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/droMoj1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/droMoj1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droMoj1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droMoj1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droMoj1/vsDm2/">D. mojavensis/D. melanogaster (dm2)</a></li> 
        <li><a href="/goldenPath/droMoj1/vsDm1/">D. mojavensis/D. melanogaster (dm1)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. persimilis Downloads -->
<a name="droPer"></a>
<h2>D. persimilis genome</h2>
<h3>Oct. 2005 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droPer1"
    target="_blank">droPer1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droPer1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droPer1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/droPer1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droPer1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droPer1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droPer1/vsDm6/">D. persimilis/D. melanogaster (dm6)</a></li> 
        <li><a href="/goldenPath/droPer1/vsDm3/">D. persimilis/D. melanogaster (dm3)</a></li> 
        <li><a href="/goldenPath/droPer1/vsDm2/">D. persimilis/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. pseudoobscura Downloads -->
<a name="dp"></a>
<h2>D. pseudoobscura genome</h2>
<h3>Feb. 2006 (dp4)</h3> 
<ul> 
  <li><a href="/goldenPath/dp4/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/dp4/liftOver/">LiftOver files</a></li>    
</ul>
<h3>Nov. 2004 (FlyBase 1.03/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=dp3"
    target="_blank">dp3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/dp3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/dp3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/dp3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dp3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="dp3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/dp3/vsDm3/">D. pseudoobscura/D. melanogaster (dm3)</a></li>
        <li><a href="/goldenPath/dp3/vsDm2/">D. pseudoobscura/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Aug. 2003 (Baylor freeze1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=dp2"
    target="_blank">dp2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/dp2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/dp2/database/">Annotation database</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#dp2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="dp2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/dp2/vsDm1/">D. pseudoobscura/D. melanogaster (dm1)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. sechellia Downloads -->
<a name="droSec"></a>
<h2>D. sechellia genome</h2>
<h3>Oct. 2005 (Broad/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droSec1"
    target="_blank">droSec1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droSec1/bigZips/">Full data set</a></li>
<!--
  <li><a href="/goldenPath/droSec1/chromosomes/">Data set by chromosome</a></li>
-->
  <li><a href="/goldenPath/droSec1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/droSec1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droSec1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droSec1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droSec1/vsDm6/">D. sechellia/D. melanogaster (dm6)</a></li>
        <li><a href="/goldenPath/droSec1/vsDm3/">D. sechellia/D. melanogaster (dm3)</a></li>
        <li><a href="/goldenPath/droSec1/vsDm2/">D. sechellia/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. simulans Downloads -->
<a name="droSim"></a>
<h2>D. simulans genome</h2>
<h3>Apr. 2005 (WUGSC mosaic 1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droSim1"
    target="_blank">droSim1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droSim1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droSim1/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/droSim1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/droSim1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droSim1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droSim1pairwise"> 
      <ul>
        <li><a href="/goldenPath/droSim1/vsDm6/">D. simulans/D. melanogaster (dm6)</a></li>
        <li><a href="/goldenPath/droSim1/vsDm3/">D. simulans/D. melanogaster (dm3)</a></li>
        <li><a href="/goldenPath/droSim1/vsDm2/">D. simulans/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. virilis Downloads -->
<a name="droVir"></a>
<h2>D. virilis genome</h2>
<h3>Feb. 2006 (droVir3)</h3> 
<ul> 
  <li><a href="/goldenPath/droVir3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droVir3/liftOver/">LiftOver files</a></li>
</ul>
<h3>Aug. 2005 (Agencourt prelim/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droVir2"
    target="_blank">droVir2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droVir2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droVir2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/droVir2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droVir2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droVir2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droVir2/vsDm2/">D. virilis/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2004 (Agencourt prelim/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droVir1"
    target="_blank">droVir1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droVir1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droVir1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/droVir1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droVir1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droVir1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droVir1/vsDm2/">D. virilis/D. melanogaster (dm2)</a></li> 
        <li><a href="/goldenPath/droVir1/vsDm1/">D. virilis/D. melanogaster (dm1)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- D. willistoni Downloads ================================ -->
<a name="droWil"></a>
<h2>D. willistoni genome</h2>
<h3>Feb. 2006 (droWil1)</h3> 
<ul> 
  <li><a href="/goldenPath/droWil1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/droWil1/liftOver/">LiftOver files</a></li> 
</ul>	

<!-- D. yakuba Downloads ================================ -->
<a name="droYak"></a>
<h2>D. yakuba genome</h2>
<h3>Nov. 2005 (WUGSC 7.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droYak2"
    target="_blank">droYak2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droYak2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droYak2/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/droYak2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/droYak2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droYak2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droYak2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droYak2/vsDm3/">D. yakuba/D. melanogaster (dm3)</a></li> 
        <li><a href="/goldenPath/droYak2/vsDm2/">D. yakuba/D. melanogaster (dm2)</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Apr. 2004 (WUGSC 1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=droYak1"
    target="_blank">droYak1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/droYak1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/droYak1/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/droYak1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/droYak1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#droYak1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="droYak1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/droYak1/vsDm2/">D. yakuba/D. melanogaster (dm2)</a></li> 
        <li><a href="/goldenPath/droYak1/vsDm1/">D. yakuba/D. melanogaster (dm1)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- T. castaneum Downloads -->
<a name="triCas"></a>
<h2>T. castaneum genome</h2>
<h3>Sep. 2005 (triCas2)</h3> 
<ul> 
  <li><a href="/goldenPath/triCas2/bigZips/">Full data set</a></li> 
</ul>	

<!-- C. brenneri Downloads -->
<a name="caePb"></a>
<h2>C. brenneri genome</h2>
<h3>Feb. 2008 (WUGSC 6.0.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=caePb2"
    target="_blank">caePb2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/caePb2/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/caePb2/chromosomes/">Data set by chromosome</a></li> 
  <li><a href="/goldenPath/caePb2/database/">Annotation database</a></li>
<!--
  <li><a href="/goldenPath/caePb2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#caePb21pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="caePb21pairwise"> 
      <ul>
        <li><a href="/goldenPath/caePb2/vsCe6/">C. brenneri/C. elegans (ce6)</a></li> 
      </ul>
    </div>
  </li> 
-->
</ul>
<h3>Jan. 2007 (WUGSC 4.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=caePb1"
    target="_blank">caePb1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/caePb1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/caePb1/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/caePb1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/caePb1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#caePb11pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="caePb11pairwise"> 
      <ul> 
        <li><a href="/goldenPath/caePb1/vsCb3/">C. brenneri/C. briggsae (cb3)</a></li> 
        <li><a href="/goldenPath/caePb1/vsCe4/">C. brenneri/C. elegans (ce4)</a></li> 
        <li><a href="/goldenPath/caePb1/vsPriPac1/">C. brenneri/P. pacificus (priPac1)</a></li> 
        <li><a href="/goldenPath/caePb1/vsCaeRem2/">C. brenneri/C. remanei (caeRem2)</a></li> 
      </ul>
    </div>
  </li> 
</ul>

<!-- C. briggsae Downloads -->
<a name="c_briggsae"></a>
<h2>C. briggsae genome</h2>
<h3>Jan. 2007 (WUGSC 1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=cb3"
target="_blank">cb3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/cb3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/cb3/chromosomes/">Data set by chromosome</a></li> 
  <li><a href="/goldenPath/cb3/database/">Annotation database</a></li>   
  <li><a href="/goldenPath/cb3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#cb3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="cb3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/cb3/vsCaePb1/">C. briggsae/C. brenneri (cb3)</a></li> 
        <li><a href="/goldenPath/cb3/vsCe4/">C. briggsae/C. elegans (ce4)</a></li> 
        <li><a href="/goldenPath/cb3/vsPriPac1/">C. briggsae/P. pacificus (priPac1)</a></li>
        <li><a href="/goldenPath/cb3/vsCaeRem2/">C. briggsae/C. remanei (caeRem2)</a></li> 
      </ul>
    </div>
  </li> 
</ul>
<h3>Jul. 2002 (cb1)</h3> 
<ul> 
  <li><a href="/goldenPath/cbJul2002/sanger/">Full data set</a></li>
  <li><a href="/goldenPath/cbJul2002/database/">Annotation database</a></li> 
</ul>	

<!-- C. elegans Downloads -->
<a name="c_elegans"></a>
<h2>C. elegans genome</h2>


<h3>Feb. 2013 (WS220/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ce11"
    target="_blank">ce11</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/ce11/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/ce11/chromosomes/">Data set by chromosome</a></li> 
  <li><a href="/goldenPath/ce11/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/ce11/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce11pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ce11pairwise"> 
      <ul>
        <li><a href="/goldenPath/ce11/vsSelf/">C. elegans self alignment</a></li> 
        <li><a href="/goldenPath/ce11/vsCi3/">C. elegans/C. intestinalis (ci3)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce11mult>Multiple
    alignments</a>
    <div class="collapse" id="ce11mult"> 
      <ul> 
        <LI><A HREF="/goldenPath/ce11/multiz26way/">Multiple alignments of 25 nematode genomes with C. elegans</A>
        <LI><A HREF="/goldenPath/ce11/phastCons26way/">Conservation scores for alignments of 25 nematode genomes with C. elegans</A>
        <LI><A HREF="/goldenPath/ce11/phyloP26way/">Basewise conservation scores (phyloP) of 25 nematode genomes with C. elegans</A>
	<LI><A HREF="/goldenPath/ce11/multiz135way/">Multiple alignments of 134 nematode genomes with C. elegans</A>
        <LI><A HREF="/goldenPath/ce11/phastCons135way/">Conservation scores for alignments of 134 nematode genomes with C. elegans</A>
        <LI><A HREF="/goldenPath/ce11/phyloP135way/">Basewise conservation scores (phyloP) of 134 nematode genomes with C. elegans</A>
</ul>
</div>
</li> 
</ul>




<h3>Oct. 2010 (WS220/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ce10"
    target="_blank">ce10</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/ce10/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/ce10/chromosomes/">Data set by chromosome</a></li> 
  <li><a href="/goldenPath/ce10/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/ce10/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce10pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ce10pairwise"> 
      <ul> 
        <li><a href="/goldenPath/ce10/vsCaeAng1/">C. elegans/C. angaria (caeAng1)</a></li>
        <li><a href="/goldenPath/ce10/vsCaePb3/">C. elegans/C. brenneri (caePb3)</a></li>
        <li><a href="/goldenPath/ce10/vsCb4/">C. elegans/C. briggsae (cb4)</a></li>
        <li><a href="/goldenPath/ce10/vsCaeJap4/">C. elegans/C. japonica (caeJap4)</a></li>
        <li><a href="/goldenPath/ce10/vsCaeRem4/">C. elegans/C. remanei (caeRem4)</a></li>
        <li><a href="/goldenPath/ce10/vsCaeSp91/">C. elegans/C. sp. 9 ju1422 (caeSp91)</a></li>
        <li><a href="/goldenPath/ce10/vsCaeSp71/">C. elegans/C. sp. 7 ju1286 (caeSp71)</a></li>
        <li><a href="/goldenPath/ce10/vsCaeSp111/">C. elegans/C. sp. 11 ju1373 (caeSp111)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce10mult>Multiple
    alignments</a>
    <div class="collapse" id="ce10mult"> 
      <ul> 
        <li><a href="/goldenPath/ce10/multiz7way/">Multiple alignments of 6 worms with C. 
        elegans</a></li> 
        <li><a href="/goldenPath/ce10/phastCons7way/">Conservation scores for alignments of 6 worms 
        with C. elegans</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>May 2008 (WS190/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ce6"
    target="_blank">ce6</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/ce6/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/ce6/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/ce6/database/">Annotation database</a></li>
  <li><a href="/goldenPath/ce6/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce6pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ce6pairwise"> 
      <ul> 
        <li><a href="/goldenPath/ce6/vsCaePb2/">C. elegans/C. brenneri (caePb2)</a></li> 
        <li><a href="/goldenPath/ce6/vsCb3/">C. elegans/C. briggsae (cb3)</a></li>
        <li><a href="/goldenPath/ce6/vsCaeRem3/">C. elegans/C. remanei (caeRem3)</a></li>
        <li><a href="/goldenPath/ce6/vsCaeJap1/">C. elegans/C. japonica (caeJap1)</a></li>
        <li><a href="/goldenPath/ce6/vsPriPac1/">C. elegans/P. pacificus (priPac1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce6mult>Multiple
    alignments</a>
    <div class="collapse" id="ce6mult"> 
      <ul> 
        <li><a href="/goldenPath/ce6/multiz6way/">Multiple alignments of 5 worms with C. 
        elegans</a></li> 
        <li><a href="/goldenPath/ce6/phastCons6way/">Conservation scores for alignments of 5 worms 
        with C. elegans</a></li>
        <li><a href="/goldenPath/ce6/multiz6way/alignments/">FASTA alignments of 5 worms with C. 
        elegans for CDS regions</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jan. 2007 (WS170/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ce4"
    target="_blank">ce4</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/ce4/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/ce4/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/ce4/database/">Annotation database</a></li>
  <li><a href="/goldenPath/ce4/liftOver/">LiftOver files</a></li>
  <li><a href="/goldenPath/ce4/nucleosome/">Nucleosome data</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce4pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ce4pairwise"> 
      <ul> 
        <li><a href="/goldenPath/ce4/vsCaePb1/">C. elegans/C. brenneri (caePb1)</a></li> 
        <li><a href="/goldenPath/ce4/vsCaeRem2/">C. elegans/C. remanei (caeRem2)</a></li>
        <li><a href="/goldenPath/ce4/vsCb3/">C. elegans/C. briggsae (cb3)</a></li>
        <li><a href="/goldenPath/ce4/vsPriPac1/">C. elegans/P. pacificus (priPac1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce4mult>Multiple
    alignments</a>
    <div class="collapse" id="ce4mult"> 
      <ul> 
        <li><a href="/goldenPath/ce4/multiz5way/">Multiple alignments of 4 worms with C. 
        elegans</a></li> 
        <li><a href="/goldenPath/ce4/phastCons5wayScores/">Conservation scores for alignments of 4 
        worms with C. elegans</a></li> </ul>
    </div>
  </li> 
</ul>
<h3>Mar. 2004 (WS120/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=ce2"
    target="_blank">ce2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/ce2/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/ce2/chromosomes/">Data set by chromosome</a></li> 
  <li><a href="/goldenPath/ce2/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/ce2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ce2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/ce2/vsCb1/">C. elegans/C. briggsae (cb1)</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce2mult>Multiple
    alignments</a>
    <div class="collapse" id="ce2mult"> 
      <ul> 
        <li><a href="/goldenPath/ce2/c_briggsae_pwMaf/">Multiple alignments of C.  briggsae with C. 
        elegans</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>May 2003 (ce1)</h3> 
<ul> 
  <li><a href="/goldenPath/ceMay2003/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/ceMay2003/database/">Annotation database</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="ce1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/ceMay2003/alignments/">C. elegans/C.  briggsae</a></li>
      </ul>
    </div>
  </li> 
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ce1mult>Multiple
    alignments</a>
    <div class="collapse" id="ce1mult"> 
      <ul> 
        <li><a href="/goldenPath/ceMay2003/waba/">WABA alignments</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- C. japonica Downloads -->
<a name="caeJap"></a>
<h2>C. japonica genome</h2>
<h3>Mar. 2008 (WUGSC 3.0.2/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=caeJap1"
    target="_blank">caeJap1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/caeJap1/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/caeJap1/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/caeJap1/database/">Annotation database</a></li>
<!--
  <li><a href="/goldenPath/caeJap1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#caeJap1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="caeJap1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/caeJap1/vsCe6/">C. japonica/C. elegans (ce6)</a></li>
      </ul>
    </div>
  </li> 
-->
</ul>

<!-- C. remanei Downloads -->
<a name="caeRem"></a>
<h2>C. remanei genome</h2>
<h3>May 2007 (WUGSC 15.0.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=caeRem3"
    target="_blank">caeRem3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/caeRem3/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/caeRem3/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/caeRem3/database/">Annotation database</a></li>
<!--
  <li><a href="/goldenPath/caeRem3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#caeRem3pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="caeRem3pairwise"> 
      <ul> 
        <li><a href="/goldenPath/caeRem3/vsCe6/">C. remanei/C. elegans (ce6)</a></li>
      </ul>
    </div>
  </li> 
-->
</ul>
<h3>Mar. 2006 (WUGSC 1.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=caeRem2"
    target="_blank">caeRem2</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/caeRem2/bigZips/">Full data set</a></li> 
  <li><a href="/goldenPath/caeRem2/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/caeRem2/database/">Annotation database</a></li>
  <li><a href="/goldenPath/caeRem2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#caeRem2pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="caeRem2pairwise"> 
      <ul> 
        <li><a href="/goldenPath/caeRem2/vsCaePb1/">C. remanei/C. brenneri (caePb1)</a></li> 
        <li><a href="/goldenPath/caeRem2/vsCb3/">C. remanei/C. briggsae (cb3)</a></li>
        <li><a href="/goldenPath/caeRem2/vsCe4/">C. remanei/C. elegans (ce4)</a></li>
        <li><a href="/goldenPath/caeRem2/vsPriPac1/">C. remanei/P. pacificus (priPac1)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- P. pacificus Downloads -->
<a name="priPac"></a>
<h2>P. pacificus genome</h2>
<h3>Feb. 2007 (WUGSC 5.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=priPac1"
    target="_blank">priPac1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/priPac1/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/priPac1/chromosomes/">Data set by chromosome</a></li> 
  <li><a href="/goldenPath/priPac1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/priPac1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#priPac1pairwise>Pairwise
    alignments</a>
    <div class="collapse" id="priPac1pairwise"> 
      <ul> 
        <li><a href="/goldenPath/priPac1/vsCaePb1/">P. pacificus/C. brenneri (caePb1)</a></li> 
        <li><a href="/goldenPath/priPac1/vsCb3/">P. pacificus/C. briggsae (cb3)</a></li>
        <li><a href="/goldenPath/priPac1/vsCe4/">P. pacificus/C. elegans (ce4)</a></li>
        <li><a href="/goldenPath/priPac1/vsCaeRem2/">P. pacificus/C. remanei (caeRem2)</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Yeast Downloads -->
<a name="yeast"></a>
<h2>Yeast (S. cerevisiae) genome</h2>
<h3>Apr. 2011 (SacCer_Apr2011/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=sacCer3"
    target="_blank">sacCer3</a>)</h3> 
<ul>
  <li><a href="/goldenPath/sacCer3/bigZips">Full data set</a></li>
  <li><a href="/goldenPath/sacCer3/chromosomes/">Data set by chromosome</a></li>
  <li><a href="/goldenPath/sacCer3/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/sacCer3/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#sacCer3mult>Multiple
    alignments</a>
    <div class="collapse" id="sacCer3mult"> 
      <ul> 
        <li><a href="/goldenPath/sacCer3/multiz7way/">Multiple alignments of 6 yeast species to S. 
        cerevisiae</a></li> 
        <li><a href="/goldenPath/sacCer3/multiz7way/fasta/">FASTA sequence for 6 aligning yeast 
        species</a></li>
	<li><a href="/goldenPath/sacCer3/phastCons7way/">Conservation scores for alignments of 6 
        yeast genomes to S. cerevisiae</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Jun. 2008 (SGD/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=sacCer2"
    target="_blank">sacCer2</a>)</h3> 
<ul>
  <li><a href="/goldenPath/sacCer2/bigZips">Full data set</a></li>
  <li><a href="/goldenPath/sacCer2/chromosomes/">Data set by chromosome</a></li>   
  <li><a href="/goldenPath/sacCer2/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/sacCer2/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#sacCer2mult>Multiple
    alignments</a>
    <div class="collapse" id="sacCer2mult"> 
      <ul> 
       <li><a href="/goldenPath/sacCer2/multiz7way/">Multiple alignments of 6 yeast species to S. 
       cerevisiae</a></li>
       <li><a href="/goldenPath/sacCer2/multiz7way/fasta/">FASTA sequence for 6 aligning yeast 
       species</a></li> 
       <li><a href="/goldenPath/sacCer2/phastCons7way/">Conservation scores for alignments of 6 
       yeast genomes to S. cerevisiae</a></li>
      </ul>
    </div>
  </li> 
</ul>
<h3>Oct. 2003 (SGD/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=sacCer1"
    target="_blank">sacCer1</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/sacCer1/bigZips">Full data set</a></li> 
  <li><a href="/goldenPath/sacCer1/database/">Annotation database</a></li>
  <li><a href="/goldenPath/sacCer1/liftOver/">LiftOver files</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#sacCer1mult>Multiple
    alignments</a>
    <div class="collapse" id="sacCer1mult"> 
      <ul> 
        <li><a href="/goldenPath/sacCer1/multizYeast/">Multiple alignments of 6 yeast species to S. 
        cerevisiae</a></li> 
        <li><a href="/goldenPath/sacCer1/phastCons/">Conservation scores for alignments of 6 yeast 
        genomes to S. cerevisiae</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- Sea Hare Downloads -->
<a name="seahare"></a>
<h2>Sea Hare genome</h2>
<h3>Sep. 2008 (Broad 2.0/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=aplCal1"
    target="_blank">aplCal1</a>)</h3> 
<ul>
  <li><a href="/goldenPath/aplCal1/bigZips">Full data set</a></li>
  <li><a href="/goldenPath/aplCal1/database/">Annotation database</a></li> 
  <li><a href="/goldenPath/aplCal1/liftOver/">LiftOver files</a></li>
</ul>

<!-- Denisova Downloads -->
<a name="denisova"></a>
<h2>Denisova genome</h2>
<ul> 
  <li><a href="ftp://hgdownload.soe.ucsc.edu/gbdb/hg18/denisova/">hg18 data files</a> (BAM
  format)</li> 
  <li><a href="ftp://hgdownload.soe.ucsc.edu/gbdb/hg19/bbi/T_hg19_1000g.bam">hg19 BAM file</a></li> 
  <li><a href="ftp://hgdownload.soe.ucsc.edu/gbdb/hg19/bbi/T_hg19_1000g.bam.bai">hg19 BAM index 
  file</a></li>
</ul>

<!-- Ebola Virus Downloads -->
<a name="ebola_virus"></a>
<h2>Ebola virus genome</h2>
<h3>Jun. 2014 (G3683/KM034562.1/<a href="http://genome.ucsc.edu/cgi-bin/hgGateway?db=eboVir3"
    target="_blank">eboVir3</a>)</h3> 
<ul> 
  <li><a href="/goldenPath/eboVir3/bigZips/">Full data set</a></li>
  <li><a href="/goldenPath/eboVir3/database/">Annotation database</a></li>
  <li><a href="/goldenPath/eboVir3/proteins/">Ebola proteins</a></li>
  <li><a href="/goldenPath/eboVir3/ebolaAbSequences.fasta">Ebola antibody sequences</a></li>
  <li class="collapse-section">
    <a class="collapsed collapse-toggle" data-toggle="collapse" href=#ebolamult>Multiple
    alignments</a>
    <div class="collapse" id="ebolamult"> 
      <ul> 
        <li><a href="/goldenPath/eboVir3/multiz160way/">Multiple alignments of 158 Ebola virus and 
        2 Marburg virus sequences</a></li>
        <li><a href="/goldenPath/eboVir3/phastCons160way/">Conservation scores for 158 Ebola virus 
        and 2 Marburg virus sequences</a></li>
        <li><a href="/goldenPath/eboVir3/phyloP160way/">Basewise conservation scores (phyloP) for 
        158 Ebola virus and 2 Marburg virus sequences</a></li>
      </ul>
    </div>
  </li> 
</ul>

<!-- gbdb Downloads -->
<a name="gbdb"></a>
<h2>Data access through the gbdb fileserver</h2> 
<p>
The /gbdb fileserver offers access to all files referenced by the Genome Browser tables with servers
in <a href="http://hgdownload.soe.ucsc.edu/gbdb/" target="_blank">North America</a> and
<a href="http://hgdownload-euro.soe.ucsc.edu/gbdb/" target="_blank">Europe</a>. Some files in the
browser, such as bigBed files, are hosted in binary format. For example, in the hg38 database, the
<em>crispr.bb</em> and <em>crisprDetails.tab</em> files for the 
<a href="http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg38&g=crispr" target="_blank">CRISPR track</a>
can be found using the following URLs:
<ul>
  <li>
    North American server: <a href="http://hgdownload.soe.ucsc.edu/gbdb/hg38/crispr/"
    target="_blank">http://hgdownload.soe.ucsc.edu/gbdb/hg38/crispr/</a></li>
  <li>
    European server: <a href="http://hgdownload-euro.soe.ucsc.edu/gbdb/hg38/crispr/"
    target="_blank">http://hgdownload-euro.soe.ucsc.edu/gbdb/hg38/crispr/</a></li>
</ul>
<p> 
Individual regions or the whole genome annotation from such binary files can be obtained using tools
such as <code>bigBedToBed</code>, which can be compiled from the source code or downloaded as a 
precompiled binary for your system (see the <a href="#source_downloads">Source and utilities 
downloads section</a>). The <code>bigBedToBed</code> tool can also be used to obtain a
specific subset of features within a given range, e.g.:
<p><pre><code>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/hg38/crispr/crispr.bb -chrom=chr21 -start=25000000 -end=30000000 stdout</code></pre> 
<p><pre><code>bigBedToBed http://hgdownload-euro.soe.ucsc.edu/gbdb/hg38/crispr/crispr.bb -chrom=chr21 -start=25000000 -end=30000000 stdout</code></pre> 
<p> 
Sequence files for assemblies used in alignment tracks, such as in the 100-species Conservation 
track, are also found on the /gbdb server. For example, you can find the underlying
<em>mayZeb1.2bit</em> sequence file for the Zebra Mbuna fish assembly, which is used in the 
hg38 <a href="http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg38&g=cons100way">Vertebrate Multiz 
Alignment and Conservation (100 Species)</a> track, but is not available as a standalone browser: 
<ul>
  <li>
    <a href="http://hgdownload.soe.ucsc.edu/gbdb/mayZeb1/"
    target="_blank">http://hgdownload.soe.ucsc.edu/gbdb/mayZeb1/</a></li>
  <li>
    <a href="http://hgdownload-euro.soe.ucsc.edu/gbdb/mayZeb1/"
    target="_blank">http://hgdownload-euro.soe.ucsc.edu/gbdb/mayZeb1/</a></li>
</ul>

<!-- Shared Data Downloads -->
<a name="shared"></a>
<h2>Data shared by multiple assemblies</h2> 
<ul> 
  <li><a href="/goldenPath/hgFixed/database/">hgFixed database</a> - contains information shared 
  across assemblies, such as pubsArticle for the publication tracks, or expression and 
  variation information including dbSnpRS</li> 
  <li><a class="changeHg" href="http://hgdownload.soe.ucsc.edu/goldenPath/uniProt/">Relationalized 
  UniProtKB databases</a> 
  <li><a href="/goldenPath/proteinDB/">Protein databases</a> 
  <li><a href="/goldenPath/go/database/">Gene Ontology (GO) database</a> 
  <li><a href="/goldenPath/visiGene/database/">VisiGene database</a>
</ul>


<!-- liftOver File Downloads --> 
<a name="liftover"></a>
<h2>LiftOver files (over.chain)</h2> 
<p>
The links to liftOver <em>over.chain</em> files can be found in the corresponding assembly sections 
above. For example, the link for the mm5-to-mm6 <em>over.chain</em> file is located in the mm5 
downloads section. The link to download the liftOver source is located in the 
<a href="#source_downloads">Source and utilities downloads</a> section. </p>

<!-- replacement for gbPageEnd.html on hgdownload machine- calling a hardcoded footer -->

        <!-- End container and page -->
            </div>
        </div>

<!-- gbFooterHardcoded.html is for the hgdownload machine to provide links back to https:genome.ucsc.edu -->


<div class="gbClearBoth"></div>
<div id="footer" class="gbRow gbFooterContainer">
  <div class="gbFooterColumn">
    <a href="http://www.ucsc.edu">UCSC Home</a>
    <br>
    <a href="http://www.soe.ucsc.edu">BSOE Home</a>
    <br>
    <a href="https://ucscgenomics.soe.ucsc.edu/">Genomics Institute Home</a>
  </div>

  <div class="gbFooterColumn">
    <a href="https://genome.ucsc.edu/license/">Licenses</a>
    <br>
    <a href="https://genome.ucsc.edu/training/">Training</a>
    <br>
    <a href="https://genome.ucsc.edu/cite.html">Cite Us</a>
    <br>
  </div>

  <div class="gbFooterColumn">
    <a href="https://genome-store.ucsc.edu/">Store</a>
    <br>
    <a href="https://genome.ucsc.edu/contacts.html">Contact</a>
    <br>
    <a href="https://genome.ucsc.edu/cgi-bin/hgUserSuggestion">Suggestions</a>
    <br>
    <a href="https://ucscgenomics.soe.ucsc.edu/careers/">Jobs</a>
  </div>

  <div class="gbFooterColumn">
    <div class="gbFooterIcons">
      <a href="http://www.facebook.com/ucscGenomeBrowser">
        <i class="fa fa-facebook-square fa-2x"></i></a>
      &nbsp;&nbsp;
      <a href="http://www.twitter.com/GenomeBrowser">
        <i class="fa fa-twitter-square fa-2x"></i></a>
      &nbsp;&nbsp;
      <a href="http://genome.ucsc.edu/blog/">
        <i class="fa fa-wordpress fa-2x"></i></a>
      <br>
      <a href="https://securelb.imodules.com/s/1069/index.aspx?sid=1069&gid=1&pgid=761&dids=645">
        <div class="gbAllCapsButton gbDonateButton" id="donateButton">
          Donate
        </div>
      </a>
    </div>
  </div>

  <div class="gbFooterCopyright">
    &copy; 2000-2018 The Regents of the University of California. All Rights Reserved.
    <br>
    <a href="https://genome.ucsc.edu/conditions.html">Conditions of Use</a>
  </div>
</div>

    </body>
</html>
    
